proposed/2022/dotnet-nuget-why-proposal.md

```diff
@@ -0,0 +1,124 @@
+# dotnet nuget why command
+
+- Author: [Kartheek Penagamuri](https://github.com/kartheekp-ms), [Pragnya Pandrate](https://github.com/pragnya17)
+- GitHub Issue [11782](https://github.com/NuGet/Home/issues/11782)
+
+## Summary
+
+Currently, there is not a great solution for developers to understand the nature of top-level packages and their transitive dependencies. The solution explorer in Visual Studio does a decent job at allowing a user to dive into each NuGet package & see all transitive dependencies from the top-level dependency.
+
+![](../../meta/resources/TransitiveDependencies/SolutionView.png)
+
+However, in comparison to Visual Studio, the dotnet CLI does not provide insight into “why” a transitive dependency is listed, although it does list everything that has been resolved as seen below.
+
+![](../../meta/resources/TransitiveDependencies/DotNetCLI.png)
+
+## Motivation
+
+A developer should be able to understand where every package in a solution/project originated from. This can help them understand complex dependency chains they may have within their projects. Having this understanding can help a developer promote a transitive dependency quickly & easily to a top-level dependency in case there is a security concern or conflict. The `dotnet nuget why` command aims to help developers understand package dependency chains.
+
+## Explanation
+
+### Functional Explanation
+
+The NuGet restore operation generates a `project.assets.json` file under the `obj` folder for `PackageReference` style project. This file maintains a project's dependency graph. Therefore, this file can be used in order to produce a dependency graph when the `dotnet nuget why` command is called. 
+
+### Technical Explanation
+
+The `dotnet nuget why` command will print out the dependency graph of a given package only if it is part of the final graph.
```

Comment:
`dotnet package why`? :D 

I don't think we'd be able to make it a first class verb.

Reply:
That's a great point Andy. @erdembayar also echoed similar concerns in this comment: https://github.com/NuGet/Home/pull/11875#discussion_r897028332 . Based on all of your comments it seems like `dotnet package why` might be the best solution?

Reply:
cc @JonDouglas and @baronfel as well.


@pragnya17 @kartheekp-ms maybe we need to have a separate discussion about this.

Reply:
https://twitter.com/nuget/status/1546878617199804418

Reply:
I get why people picked `tree`, but in reality, it's a graph. It feels kind of incorrect to use the incorrect data structure.

Reply:
Yes, others that I've discussed this with also expressed similar concerns about using dotnet nuget tree. After also sending out an internal poll, the results of which are below, @JonDouglas and I decided it would be best to use dotnet nuget explain.

If anyone else has any feedback, I would love to hear it!
<img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAABPAAAAPnCAYAAACht7KuAAAgAElEQVR4Aey9229UR7r/vSX+i9zGd5a4QMoF0kgTcQFSpFhKNEij6IdGDsiK7Iw8ROg3mEGWYzlpGHGKY2KyIW/gNYyIhxkSwxy8NwmJJzIkMOYw8cBszMiJYYKsHcsOBPV+hfbzqrp79XpqnXr1ya5uf5CQl9u9alV967OqVn3XU1X/9uh//j/hPxrAAAzAAAzAAAzAQGMzIPxDARRAARRAARRAARRoWgX+jYf1xn5Yp/6oPxiAARiAARiAAcMA/1AABVAABVAABVAABZpXAQw8IhCJwIQBGIABGIABGGgCBpr3cZWSoQAKoAAKoAAKoAAKYOA1wQM7kRdEXsAADMAADMAADPBYiwIogAIogAIogAIo0LwKYOBh4BF1AQMwAAMwAAMw0AQMNO/jKiVDARRAARRAARRAARTAwGuCB3aiLoi6gAEYgAEYgAEY4LEWBVAABVAABVAABVCgeRXAwMPAI+oCBmAABmAABmCgCRho3sdVSoYCKIACKIACKIACKICB1wQP7ERdEHUBAzAAAzAAAzDAYy0KoAAKoAAKoAAKoEDzKoCBh4FH1AUMwAAMwAAMwEATMNC8j6uUDAVQAAVQAAVQAAVQAAOvCR7Yibog6gIGYAAGYAAGYIDHWhRAARRAARRAARRAgeZVAAMPA4+oCxiAARiAARiAgSZgoHkfVykZCqAACqAACqAACqAABl4TPLATdUHUBQzAAAzAAAzAAI+1KIACKIACKIACKIACzasABh4GHlEXMAADMAADMAADTcBA8z6uUjIUQAEUQAEUQAEUQAEMvCZ4YCfqgqgLGIABGIABGIABHmtRAAVQAAVQAAVQAAWaVwEMPAw8oi5gAAZgAAZgAAaagIHmfVylZCiAAiiAAiiAAiiAAhh4TfDATtQFURcwAAMwAAMwAAM81qIACqAACqAACqAACjSvAv/mStGm9rRIS0vw/7OSuZStLItXMiq9jExVlkp9znoyJ2O/apPWUHmD5U/6fQXLlJ2Xmc9HJdO5SQ5fqY9EwVTnLx2WLc8k6ZH0t20y+m0wRfP7vIxu9c/bdmY+6kvpP/t2VLYV6zTumumT45so0CwKWO37Hqda42aRuOJyzP2xR9rWtkjLM5sl82mVbWDFueBEFEABFEABFEABFEABFECBUgo4buC1SMvGAzJViYfnqoE3f1Eyz7cqc9EYSK2y/qfb5cCZizI9uyRLC/b/+btTMnluRDI7tsimoom1UgbelGSKJlWLZOpt4D1ZkqnhzWGzc+162dY7ImOfT8tchF4XzxyTvo71hfPizDQMvFINBH9HgVoogIFXCxVJAwVQAAVQAAVQAAVQAAVQYDUr4KaB95PNslmZRJuGp8uvIxcNvHtj0lk04PKRX+t+dlgm75XhUD7Jyvzdi3K44/AKRRUup4GXlak9m2yzc22b9JyZlqUnKZFYmJGLQxkZIwIvpWB8DQVqrwAGXu01JUUUQAEUQAEUQAEUQAEUQIHVpYCbBt6eizK151ll3GyWkbtlVoxrBt4PU5LZ6E/XNFF3207MSBnWXZkC1Ovry2fgzZ3ZZkXetT6fkYs1neFFBF69KCFdFNAKYOBpNThGARRAARRAARRAARRAARRAgfIVcNTAmxJZGJftZl0eLxJv66jMpY26Mjo4ZeBl5WKvnjbbKtvOzJVfW06csUwG3t8Pyyav7s3PjRmZ+qHWAmDg1VpR0kOBKAUw8KJU4TMUQAEUQAEUQAEUQAEUQAEUSK+AuwaeiCz9cbuKwGqVbWfLCL9yyMDLXsnIs8qM2rRnqgEj7zyolsPAm5ORnyjzdu02Gb3nXb+WPzHwaqkmaaFAnAIYeHHK8DkKoAAKoAAKoAAKoAAKoAAKpFPAaQPP7BI61qEi19Zul/GFdAVLH4FXqSGV9ryAGfXjTGWbcqQsdtzXlu5elJHeTmnbuM6Panxmk7R19snIxzOJa8pZg29lRBajI4OfVbnLZPbzPmXctkjVu8PGidLgu9Au3Z2U0YPbZcsLm2SdqoPWH7Xl6vXYmSmZSzNH+0lW5m6Oycgb4bTWbWyTLTsOyOjnyYwEJbaY0Txk52TyVJ90Pu9tMNIi6zZuke0Hx2R6KZhK4feFaRkz5VTnmDJu2XFMLt6NO8lOy7X8FHOXW9NyUkaP9knnC22yXkcdp7w/i2lJQpu0NCMXA7q3PLMpV7djN9Np6F+ncPRkSWY+zm+u0/Yjv502zHT2jlhre8bqH0q0nA+Wr7zZ+RmZNBvjdLaJLmtLyzrZ9EKn9J26KDOpZIzPc/beZL6NLmq5Tjb9bLscOBe/5ufSzTE5sGOLylOrrH9+i2w/mjY/ImnrZv7MNr/v2Doq+nXa/JXRXD78TZZMPowuk+naoIhqz96bCrdva9dL28+2yzHdZ7H7d4R6fIQCKIACKIACKIACKNCMCjhu4InIvVHZps2J3ovpotdSR+DFD6iSKzzleX8/bEXfdZ5LNcpLvnQ5f12akmM/U6ad0tIy4J7plJF/RLs91gAv7nz9uTZsyslr7rtLMv4LFX1XV8OzMSPwlv4+Kj2hnYyVZmXUxfylw7KtaBjEpOGl98wWOXxJD9vjK9dipsDD0rXDslkbVF663s+1bXLgimYwKzOnOi1z0mI2d16rbB6eLtkmuJYfkazMnOmTzYFNbcLly9dJ6/M9Ml4yCjW6TZo71yNtSbq3tMimX46XtUTB0rVjsqVk3lul7eBk7uVAlP7x9KT9S/3Lm/3HqPT9JEX7aVg0G+z8sdTSCBF5jttp27svWlqk9fkD9hICP8zIyM9L5GvtZjl8U99P0bqmrZtIAy9N/5IyH8XcPZmTsV+1WS9xIu+LjT0yNisiGHhF6ThAARRAARRAARRAARRobgXcN/BEZObEZv/Nf8uzkrlUelDiSgTezIk2lfftMr6c/l3Errdm84z1z7dJWyBqKz9A2iQZy0DJwz911Hzf+29He5loG/9vheOjU5XfNdmL0qcGrs8OVbADceqrN56BN3cm2tDKRd2ZOlJRark6jTVTszI9vDlykFys00hjrzLDzEwjL65paKJoTF51NGixzr0Na4L5S+K29JqSQZNipfMjgWg5z6Aoah+sR6NPyXUgw+aQvRGMp2Eg0q+gfdqp/XNnoxn08u5HYeXNR5PuxT3KHI5lMvWNW/hi/ctrceMxaiIjC+2hFTGZ+3t0G+qXLJhne6dt7z4Oapjj4ycjMmPWgf1hWg7/1I94bEm6n1IsP2CVMaFuQgZecGMmT5eo+zpFPnIa/TAlB6JeTnhlDN4XJt1L+iXfNhmN3HHcrwGOUAAFUAAFUAAFUAAFUKBRFWgIA0+ezNhrom08UHoaqhMReGYKsBq4doxZ047qCk1wExATPfX5nGT1RiBm+t7NUenRu+OWnKYcHIDWuBTXDihTqVUyV2qcvpVcYxl4thnTIi2mTj+ekfkIPzv7rZnyl5EtQ9Fm6swp27wzEV4jVwJ8GK2y8zLz8eFAtFWZhtnPt+c3pFm7WQ4EGczOycU3AtE2Px+TuaLh1yptb1y0p+GZKb8fZ+yoshLcWiaFA/kpGnhr18u2PaMyNbtk35s57edk8qBdT8mGtn1v9h08kDdNTWTYmcA0TKPh5wcCEZHPyuGb1g0S+sUyPo1hVWBwSbcr5qyFaRktRlE9K8/+WLWDCSZR6IKJH9S/vHluWmV9R0ZGr8zJUvBei9Lxx4dlOqhHsRx2njt/kV/ntfWnB6wpx+br2XsXJRMwszrPzcnUnk35l0Jr2yTzceCejbifWn8xLknvjax7I6FubAMvIwcKy1us+9nhUN5z9f/LQj494/PnY4n5kCdzMqqXzGhplbZfjcjUvYDoOc39Nql14yYV5Y6BV0SNAxRAARRAARRAARRAgaZToDEMPCN7YFfSTcMlIrOcMPDswdqzpfJcM7wCawduzMhk0gguEElRjklQa4PNGiS21DtisYEMvAD/rR0jMlPhrrzZawVjpzCw3vTLsdLTJ0NRP9tkLCHSxTIFctfZJiNmulvkv6xMvvGsilRdJ+sK0zOTosKyn9prJW7/YzzkruXHGHijQ5MyH2v0eEIFtFnbJ5MBP8P7ZtEU9AwT89NEKMXqHm5XW/sn46cjZ6cko424Uu2KiIRMZ5OnBJPIL0uaI7t9zUWp1bK8IjJ15rBMJnDu5TJ7SW9U1Cp9n8dVUkSet47E338/TNqaP7OuMKU8KdIvuOt5cjtq3RsJdWO3zV50ZX6KtKeD/dMsh6AiBVs2J7QBInb6rbLtdInpyEuTktEvnwrtDBF4di3wGwqgAAqgAAqgAAqgQPMo0DgGnmRl6qB+o+9NtYupDBcMPGttnnpuxmBrYG8CkW4HV+ucMkyCWht41mCyJSPR8WN2eSv/rVEMvMBGKCWnUiYpEkjLm5aXdIr3t2/HZJtaSy3J6LXrsUU2n5jxUon+uTAmndp4Mscl8zYjIy+oyK6E9TFdy0+0CDGf3h2RtqI2z8rhv8d8LzQtt1X6Po0zkrw0sjLZr0yWtRmZijEV506ppQzSTok07bYXMeaVIcEk8nKV7mfQDKttedPlwfuWzWL8y5pgnkv0Y2Y39nOdytzO817yfrKYaUnkwLo3EurGNtjS3J8iMjsim716b0noA7OT0qfbljcSjGRPcvPTrJGrzmtpIQJPy8MxCqAACqAACqAACqBAcynQQAaeWftnyn7jvnU0PnLBQQMvtdkVMP689bGCP6N3Z7U3gUgyWSyUn0zL4WJ0TdLUVXsAmrpM1sXif7EGk/9nVErEYMQnlOovDWLg3dQboZSe5phYdGtTlfLTstajTDB7rHps6ZSxkrtH23Vh1mqMj2LySzh1UJlPCby4lh+/BGmO7HsuPtLQ/p4xQFPdP5cyyiDaIqORm2UEDKpy1qYMmrMJJlEaNfzv1LO8/lXSHlmMxU5bDeS51LRSc/Fgf5D4gsXL7ZQcUMbWloRoNivfCXVjG3jp7k+RORn9P8pkf2PSy6D100TT+v1bmvbCP316SEfvYuD5ynCEAiiAAiiAAiiAAijQbAo0loFn1gWypiq1xE+zWa0GnrUJRJuM3E2LrJl25Q+04gd89gC0rgbe1tE6rxlom0bRhmha/YID7doNJKf2KJMqzYA/IcvWpiovjEiJuLhwSlZkT3w0mGUKJBhr+gLWOS19crFU8JgEp93FR2xaaTuQH13u0sdpObXvzbZSUY/ehQMGUeQ9bdV7Oe2KuUhgLdAEk8jLUrqfdSxvugxY37IMrti2y85zfDurk7bPaUmINPXPsplJmrZs3RsJdWOVL+X9afJjpR+jy+Qbft+Trnx+ScVaN7V27a66AocogAIogAIogAIogAIo4IQCDWfgiQTW91kbsxaXgwZefORMgIXAgNqPTFCDnLjpSNZgJt7QCFwx96s10IodyNmDycjBflTiKT+z8lCJwZTyOvmv2YNcNw08O/KpujwGojMrWpPRjuyJY9qqx1iW7MqyDIK0dZ/yPnctP3bJC789ycrSwrzMXJmUi2eOyYGDfdL5QpsEdySNZyBwb16KvErEh4HzIjaOWfrjdj9Cau2Bsqe2V6J/REYDHwXyXcPyBi5k/ZpdWpL5u1My+fGoHDt4QPo6I3ZUjjGqgusUpms/7XYqrTGbVvO037Puz5QmuBHOOi9Sl0Abd3be0rvkL1Z/iYFXUi++gAIogAIogAIogAIo0LAKNKCBZ3Y4HM/vallYWydyl72UA/vKBlSmvgODx4hBr8yPyTa1/k+6aAsRyQ3kl2RpIfh/WkbUdKSogbw10FbXjjMBYz+PNV1SlLuK28EaTLIGnsiTScmoekw34I+rgCoHyrlkbTMhisHc3bFHmc2xLNn5LD3Qt7+f+y3lfW5x5UB+iiVZmJGLR7fLlo3rfINM1XfU/Rmneao2qXhhfVD6np47vcXPX6QJo9MLH1eifziV4Cel8x08I/97mec9WZKZj4/J9p9tKmwgodiOq6tYjcq8di7D6e65YFnTap72exXdn2kMvGrbOAy8YNXzOwqgAAqgAAqgAAqgQJMq0JgGnhkUnN0mrcXBU8Ti5SkH9vUc9AbTbolZ/yc9W6UHctYgq6hPigFn8LuxJkclA9AySnhmm28UtMStx5U+veRvltYz+fzAX+sxkKxpmrWoO1uzuKl5aU0BraDFbqwBos8QkZT3uWv5kSfzMjm0pbQh9MwmaXths7QV16dM2AggzUuFgHz5X0tzUYl++lLVnq/T8o9L59v/rj5Kf978pcOypbAjcpSZmv9snWx6oU02P6/WYovlN/21/Rzb91y8geufYY7Sap72exXdn2kMvGrbuGrPt2XjNxRAARRAARRAARRAARRwVoGGNfDy6yqptcF+nJHJH5TOKQf2QZMtfYRTmoFYYO2nHx+WaZXF8g9LD+SsQdba9dL2Qltl/4/G7f+aptzll6x4hjUFuEXipmgWv1/VQWk9y0q+HgPJmqZZi7qzNcPACxCSqt2J2Jl17XrZvCMjI+cmZfLKjMwvLEm2uBusrXm8gVNp/ZY+L63JE1Cj+Gu15xcTsg5K59v6evGXdOdlr2Rkk/Vio1XW/3S7ZE6NyeTnUzLz7ZIsqXUarbYXA6+odkldqm3jqj2/mFMOUAAFUAAFUAAFUAAFUMBtBRrYwBORe6PWFNVNB6ekOJ5KNZA2lZNuMBeuxnTnzZ3arCLKyt/1075u6YG8iUwsRoqUsU6RfZ2k39KVOymFxL9Zm3C0SEuVmzYkXktK65l8fuCv9RhIWmnGbxoRyEnMr+nWr4s5ufBxOs0qMWxKDvSjMpbyPncpP9nP+1T08DrpPDUtS0WzLqqQ6TSvZ1tm6ZdqEwW7HNb5sdG99jmlf6u0LUpxXnZS+tQurut+PiLTJXZSTsdvimuHCp62/u0T02qe9nvpymfnwfxW8jyrjSt3g5T6bR4ULgmfoAAKoAAKoAAKoAAKoMDKKtDYBp6IzJzQBtkmOfz3gqApB/YVD3rTrtszOyKbVRRHawWDXx+RFAO51OX2Uy3vqJIBaDlXsDdaaGnZXMZOuuVcx3w3hZ7lJGkNRGu1mLqtd9+nRYu6nJwVvmuX99kabGIRl5+0poAuRMmBvv6yd5ySd3fyY/P97B710sErU+hnYO3CM3GL/Nus1DKa2FoDr+wXA3aZ46I2Q8Uu+UH9ymutJfrjjEyluO2sHZ6JwCvWXsn7OvDSJpN6M5LCJf5+WJ4t9rG1aneL2ecABVAABVAABVAABVAABZxRoOENPHkyLYc3qjXefjIiMyaaJeXAPmjgpZ6yeXdE2oqDhhaJHyybwaua6tuySTJXUowGIxGxDZjIqXT3RmVLMV8VRDNEXld/WOmgWaeRfGymrvkDshZp7RiVucQIpeT04v+aQs/4k8N/qYuBZ+extcropamDisWfjMhcuBTJn1jcx69R6I5hli+OO/kJ3D9pzIqAwRF53+eKGUg7amOdyNotfV720z4/srdlu4wvRSYU/eGTKcmoaLZGMPAsXlKtXWp2R1f9EAZekYWSBp5My2G1xmO5Lxas9Fsw8IrCc4ACKIACKIACKIACKNB0CjS+gWeq5O+HrbWKtp2eK8PAm5NRtbNrWoPEnhqbZOCF89eyMSOT5QyAi9jZZk70QD4wGBqqbtW94qWLB6UH+8WvVnxgl9NMCd48NO1Pj6443eCJ9nWi9Qyek/B7XQw8EYu1tdtlvMRUvoQcij19U0WsJp2k/mZFvCYYgJYBktJ0tAbisQaIyow5TGnUO5Mfi5F4A1SX0q6zldnEQqx8t0jnufQNWDD/7ht4druQavfwwJTbllh+K2k/7fykbafSMp/2exXdn2mm0OY23FAvFsxasalf2MzJyE+UcYqBp5sOjlEABVAABVAABVAABZpMgeYw8CQrUwc3+REia7fJ2JmM/3tLRuK2ZDD1ObVHDR7W9slkqQC5HyYloyIGjMEUH4GXJ2Z6WOXPRMht7JOLcTPhYiFLN5CzDZ9tMnovNsEK/mAPQLedLbsQ6a5p1jfUUTstrbJ5eKrEWmHpkva/lU5P//sljiyTo4aRIAtj0lmMqmyRZ9+YrNzMNBGrmt2tZUQ3LozLdlUnSSZOWlNAK1qRQdBoBt7SuGxXdVlyuuCTORndqg2KFTLwJGCUbMzIlN40SFekPg5GSJuypzR0dTLRx3ZbVKoN9tModV5gym+KCLy502rtUVNGDLyi3Gnua9vkbZVtZ9LFBtu70Zv7pIbtbrEEHKAACqAACqAACqAACqCAGwo0iYEnIgFTrfUnm9UU12QDzx48tMimpKi1J3My9otnlTmYH1yXHDyagXiHMgrNIG9tm/ScKbWAvQJlaUoOvOAP5mMjMQJGS+qIvydLMn2mR45dU9cMHQYiFvurMJNCadsfzI/3WJGVxihd97PDMvmt/b3E3xZm5OJQRsYiz2kQA09EbAM4pZn57UUZGQ8brGY6ZKsykTb9crz0FOUfpuXwTxW/G5OjZDDwzH0a1e7YBmrrL8YlPpYtYrfalpUy8ESC3JSc2v5kTsZ/GXhx0RAGnrnfVBtfIuo1vFstBp5uk9MYePJkJhBJV3qpieyVA9KmXijkN2/CwNPac4wCKIACKIACKIACKNBcCjSPgSfhAWb+gT5uIK0qMhiVZKK9Dk7KXCASL3tvUg7/bF3OvGvt7ZMeZYKUNPDM5YyJ9/P8+X7eWqTlmU3S2TsiY1dmZH5hSZYWliSbzeZ+Ln07I1PnRiSzY7OsU9cz58caeGba0tltlknTsnazZM5MhcpkspVdmpOpMxnZ9qO8QVOqLNbg1qzp97myIJamZeRsUryj0j3F4dzZzlC5W1paZX1Hn4ycm5KZb/N6LWVNOfLHczcnZexURrb/dH1Bg7hBnW3gbTkxnde8UAemHhL/B/iwpxnGXTNFoaO+8mRajj2vDLScmZmRsZvzkg1ON8uZlltyukUzEjaGWp/vkZErc+G0nizJzMeHZcszvnHcsrZ0VCcGXny7Mz2kzKGWFtn0y1GZVreQqX7d1rT8pE/61DT/6Do1Z5WKLIsCq5zz5mUs8BLCcDN6M5D5J1mZvzkqPR6vGw/L4TcUP85H4InITb0xgomY7pHRvwfKmZ2TyaH8fWY22+nr3eK/2CECrwhbKgPPMH/tQOCFzSbpORXRZ/2gdW+V7cOH1W70NW53i6XgAAVQAAVQAAVQAAVQAAVWXoGmMvBEzELitsmRN8qiImFs8SOjKIxJsrFN2l5ok03awMhNH6twsPxkSaaOeoM+NagNmHOWwRf1t7VttnFmF8cMh2R6eLNt4nnpPLMpV6a2FzZFmGOlpwPLXXtn3Vxec2kW0qvZAD1fqKVrx2wDyStH6p9xgzrbwCupeeB6ISOlXlNovbr9YUoOeKaIlZdWWf98BKdJJm9cdJRivm1jhNm8drMcvhl0Lr0M+j8x8OINPPlhSjJ6451CXUa2NTmz1J5GG+KuKHuFbVI5xl9M3lta1smmFwyDgTblmU4Zu2eWKVBtXc3ah3qWN2xy++1cm9j3Rn7K59wZNY0WA69IZVoDz5xgNNTRwV6b7N0bbYWXTPnPW6Xt4JRk693uFkvCAQqgAAqgAAqgAAqgAAqsrAJNZuCJSHD6aG5wXNrAM9Uw/2kmYkqOGni2tEjr85nC2nWVDh4LFf7tpBzu8CLE7Gt4g5aon60/2hYbSReF0vzHpctUvM7a9bJtaFLmgxFdEQnHDbRyadVsgK4u/GReJoe2yfrQlKlS2q2Tzb1jMhPpOTWYgWfkyOmQ0gB+ZrMc0NGRSs78YVZmzvSUZD7Ph4l6PCZTgSCkUJKFDyoxbMoZ6Bev22hr4HkZvzcuPREmXvFeNO3Wxh4ZmzUn2JyuqIFnsvPDjIxERRJbprJuKxvRwMtHTEdOAbbKuUl6zuXXa0vHbyX9Rtr69+DK/0x7D6b9Xrry2Xkwv5V7Xpp+uKVlk/ScmcmvBYqBFxadT1AABVAABVAABVAABZpSgeYz8KKmj0auRRVTn0szcvFUn3TqSJK166Wts09GPp5RmyhUMhCLuOYPczKdmyK7RdqeDxh65rovtEln74H8dNH5SBcqItHAR7lpkCPS19kmdgRDPmqms/eYjH6uyxY4P+bXpb+PSaazzTfVcjplZOwfFeYz5jrWx0+yMndzTEbe2C5bXlDXLgyq85EandJ3cETGbkZMCbUSswfGlnliDdKjTcKQkbKcA0kzTdZME/6ZrUHrj9pky45MgFWr0OFf4vjI8dcpfafGZPpeeXWa1hTQmSl3oJ87t1ENPJN5s+bkuQOBOlwnm362XQ6c02tj2pyGuCuKWGmbVNl5S3cvykhvpxWNlufvgIxesdderISHYrFiDyrLd7lTjZdujsmBHVustnPdxi2y/eCYTKsdodPxW0me09a/LVRazdN+L1357DyY3yo678m8TJ2JuDdeMO3RRZnRLxKWs90NF49PUAAFUAAFUAAFUAAFUGDZFHDGwFu2EnMhFEABFEABFECB5lDg2gE17bZPLpb3rqE5NKAUKIACKIACKIACKIACq0IBDLxVUc0UEgVQAAVQAAWaTwGzYVMxevrHh2W6+YpIiVAABVAABVAABVAABVAgpwAGHiCgAAqgAAqgAAo0oAJm4yq1xEHvxfy6eA1YErKMAiiAAiiAAiiAAiiAAqUUwMArpRB/RwEUQAEUQAEUcE+BhTHpVOuVbv+jXhzPveySIxRAARRAARRAARRAARSoRgEMvGrU41wUQAEUQAEUQIEVUMBE37X602fX9skk69+tQD1wSRRAARRAARRAARRAgeVSAANvuZTmOiiAAiiAAiiAApEKTP0/PTIS2L048ovmwydLMnmwTW1e0SLbTs/Ffp0/oAAKoAAKoAAKoAAKoEAzKICB1wy1SBlQAAVQAAVQoIEVmNqTX8uu9UfbJHNmUma+jQin+2FeZj4+JtufV5F3LS3S2jEqc08auPBkHaLf7t0AACAASURBVAVQAAVQAAVQAAVQAAVSKICBl0IkvoICKIACKIACKFA/BTwDr7ijbGFtu3Ub26Tt+fVWtJ3+TutPD8v0D/XLFymjAAqgAAqgAAqgAAqggCsKYOC5UhPkAwVQAAVQAAVWqQJz57bL5mfUjrJqcwpt2BWP17ZJz5lpWSLybpUSQ7FRAAVQAAVQAAVQYPUpgIG3+uqcEqMACqAACqCAewo8ycrczTEZeWO7bHmhTdav1YbeOtn0Qpt09h6T0c9nMO7cqz1yhAIogAIogAIogAIoUGcFMPDqLDDJowAKoAAKoAAKoAAKoAAKoAAKoAAKoAAKoEA1CmDgVaMe56IACqAACqAACqAACqAACqAACqAACqAACqBAnRXAwKuzwCSPAiiAAiiAAiiAAiiAAiiAAiiAAiiAAiiAAtUogIFXjXqciwIogAIogAIogAIogAIogAIogAIogAIogAJ1VgADr84CkzwKoAAKoAAKoAAKoAAKoAAKoAAKoAAKoAAKVKMABl416nEuCqAACqAACqAACqAACqAACqAACqAACqAACtRZAQy8OgtM8iiAAiiAAiiAAiiAAiiAAiiAAiiAAiiAAihQjQIYeNWox7kogAIogAIogAIogAIogAIogAIogAIogAIoUGcFMPDqLDDJowAKoAAKoAAKoAAKoAAKoAAKoAAKoAAKoEA1CmDgVaMe56IACqAACqAACqAACqAACqAACqAACqAACqBAnRXAwKuzwCSPAiiAAiiAAiiAAiiAAiiAAiiAAiiAAiiAAtUogIFXjXqciwIogAIogAIogAIogAIogAIogAIogAIogAJ1VgADr84CkzwKoAAKoAAKoAAKoAAKoAAKoAAKoAAKoAAKVKMABl416nEuCqAACqAACqAACqAACqAACqAACqAACqAACtRZAQy8OgtM8iiAAiiAAiiAAiiAAiiAAiiAAiiAAiiAAihQjQIYeNWox7kogAIogAIogAIogAIogAIogAIogAIogAIoUGcFMPDqLDDJowAKoAAKoAAKoAAKoAAKoAAKoAAKoAAKoEA1CmDgVaMe56IACqAACqAACqAACqAACqAACqAACqAACqBAnRXAwKuzwCSPAiiAAiiAAiiAAiiAAiiAAiiAAiiAAiiAAtUogIFXjXqciwIogAIogAIogAIogAIogAIogAIogAIogAJ1VgADr84CkzwKoAAKoAAKoAAKoAAKoAAKoAAKoAAKoAAKVKMABl416nEuCqAACqAACqAACqAACqAACqAACqAACqAACtRZAQy8OgtM8iiAAiiAAiiAAiiAAiiAAiiAAiiAAiiAAihQjQIYeNWox7kogAIogAIogAIogAIogAIogAIogAIogAIoUGcFMPDqLDDJowAKoAAKoAAKoAAKoAAKoAAKoAAKoAAKoEA1CmDgVaMe56IACqAACqAACqAACqAACqAACqAACqAACqBAnRXAwKuzwCSPAiiAAiiAAiiAAiiAAiiAAiiAAiiAAiiAAtUogIFXjXqciwIogAIogAIogAIogAIogAIogAIogAIogAJ1VgADr84CkzwKoAAKoAAKoAAKoAAKoAAKoAAKoAAKoAAKVKMABl416nEuCqAACqAACqAACqAACqAACqAACqAACqAACtRZAQy8OgtM8iiAAiiAAiiAAiiAAiiAAiiAAiiAAiiAAihQjQIYeNWox7kogAIogAIogAIogAIogAIogAIogAIogAIoUGcFMPDqLDDJowAKoAAKoAAKoAAKoAAKoAAKoAAKoAAKoEA1CmDgVaMe56IACqAACqAACqAACqAACqAACqAACqAACqBAnRXAwKuzwCSPAiiAAiiAAiiAAiiAAiiAAiiAAiiAAiiAAtUogIFXjXqciwIogAIogAIogAIogAIogAIogAIogAIogAJ1VsAZA+/JkyfyP//zP5LNZvmPBjAAAzAAAzAAAzAAAzAAAzAAAzAAAzAAA8vCgPGjjC/l8r8VN/D+93//F+OOG3JZbkjMYcxxGIABGIABGIABGIABGIABGIABGICBOAaMkWd8Khf/rbiBR9QdN07cjcPnsAEDMAADMAADMAADMAADMAADMAADMLCcDBifysV/K2rgmfDE5awErsVNDwMwAAMwAAMwAAMwAAMwAAMwAAMwAAMwkMSAi9NpV9TAI/qOGybphuFv8AEDMAADMAADMAADMAADMAADMAADMLDcDLgYhbeiBt5yVwDX46aHARiAARiAARiAARiAARiAARiAARiAARgoxYBr02gx8NhAgmnMMAADMAADMAADMAADMAADMAADMAADMAADigEMPKVAKbeTv+OIwwAMwAAMwAAMwAAMwAAMwAAMwAAMwAAMLDcDyr5y4pAIPOWuLjcMXI8GCAZgAAZgAAZgAAZgAAZgAAZgAAZgAAbcY8AJ105lAgMPA48QWRiAARiAARiAARiAARiAARiAARiAARiAAcWA8s6cOMTAU5WD4+2e402dUCcwAAMwAAMwAAMwAAMwAAMwAAMwAAPLzYATrp3KBAYeBh4OOwzAAAzAAAzAAAzAAAzAAAzAAAzAAAzAgGJAeWdOHGLgqcpZbjeX6/EGAQZgAAZgAAZgAAZgAAZgAAZgAAZgAAbcY8AJ105lAgMPAw+HHQZgAAZgAAZgAAZgAAZgAAZgAAZgAAZgQDGgvDMnDjHwVOXgeLvneFMn1AkMwAAMwAAMwAAMwAAMwAAMwAAMwMByM+CEa6cygYGHgYfDDgMwAAMwAAMwAAMwAAMwAAMwAAMwAAMwoBhQ3pkThxh4qnKW283lerxBgAEYgAEYgAEYgAEYgAEYgAEYgAEYgAH3GHDCtVOZwMDDwMNhhwEYgAEYgAEYgAEYgAEYgAEYgAEYgAEYUAwo78yJQww8VTk43u453tQJdQIDMAADMAADMAADMAADMAADMAADMLDcDDjh2qlMYOBh4OGwwwAMwAAMwAAMwAAMwAAMwAAMwAAMwAAMKAaUd+bEIQaeqpzldnO5Hm8QYAAGYAAGYAAGYAAGYAAGYAAGYAAGYMA9Bpxw7VQmMPAw8HDYYQAGYAAGYAAGYAAGYAAGYAAGYAAGYAAGFAPKO3PiEANPVQ6Ot3uON3VCncAADMAADMAADMAADMAADMAADMAADCw3A064dioTGHgYeDjsMAADMAADMAADMAADMAADMAADMAADMAADigHlnTlxiIGnKme53VyuxxsEGIABGIABGIABGIABGIABGIABGIABGHCPASdcO5UJDDwMPBx2GIABGIABGIABGIABGIABGIABGIABGIABxYDyzpw4xMBTlYPj7Z7jTZ1QJzAAAzAAAzAAAzAAAzAAAzAAAzAAA8vNgBOuncoEBh4GHg47DMAADMAADMAADMAADMAADMAADMAADMCAYkB5Z04cYuCpylluN5fr8QYBBmAABmAABmAABmAABmAABmAABmAABtxjwAnXTmUCAw8DD4cdBmAABmAABmAABmAABmAABmAABmAABmBAMaC8MycOMfBU5eB4u+d4UyfUCQzAAAzAAAzAAAzAAAzAAAzAAAzAwHIz4IRrpzKBgYeBh8MOAzAAAzAAAzAAAzAAAzAAAzAAAzAAAzCgGFDemROHGHiqcpbbzeV6vEGAARiAARiAARiAARiAARiAARiAARiAAfcYcMK1U5nAwMPAw2GHARiAARiAARiAARiAARiAARiAARiAARhQDCjvzIlDDDxVOTje7jne1Al1AgMwAAMwAAMwAAMwAAMwAAMwAAMwsNwMOOHaqUxg4GHg4bDDAAzAAAzAAAzAAAzAAAzAAAzAAAzAAAwoBpR35sQhBp6qnOV2c7kebxBgAAZgAAZgAAZgAAZgAAZgAAZgAAZgwD0GnHDtVCYw8DDwcNhhAAZgAAZgAAZgAAZgAAZgAAZgAAZgAAYUA8o7c+IQA09VDo63e443dUKdwAAMwAAMwAAMwAAMwAAMwAAMwAAMLDcDTrh2KhMYeBh4OOwwAAMwAAMwAAMwAAMwAAMwAAMwAAMwAAOKAeWdOXGIgacqZ7ndXK7HGwQYgAEYgAEYgAEYgAEYgAEYgAEYgAEYcI8BJ1w7lQkMPAw8HHYYgAEYgAEYgAEYgAEYgAEYgAEYgAEYgAHFgPLOnDjEwFOVg+PtnuNNnVAnMAADMAADMAADMAADMAADMAADMAADy82AE66dygQGHgYeDjsMwAAMwAAMwAAMwAAMwAAMwAAMwAAMwIBiQHlnThxi4KnKWW43l+vxBgEGYAAGYAAGYAAGYAAGYAAGYAAGYAAG3GPACddOZQIDDwMPhx0GYAAGYAAGYAAGYAAGYAAGYAAGYAAGYEAxoLwzJw4x8FTl4Hi753hTJ9QJDMAADMAADMAADMAADMAADMAADMDAcjPghGunMoGBh4GHww4DMAADMAADMAADMAADMAADMAADMAADMKAYUN6ZE4cYeKpyltvN5Xq8QYABGIABGIABGIABGIABGIABGIABGIAB9xhwwrVTmcDAw8DDYYcBGIABGIABGIABGIABGIABGIABGIABGFAMKO/MiUMMPFU5ON7uOd7UCXUCAzAAAzAAAzAAAzAAAzAAAzAAAzCw3Aw44dqpTGDgYeDhsMMADMAADMAADMAADMAADMAADMAADMAADCgGlHfmxCEGnqqc5XZzuR5vEGAABmAABmAABmAABmAABmAABmAABmDAPQaccO1UJjDwMPBw2GEABmAABmAABmAABmAABmAABmAABmAABhQDyjtz4hADT1UOjrd7jjd1Qp3AAAzAAAzAAAzAAAzAAAzAAAzAAAwsNwNOuHYqExh4GHg47DAAAzAAAzAAAzAAAzAAAzAAAzAAAzAAA4oB5Z05cYiBpypnud1crscbBBiAARiAARiAARiAARiAARiAARiAARhwjwEnXDuVCQw8DDwcdhiAARiAARiAARiAARiAARiAARiAARiAAcWA8s6cOMTAU5WD4+2e402dUCcwAAMwAAMwAAMwAAMwAAMwAAMwAAPLzYATrp3KBAYeBh4OOwzAAAzAAAzAAAzAAAzAAAzAAAzAAAzAgGJAeWdOHGLgqcpZbjeX6/EGAQZgAAZgAAZgAAZgAAZgAAZgAAZgAAbcY8AJ105lAgMPAw+HHQZgAAZgAAZgAAZgAAZgAAZgAAZgAAZgQDGgvDMnDjHwVOXgeLvneFMn1AkMwAAMwAAMwAAMwAAMwAAMwAAMwMByM+CEa6cygYGHgYfDDgMwAAMwAAMwAAMwAAMwAAMwAAMwAAMwoBhQ3pkThxh4qnKW283lerxBgAEYgAEYgAEYgAEYgAEYgAEYgAEYgAH3GHDCtVOZwMDDwMNhhwEYgAEYgAEYgAEYgAEYgAEYgAEYgAEYUAwo78yJQww8VTk43u453tQJdQIDMAADMAADMAADMAADMAADMAADMLDcDDjh2qlMYOBh4OGwwwAMwAAMwAAMwAAMwAAMwAAMwAAMwAAMKAaUd+bEIQaeqpzldnO5Hm8QYAAGYAAGYAAGYAAGYAAGYAAGYAAGYMA9Bpxw7VQmMPAw8HDYYQAGYAAGYAAGYAAGYAAGYAAGYAAGYAAGFAPKO3PiEANPVQ6Ot3uON3VCncAADMAADMAADMAADMAADMAADMAADCw3A064dioTGHgYeDjsMAADMAADMAADMAADMAADMAADMAADMAADigHlnTlxiIGnKme53VyuxxsEGIABGIABGIABGIABGIABGIABGIABGHCPASdcO5UJDDwMPBx2GIABGIABGIABGIABGIABGIABGIABGIABxYDyzpw4xMBTlYPj7Z7jTZ1QJzAAAzAAAzAAAzAAAzAAAzAAAzAAA8vNgBOuncoEBh4GHg47DMAADMAADMAADMAADMAADMAADMAADMCAYkB5Z04cYuCpylluN5fr8QYBBmAABmAABmAABmAABmAABmAABmAABtxjwAnXTmUCAw8DD4cdBmAABmAABmAABmAABmAABmAABmAABmBAMaC8MycOMfBU5eB4u+d4UyfUCQzAAAzAAAzAAAzAAAzAAAzAAAzAwHIz4IRrpzKBgYeBh8MOAzAAAzAAAzAAAzAAAzAAAzAAAzAAAzCgGFDemROHGHiqcpbbzeV6vEGAARiAARiAARiAARiAARiAARhwkoHFr+StfX2yZuduaT3woXyxSD05WU94GnUzHZ1w7VQmMPCAvW6wN2fj9kgWZ6/Lid8dl5ffzMj63btzHZrp1NbsHpANbx+Xt8avy9ffV9i5fX9PJsY/lNfezsiGXpX2rj7ZsG9Y9o5fldv/XWHasA7rMAADMFA+A48fydfXLsjedw5Z7fLT/Rl5+b3z8tHd78pPM6EeHtw+Ly+bvmXkek3TDffJj+TBzGV5970h2dCfH5yZvuyp3oy8+M5pOXHtQeXXz/Vlp6VzX0Zad/l9WevAkLx27rJcn39UedoJ2oXLSH+JJjAAA8vIwOOvZG+f3+at2XVcxisdEzjS1n3xG1Wenbtl3eht2m9H6oZ7e3nubeWdOXGIgccNSCOckoHFmQvy2pv+ICdn2hnjLur/roy8dnE2vbaPv5MvfnfIGuhEppu7Vp+8fO62PHi8PI0WnQM6wwAMrFoGZj+V1wZi2nnV9q8/ckGuVxuV8OiBjJ8+JE976dbTwPtvP6Iivq/ZLU/9+rRMPCiD/8ffyfXxY7JBmXbR6Zt+7K4spux/Vy1/6JP+OQqt0MoFBv52RtZ5bXjh564vG/uFBQZeGX2gCwySh5q3hU64dioTGHhAXnPIm/VB++vxQyGzzkRgbAhG4qmO+7nRr0oPUL6/LW/9OmqA2Cfr38zIhoE+eUql6Q2G1o9cL502fMM3DMAADFTEwOJ/nZeXQkZUoV3WEdKF9vmpgdMyUYmJ9/g7uT15Rl7UEd0mzXoZeA8uy67+cJ/TOpCxIvG8vmbN7iH54JsUA5jH9+SDtyNecpno9DftSDwvbfqxFLpy/1Z0/zbrsyjlcvmeeSTj74fb1jVvfyoPnLmPzUyi2/LB74bluaEL8nWafOkptEPnq39ZleaafId2zyEGlHfmxCEGnkNw0Cm73ClnxTPwNgx9KB/NPJDFYATc9w/k+uRpec4a8A3I6zdKvHm7d0Fe9Ay6XRl5+XeX5fqDwDm5KVznQ5EgL/3nPRp47mEYgAEYqDUDi5flNd2W9x+Td6/ds9v9+dty4mjGesGy7sTV9C9WHn0nX1w8KS9FmIE5g6seBt7ju/LuHjXA3LVPXp+8Kw8eqf73+3t2JKDpn/b9WW4H+zxL80WZGBmwXnJtOHpBvrCmyuan7O4trKXkmXj0Y0p7S1M+57kYBhqKge8vy2ve87z1c0g+mHejLr2xTK793ZfSwKNd4hlrlTPghGunMoGBt8qBbKiOcYXr6uu/fCgnbqdY6+ibC3bURqk3bwUDb8PRT+V2qXUyFq/L6zpyognW1oBBNx7qqAfqAQY8Bh7ZZlT/yYTIuqBxNSB7p710on8uznwue9/J+FNlvYHeLhMBp8y1Ohh41uBt15CcSIisuz0+ZJmTnZ/F93+Ll49b331pPOHl0uMHMv6+MvtMP1ZJ5OIKPxNwv0TzjS7osloZeHBxyH+J8etj0vmm354/d66MZXXq2LZZfQAGHsZcHVlrpnZAeWdOHGLgAS6NVx0YuH1un9+J7yyxgO29z+WtctbLC6yv8drlQLReHcrTTI0wZWFwAQMwkMjA7Hl5zjPVdqaIon58W/Yq4+2p95Oj8KwBVOE6T+87I+P/MsahP+Cr+RTa76/KLhVVWDry7Tv56B2Vnz3n5XZk/zJrRfWlikL8/ivrZZQrg9tELiLLzr2EZjAAA4aB7+SDt/32ct3oV3Z7/mZc+7m8/Fj9DwYeY2D6tVQMOOHaqUxg4AFuKnDpnMvsYKf1IraH5MS9Ms9P5PKuvKve6rEbVC21JS3udRhY7Qx8Maqiw0pFUBfaaivyYuewfJSwW7geQD3Ve0j2funv9lpPA+/BZ8P+i6W+M/JF4pTYwn1gvTCKiS60+rt98u5sunvI0syRwe1qZ5/yp2MXndApxEDgxU8uEvvaSRWZPCB7/7byuun+Zw0GHmPgxPHmyvMaus9WKL/KO3PiEANvhUBwBUjyUafGSa9rt7PWBl5WJk74b/lqHqXBPUGHDgMwsGoZuC17+/z2NWnaqNV/zn8qLxej9nZLUmT01/85LM+9c1pOXPONOy+t+hl4djRd6hc/j7+y9Njw0d3QvWENCH/955govYi+9r8/l86iZvsSp/N6+vAzQsdVe6+iBfeDOwzc/ijjvyDxXkgE2s+n6rAsQrkMWO01Bl6oPytXT77vzj1Yz7pwwrVTmcDA48GHxqseDHzzZ39jilJTaCu4fv0GeaujIa5nI0/aMAQDDcyA1XYPyQf/SluWe3JC7Sae2iALtP+Vtu2Lf/swvxnGroy89lnE+nOPr8quolm2W16/kbZcgRdG73we2k2x0jxns7Zmqc3SgGbcb+nrEq3QCgbqwEDAqNNLAnxxWkV012Ld6kffyfUrF2TvO4dCu4abncRffu9DOaE3XLICCvyXU94mQuGfEUEHVhqBvz++Lq+rpRnKNimtKO+MvDuTXD8P7l6Wd98bsstudjp/+7i8G9yQib6CMX6NGFDemROHGHg1qlg6xOQGd7XpY01VqvkbrgojKWCdjgwGYAAGEhkwmzEUBzRmmmkZellGVoTRlaYftNJIG60RGECt2RkxjXXmvGwoGnjJU3yD+bQiNiI0qSjPBV11NHmlpmcwv/zO8xgMwMCyMmBNlQ0YXFbbu1sqflHx+Du5Pn5MNiizrNhXFdt236B7/VqBAct88/8ef24g/6atttII/902KU/KRJrlGQp9gHVu7DqrWcn+91fyVmAH88gy7B6Sd/9rMbGfX1Y2yniGIF/utltOuHYqExh43Fg0crVm4PFda0Hvijvr2HxVOMUrNj13G0w6M+oGBmBgORmwpkGVacJZRpc3harMdrciM8waXOUHaMXBW+H61RiT2WsnfVMzIqLcyvOJq2U9U1jnpjUsy9R0OfnhWrRXMLDaGHgk4+8rYyy0bqq9yc+a0N9T6PVoVk683afaYf96Jupuw5sZ2dDrf2aMrWIf8K/P5TXzd++//t6uPv9z7+9vHpOPgpHnVh8TNvCyVhSdunaptjoQufjyxZidzr+5IC/vtstnypgv+0B4R3ez+dQ1TDzaohT3VilGC39X3pkThxh4KSuOm6B2N0FTa/loVj444ofLP3Xognxda8asjjKiI6319UivrAFpU/MNC7DQ5AxUZShd0UbXSZmoQKuKrh8YBEVF4FnmYrlR4da04nCfY236UZZx+UA+OKQGZeXmqwJ9aZ95loMBGKgpA9ZanrslyoS6fW6fMt8iIqST2rLH9+SEbid37haza/lHMw9kMRjp9mhRvp65KidGDsleLwIvkHZFfUEpAy9rBxaknkZrRS4OyQfzEWzOfy6dOupwt9n4aVYWH6nvPn4kD6YvSKfaDX7NrmH5KCq9gB41ZYG0m/YZ2QnXTmUCA4+brWlvtmVrlL9flAezt+WD3w3Lc+oN0fojn8pt3cHUhLV79oCnkjd5NcmH6jhJj3sIBmCgKRi4Jyf2+YbSi3+KWEsuqZwlItXS9EkVGXjZrJRaA89Kt8woOXv6VHiNImvJiKjpu3GaBTb+YEdE+tU09wjfgROXGLB2046IUM7lNdDW6TXySpXFekGyc7e89NFdWYxrU1N8Xh8DLyuWSbkr3TRa3S899f7ViHIFxjz9J2UiYYf37OJ1eV2ZeOtOf8WzWQomSjHI37PKOnPjEAMPsGncymRAdzhR6y88/eawvPVleHfBWjSAt8eH7C3pp3mQq4WupAFHMAADuU0VtIE3XqaBd+O0ap/DkWppGLP6lxpOKa0q3X9dkJfUGkvFqVle3xmIQFl3ImogFuRrUSZG/Gj1XF9KBB7PYx5T/ISFhmDAfukTbUKZtu87+eBt/+XQGrOWaDB6Lqq8s+flOdX2rh+5HmFyBdvW5N/rZeBlA3kN9RPB8n1/VXapyLpdXz4KMb/45XG7T/0muWymj7XOqcWmIcF883uontI82zT6d9yw7fxcYOBxI67KG7GahsQaCKmONbcew77jsvfcp/LFN+GOqJprmnMfXDkp69X11o/epu64f2EABmCgZgzYg7EXyzXwSk4zKj34sPoXVwy87HV5XfU94YHZo5AZ99KfZhO4fCS3/6RfRhUGthh4CZqVZqfaZwzOR2MYKJOBgGkVZUJ5mtqRygOy92+lr2Vt8JDW9CvRH9bNwDO7iqsXYKWm0VpGW2TZAhv2pY2mCywpEe6vSuvu1Rk/0cpjwLfO3DjCwCvR0HkVx09uYo+BL36nFoONWDjWi8oza1SM/6s2Rt7tz47Z5t37l+VBmrd38M2ACAZgAAZSMmAPQDDwvH6/lIGXleziZXlNRVOYfrC4TpO3lERufabLsre4k+CAdL6j1oaqx5qxsJ+Sfa+u+ek96/ETFkoxYE1vjTShlIaBiLP4aL3COYHdxaPW1iuVv6i/18/Ay4o1nThxGq298UfkDuTfX5bXii+Owks3RJUt/9kjGX/Pj3YseykM+gz6jAgG3LDt/Fxg4EVUUnyjoBpizuMG1wyYBVQj1sFbsytT3U5Ijx/I+ImMCiHfLesx72BPs8cxPMBAjRio0sCbOS8bigOOJppCaw2k4ncYXPzbGXkuYOJ5L7SifprpYNfHD/mLu9cw4pDnOJ5XYQAG6spAINKr9Hprtmm1Jm69PK8/r0F/ElX+ehp42cBaf7HRb1afErOph7UkRXmbQtUrkj1KTz5bHe2Mb525cYSB5zWU/KzRAGh13MiJDfbiXXlXb/e+a0g+uFeBLg8uy64B/y3Smp198vK56havTcw39wD3AAzAwKpmoEoDz9rEorwBh9c212vglVDTUwAAIABJREFUUVW65UwNnv1UXrP6Ld2H+cfPjX6ViyK//VGmaOBt+Ogu99+qvv8qeE5CL+6ZFWLAmgK6c0D2plmT2tp1NXrHWq8vyNZgV/NiWkqjuhp4WXvaa9w02sXLx4vtftzmRfaUY7/viHoZlPgZL4ZoIxT/UfdEms/csO38XGDg1aBS01Q831llD2aPb8tetRNSyVD5AIcPrpy2Ixl2H5K3bnxHIxzQiftqld1X1D9tQF0ZsA28yGk9SdfXBt6u0/JF0ndj/laV0RaTpmknrXTf+VweJHw31K5aBt6QfPCvEu3O40fy9bULsvedQ7Kh1x94Pd2fkZffOy/jao1Yna/Oz+jjQtqXU098l/YRBpaJgWA0nd/OJZpJxQjtwvcT1v2syGhLUf8VpWv1AcnR5ZaxGTmN1jb54qYGW/kM6lbO7xh4y3RPlHguSMGmy/2fb525cYSB1+BAuQz7as+btRZEqVD5IoeLcv0je3Hvp98+L9eTtk0vntvcjedq54nywzcM1J8BbSitKfPB3xpwJAzMkuqxmusnpVtV3rQxubOyyMLovN2TE7/2Br7lrHFUfw6i88t10QUGYCArwamiZZl2lvkUM300m5Wq2uyEcUFF6ZZh4GUDa/2FptHqXcsTdom18rmrTzaYNccr+f+7rzCwEnjgfk7Xprth2/m5wMADahq2ejHwzZ/lxWJHnWJwYta7e3/ADytnyixs1otN0oUtGIhk4Paf1KYKZZpwX/zGM6N2y1O/qWzQUC8Dr5rpWF//55DfL9VyowlrIHdSJtiYKZJJBljpBljohE7LxYD9gt5v9ysx8uIivS0Dy2yQUaM+20o3bR9XjoEXiPgOTqPV2iXNTrL6nV//WW7XqPzLxQjXaa72yLfO3DjCwKNB4IGxXgxYHd5uef1GUmO2KBMjyrzbtU/2MmUWNuvFJunCFgxEM2CtO3Rcxr9Parf13+zpt5VOB62bgVfuCyXFh85T3GCzksGKXgcpaSBXSdqco9nkGB5goHYMzMq7e5Rp9/an5S1JkA2cH7d7rRX5XE5flFzXy2HgZf92RtZ5AQzWNNrv5IO3Pe0GZO/fEvJqlb+Wkd8J11T9Xu144XrNoKUbtp2fCww8btboQQy6VK+LNWBKXjPi9riaNrtrSE58Q4PfDA0+ZYBjGGgwBnRU2M74HVdD9Wqdl9zeh85V/a02y8qdwpuUbjZ7W/b2eQOn3fLi+L2UfZx9Xmg6lMp78vWDHOiB3G7Z9eWjlPkJpsPv5emOXugFA1UxYO0Ou1sqeVmjo9BM1F5k+xcMArhWm3pbFgMvsENvsd/Qu9SWiiq0xlApZjFV3BfVRteqmCLvzvf/vnXmxhEGHjeN8zdNozaKVgdtvYEKdBb3LshLu7yB1YC8fm2ROuG+hAEYgIEVYcA2ltJGhlntfdppSRHlq5+Bl5UvTqso7z3n001Jio2kCPRjEWVJ6rsXr5z0IzT6z8gXTJ/lfi+ToSS++Ft19yf6xetntaM7h+WjStaotl747JbofsaO1Huq3M2HYu6nZTHwsln5YtTvb7xptPraz52bLdHm2S+P1p2ubFkKWI5nGW3Sa+OGbefnAgMvpoED6vRQo1WEVt9/Ja+rXWiTOh79MBDdiUekD7clOn40476EARiojIEHnw3LU970n52HSkdEB3YdryQiw6urehp42ekzsr5YrgHZdaXUyyJ7t8D1o7dr0+7+91Wrf3ztcql8VFaPnqb8RD8YgIGaMBCILFvz3mVZrOh5PLiL7ZB8MB+uI+vF0M40bXY4jWC5tYm2Ju1u6VY0YMoI89nz8pzX3+SCGPQyE+nSuH1OrUnL7KTa9L8V8VqaqyBnzfa7b525cYSBB8g0CCkY+OKjY3Li9nfptHo0Kyfe7vMX/TYDwHtxjZ/9hum1y0wjarZGn/LEsc/nsOEoAwFD7qlD5+X6YlxeA+uXmsi2KqLJKjXwFv/2obzUu1vW7MrIa5/FTY+1Dbk1/cdl/EFcubJiL+1wXMZjNYhPI8T4vy7LLvVyq1ZRJaHrpOjXOaeMekPPdM9/6NTUOi1+eVy93ImZ+pqWgWsnrbRevhgxxgj0RWtSrY/9SL6ePC8fxY07rHVe43fBtdrHSgw8a62/Adl77bK85hl6adcNnP9cOoszlHbLmv6TMpEm4vHxd3J9/Ji8lbj2OO2fVcdpuV2l33PDtvNzgYG3SkHkpi2v4fYGVa0HTsqJK3fl68UIo21xVibGT8qLu73psPmfLyWtNfSvC/KS16GVs94S3Db1QyL3Z3n3J3qhV60ZWLx2UkWr7ZanBo7JiWl7gLX4zXV594g/TWjNzgHZO11dXXh9TW43w5Hr6dq5YFTIzoRB2Td6yYbdsmb3Idn75T1Z1Kbj/F356Df7rMFlyajCexfktaHTMf3jI1n811356PQheVr1d7nBWC1MQfrDdJygEzrBQBUMPJLx99Tz/a4qN5YItttxyxpMfygbtIm1s09eHPlUvngQHIc8kgczl2XvPhNAkBA4EJy+e+jPcru4WVOM+VeRgZcVHe334jvDxWUTyglWMLvR+hHx5iXVPtk1fl2+LubZ73MXF/LjsNzLLMZUVbDua1rrZ6tGTc+3ztw4wsCjM+MGT8GANahSA5Cn+zOyYaDP7lyKf++Tl8dLrPFgdYrqwaCYRsrPqlhzqVEbU/JNBwsDMFBPBqwINNUmtw5kpNUaUJl2ekA6YyPf0teT1dekNfAi+pHiouER/VvQnMyZhTt3i+nP1gdeQJm/PTf6VelpYhF5SOofnxo4nS6SIiL/9axz0k7PKlqh1apiQG/AsHO3eOu6VaOBXiduzc74jRoWr50OmHiFscGuPtnwZkY2vDlgvxxJMvCyi7YRWejbTL+2wURxR51rte8J5mCwvQ5olutryjY+F+WLUfuFktdnrdk9EFP+vD5J/WA19ca5q6/tc8O283OBgRdsbPgdQy+Cgdt/OhQxYCt0oGpg53UqT+87KR/8V4p1fQK7WXnnl/0TAw9uI7jlIWP1PWRQ57Ws80dy++Kx6IGTbvd3H5K3btjReZXWQ0UGXjCSIykCr9BOPLjxYShaPNTvmOm4F0u8hPLaHWuAF983rjHRI6evygMd8eelwU/6MRiAAUcZ0NFkuWjrv9Wgr9HrxO3cLUnrZWf/dbUQXZfUvub/9vSbp2U8aarp4nVrHVK77Y8w6Kz2PeLvsXVmbwplrpNYxth0TGTgydJ9sdcv7+qTl05fla8f1aCOYvNE2pU+5zTieb515sYRBh43Jg8LaRl49J1cv3JB3npvKPy2K/cWaEheO/epfPFNMKw9oZG/dlKtlVe6U7Y7WfV9DDw4Tssx34MVGCiPge/vyfi54/KyjnIwkQ9vH5d3J+/KgxoOEioy8LJZSbcGXqAvevSdfHHxtHTu0xGFfbJ+37DsjZmiFP/gbaZvXZUTvzM66fR2y1O9mZxWb5WdZiC/cFset+iFXjBQIwbsHWHX9NVq52y9sYOZHnpSJhJfbuSnyb57YlheDMz+aR04JJ0nzstHMylfJj16IOO/G5YX+/01u5/uPySdv4t4wVKxgZeVxcvH1ThnQPZWY3yacdjkeXntbS9a0BsH9cn6NzPy8nsf5pZxqGWfHN/n0T+tJm3csO38XGDg0bnVqHOjIVtNDRllhXcYgAEYgAEYgAEYgAEYgAEYgIFmZsC3ztw4wsDDwMPAgwEYgAEYgAEYgAEYgAEYgAEYgAEYgAEYUAy4Ydv5ucDAU5XTzM4xZePNCAzAAAzAAAzAAAzAAAzAAAzAAAzAAAykY8C3ztw4wsDDwMNhhwEYgAEYgAEYgAEYgAEYgAEYgAEYgAEYUAy4Ydv5ucDAU5WDC53OhUYndIIBGIABGIABGIABGIABGIABGIABGGhmBnzrzI0jDDwMPBx2GIABGIABGIABGIABGIABGIABGIABGIABxYAbtp2fCww8VTnN7BxTNt6MwAAMwAAMwAAMwAAMwAAMwAAMwAAMwEA6BnzrzI0jDDwMPBx2GIABGIABGIABGIABGIABGIABGIABGIABxYAbtp2fCww8VTm40OlcaHRCJxiAARiAARiAARiAARiAARiAARiAgWZmwLfO3DjCwMPAw2GHARiAARiAARiAARiAARiAARiAARiAARhQDLhh2/m5wMBTldPMzjFl480IDMAADMAADMAADMAADMAADMAADMAADKRjwLfO3DjCwMPAw2GHARiAARiAARiAARiAARiAARiAARiAARhQDLhh2/m5wMBTlYMLnc6FRid0ggEYgAEYgAEYgAEYgAEYgAEYgAEYaGYGfOvMjSMMPAw8HHYYgAEYgAEYgAEYgAEYgAEYgAEYgAEYgAHFgBu2nZ8LDDxVOc3sHFM23ozAAAzAAAzAAAzAAAzAAAzAAAzAAAzAQDoGfOvMjSMMPAw8HHYYgAEYgAEYgAEYgAEYgAEYgAEYgAEYgAHFgBu2nZ8LDDxVObjQ6VxodEInGIABGIABGIABGIABGIABGIABGICBZmbAt87cOMLAw8DDYYcBGIABGIABGIABGIABGIABGIABGIABGFAMuGHb+bnAwFOV08zOMWXjzQgMwAAMwAAMwAAMwAAMwAAMwAAMwAAMpGPAt87cOMLAw8DDYYcBGIABGIABGIABGIABGIABGIABGIABGFAMuGHb+bnAwFOVgwudzoVGJ3SCARiAARiAARiAARiAARiAARiAARhoZgZ868yNIww8DDwcdhiAARiAARiAgWoZuHZcure2S0ffeZmtNi3ObxweZ89KT3u7tA9dapw8wxd1BQMwAAMwAAOpGHDDtvNzgYEHuKnAdcZVvzwo7eZBOfi/o0t6Bo7LJ3cWGqs88Ed9wQAM1JuBvx6VLtNmbt0vE0u8Ja5bf4aBtzrvZQy81Vnv9W63SX/Vc/Vw7pKcHeqV7lc7iuOejle7pXfovNxaoC+vW1/Ovbfq770gW7515sYRBh43aWPdpJ6B90q3dG9X/4udW5cMXsbECzY8/M6DDgysXgau/ntX8eG/9w/zjdXmO9dH35Hxfb3S3X+24aPs7vzHfunt7pezs6v33qhJu1grA2/+qhwf6JGufyeSryb14lzbwX1GvaZnYPbDXukoBCt0dPvjna6OfBDD4GU/rfkvj0t/T5ccVZ9VrDXtEM9ItJ0hBtyw7fxcYOABaQjSihv95dCyYOD1/H42lO/5L4/KDqJMQro4XZ/LwQzXgInVzMDShOzf2i7tfb3Sa9rHHafkzmrWo+qyX5JBo2NP4xt4l4bMQLAHA69aJmpl4NUqnWrLw/n0mTCwsgxcK0TN956SG/O+UZd/nn8o9z87KqN/9T+f/X1P7iWdNvUqfvanHVrZuufec1J/3zpz4wgDjxvFyRsltuNJMPDMOfkBSZccveZ3bLFpUfeNVffUF/UFA2UzMP+n/tyDff+fZmXioDFsdsipO7SPlfcLGHiVa9ek3NVqwFurdGgny24nYbpJ780GvRduneyW9vb0YxkMPPilDasvA27Ydn4uMPAatHFftTdqCQMvthN7PC83/jAovd5U260dsmPguEzMPQw/6M3fkPNmzYlXCmvtbe2Q7r5BmVDTjKzIhfsTcnxgh3SYKJf2dul4tVcG/3BLFmLYKq5p4aXf3iFdPf1y/LP78jB4jn6gN2X4/X7Z4Z33yg7Z//sbMv843Gg9nDN56hEv1L49t0bgqNwIrn9VY11WLZfBeuP38H2FJiugybyc71Nr3xXaz65/v2rn5cH5EtF5D+WTfRHmX9r2w2rH7svEv/fkpwb1npX7BS4i28W+QTk/HbMkwsP7MvF+v98ednQV1gWalbM9Jq+DcinEnIlcOC79O7z1hDqky1zjZpppxQXjrjClyVqH1du8QJdTXVv3F/M3z8p+7/omz/8+IfcLbfjC9HkZ7OsqTJvqkG7TR90Pt++5drbQTxXb+Fd2SP/7E3L/Ycz3Q/kp9G9WeYKaPZT7l8/KYF93sX/L9yUxfae6RmRfMH9HPnm/X3q8fjjXX/ZI/7k7No+P5+XOhfyUMG8KWfsr+XWfwtEo3ou7fCRh1frqZ4zHcYwFNI6p97wGKZjzzrfqIl8/wdkG+b49+LwR/RwQWQel6oi/2yyiB3qsAANXj5j7v7SB5415rP4o147YkdWp+tcU7ZB3vehIv7i+t9AG9nh9mxkn9Uj/BzfCY54V0Jp2MtCfUQeRbZ5vnblxhIEHqJGgOtug6YfrUN0VBqvt/TKuQ84Xbsmp3vzDcMev9svwkWEZ3teTH5Bs7ZWz/1SN1z/PSq8x4rZ2SM++4dx3BwvmnO6wvAHZ6F/y3y+m+1a/dBfWp+gauhQy8e5f3F9YTN4YiIP5vCSd43WoQ2flfF+HtHf3y6DJvz7nvRtWHS5cHsxfo6Nb+t/Kl2F/blBod+jZOujiLDchVlSd8zeLH+qwidi4cyq/rMDBifyD8uOrcrTLtG+Dcsky/udlfMC0kTHReUufyH4zKOg7L/Pe/VJO+1Fsxz6Rq0f89fj8aagFc0y1u7k2LvdSZIecuhWok4VLMtxdaNO3F9rEI/ulx7zc6N4j/f/X/C1gRj2+L58czF+7wzun2I6mWTv1joybtvdIf17Trl7Zn/t9WIb/o2BAFctpr2Hm9RfHfz8oXcV2uZDf9nbpOnJV7ufabWPa5fuF/T0Fk/HVYbkaePGyMH0q1E/t/1Xh+71nZdaq24B22azc+Y98v9C/w+jUJb0H878PHxn3p1crvdqNOVjoSwYHugsGYxrN/Gsv/PWo7PBecnn98JFhGQyu+aa5etXXuFi+YJ9djLzvkZro6z1j/HZCzprnhld6CvU8KP3bvToZlEt6AfmYes8qDROZm78qo4alg735vntHf/7Z4MiwjH7pm8vF54ciQ36eop43aMt9/tACLRqJgYef7c+vW9s7KncSXsrMfzmaayv29+b7th0DXls+KleL46CU/WuKdqh8A29BLg0F+t0j+/PBFE2wDEUjMUVeq2sD3bDt/Fxg4HmDEX42xiDee7gOrYH3UO78tjfX4ZnBkB/J9rDQeXTJ/ov37TJ6Zl2xE3noTzELDhjnb8itUASeGQh3yeBf/AfsXAO5cEkGX80PjKypvP8clR4zCH51v3wSjKp4eEdGCyZj/59Uet7AYGu7dA9N2NF2Xv7b98snxQHeHTllBmVb98sneoCRzcrDuRtyq9ih10cXOojqOgj0Q79aMnDjPfPg3CVH1Vo5+c/aZf9ndvTxw4v5AcOO3wSiobJZ8afhem1TOe1HVrJeO2bW4TPt3z/ta2ezV2XURKIFByreOkBD2hBbKEQDtkvvb++otj4r2cezMp7JDxaCBp438Og9GYiO9trrrkG5VGxHkzhMmELrldPKrxchZtrlfjmv237v2u1d0vVqu/T/QfdRCzLxVr4sVp+wdEkGjQkb0Y/M/j7fBwajtuKY8ozFqE0sZn+bX1Op6+AnxQhBL52Hd0YLBmK/jD9I0qrwtwfj0p97MdYrp+IiKnPPYH6ZQ/VkOPxL4eXUq0flhjIp8+Wokb6FZ4z2re3SFexzs/5gtEu/OIup97KZi0knp7v3t95Tgd0nvTwZQzV4X6WoG5597edC9EAPJxjw7ut2ae/okaOfzcpD1eZ5bbH302trdKCB97fy+lfVXwf6MZNe8nUiIvC8l4j7PgkENDyU+zdv+S8EndCc9tJnBi2CWvjWmRtHGHg0Go3VWXsP18FdaHPTSjukR01Hyt183kNvqPPIN0753Rm9yDSv89GGWHQj5g0YQlPRPJ7+ml+A1n/I98xBezBtNRALn8geY/DpRea9/EcYcuZcLx/DX3r5LAwut5+SW15eon566dZYF6s8Udfls8a636ivxq0vL9qu66hc1Q/+3gO1jqYz9extdqHbn1z9F6Lztu6XCc/gKqv9UAOCgJlYur2IMMu8awfz77HqtaM6Au/xJRk0BtKOU3JLa1E4Z/4PeeMrevDjta3ez4g8edf28hYY+HjtdJQ5ml/rKBDd6KXnGZhH/CnP3gBq/8WIqcVenRdfSnl5jv6Zz5fX/6nveCwE2fHylc3KwoU9uRdmUWUK1qu3C3JknlWa2Tg21Xdu/b9mbSjbgK6lvlnvGSOu7J7GXUflhpevqHqvhLmodArXyJcxIhrV/N2bAh/gLlgP/K4Y9+qOn43bxzV73eWWzdkj3YXIZRMNfDRqqZ2Sxloc9zF9WUI75PU/0X2lN4ZS0e+F9rT75C04a3Zem7x8bth2fi4w8JocuKZ7YPMernNrPOSnUOXWfnhlj5yfDb99vv9hYWD2l+gOLNgZ3Xo/H/Fgog5mvcFqBCPegME3zgLpP54I7FR4Iz91LdFY86ax9crZuUJ6Xkeq1orSdRrMfzY7L+OZvC69J2/IfDCipVCWeumi88ZxgIkIjtAIjerFgDcFJ/zgXIjSDe0+6r1kCEyjjTAHym0/ihF4ie1fVh7+84ZM/OGUDJspNtu7/TU8lRkXjgYMMhQxiPhyOG82/Ta8e3lO/0K/ki5yLWbQY+5vr70OGCn5/iJmPaO/DMbnLZTe/fy0zvZBmYgwIrPZiLIntDv5fEUYeAXjMMyO0np+XPpNPxzTN/lcF/o+bXjF5Mmr2/0Xw315Mb0IU7N2+malaOAp07R47UK+J94yfazSLVRPWclWwlxUOrlrXpVhM4j/v6MyG6ldApOR31f1yN8xFmDAbQbM2qFm3VjPyOseDM3iCY8Fwvd4mv4119bFtkMVRODNj8sek++tvXLq2rwdMQ93bnNH/Vj141tnbhxh4AGoBWjwQdW534MDrccPZfaz4fzaOh37ZSIwbdSbLhZe4FWZf+3t0vthYerS41k5a9aayxmE+UXEP/mHN23M7xBjBz5FngIP1N5gZ2A8MWQ8lG5CR2rqJrLTVutD5dbyGzorlwKbddRLF+d4KdaHX3fkES2an4GYTScK94MXcRaKniq0r/rz/HftyOGy2w+vHdv3SfQDvFn3rNjutktHd7f0mLXg3t+fX3ZAGXhexFrsy5MIE8szhkr1A+0Jpo3PTKBt122MV85IA08ZPvqcYJ+m/xZKr2CGBV9ghX5XL4F0eoHjUH9T+LunlzV1N3BuNpugg/6ut35iSaMvK3muYoxOL82QJl4kei309Q28JDM3pFtEnjwNy2IuIp0cd97zQ6ie7eeY9q3DctXTiZ+N9WxLfVFfpRjwjDzTDgTWAo0cC3jpldG/5tqbuHYobszhXSei7zXpLVwetqIIB39/KbxcRjENnk/9Zw20cEkLN2w7PxcYeDQajdVpxgx2Fry1cQIGWX4nJ39DitwGFke8RV79n3qh6Gz2oczf+kSOFncDbJfuX49bC4OHHuBDHAUGN3Nn87s8BvIXbJxC6SZ0pObc2E7bGJuXT/m7HbZ3SM/7V4trUNRLl2B5+J0OGAZWgAEvaq7UgD84TdCbHlicRlvYGCgQOVd2+5HYjnmRf12yx+yqbUUNF9pRZeDl28ju8MYWxTY4HIXmRQx29/ttfmRf4G1GUUwrqu4Cbbv+bkw5Q+26PiemT8vdN6H0CpFYHd7GCnHl0QuYR5Uh/1lcvjy9amLgpez7THlT7bwY0sRNA8/TsCzmIsqW48DTsNvf3CKSX70JiWaM48Z6zqW+qK8EBuY/K6wFqsYTsWOBbHn9a3S/4/ch8dcx3wn3vbn0TFmWZuXSb/arXeN75PhfI5aBSCh3MS2+w/2xAgz41pkbRxh4KwABjZDfGZStRexgx1/YXK+zU+xsYqbQlrz+/I1iZIiOSik5iPQeuIsRJ4WBV2AgbF/fi5pRu+jGPdAXuC2W73K8pg/nPpHBwo6N3sL1xfNqrItdnvg88T20gYH6MXDnNztyUcQdr3ZL9/aY/7l1Q+3IOlMn+fXKCtNoC+2PbvvMd8puP5LaMS9CSw1GfDbCBl7pa9+SU9tNZFJ4HZ4dcVNoy3oOWEkDzx8gRU+hLY+pOAPPm/6ZOIU2sd50Pgp6JfZ9+e97plfiFNpbp6Tb7NyrNpGILYep19hnhpgpz4Xvx5fdm8as1sqN4ruQTlnMRaWTY7OgYewUWq03x377gRZo0YwMzMpoYKf1Yr8YHAskttPh/jXHS2w7pPr+4HVy7VRE3xvqWx/K/QuD+Yg8va5u6HvNWG+UqZHbIzdsOz8XGHg0Go3l5Kd5GH91WK5669cV1suJ3WwiTf170SxqClB+wNAuwYGt1zh5U9R6/+BNv/Xfglk70+rrL3wi+81aEXpx9oSO1FwrttPW6ZrjwqYaxSliddLFKz8/6ahhYKUY8Na4Uy8Cgu2B2ZX6s/yus+0HJ+xprQWDxLRt+falV84Hdxott/1IaseS/ua1WxFmXCjfXhlvnZIduchDZeB5UxB12+p9v+yfK2ngqWmmamfhSu+1WOPL28QisNurvs5CYddiv4+L4917MVViaqypB28Ti0gzN5/+rZPGnLaN59hymDTTPDPoKc+F71ubSWlGvOcBzVIUw5UwF5VO7tr++rihe1HnjePGep6lvqivihgoGGVqynzsWCC2TVHjAt2/mvwknOMtDRD5YsJrG4PpRZTR29gofimMuP6Ez3U/zPHy8eBbZ24cYeBFNCzcEMt3Q5StddLDeDYr+Yd7/Xbei1joksG/eGaaX76HcxMy/IG3w9+83Lp53x7MGj68AaF6yPcMPLMOxegde8Hth3dGpdcYcV2DcskzEk063uLbrw7KxLyfh5wGD+/I2QGzgUaXDH+p0kvoSM154U57Vm7dDJfTG2z56/rUR5ey65P7jwdYGKgtA57pVYz+DbQ1nt7edNn2oEFXMAB7hmW4t13aI82UctqP5AFB1osQ6Bq01zBduCXHe7w1vpQZ9/iGHH3VfB7Rppu1fkyegwZe9qFcGspvUNT72zvhNn7+hpx9fzxmg4CgfoVo6uD0Y6NrTHtdU4PJu0ZUP5J9KPc/G5anJVyXAAAgAElEQVTR4q7kwbzbvxenrEaYgd46h11DEzIf2DDj4Z2z0t/VLu36ZZnHVdTPYt+3P7T4enb+qhxVffD4QL7+oupp/sujeXO296xVVzXV1zPwzNq4QVYe3pHRHF9dMni5VD9dAXNe5L42Bwt6PrxcmDbXOyp3rGnmWcnmdqs8LuOzdv3SH6MHDDQmA1dP7pFTl+/IQvBezz6UOx/2S5fp49TLNy96OfRCpdz+1bQ3Ce1Q8SXLq4NySa85btYPj+p7Z2/JjeB4J+vNmIpZtzSqD+Gz2j4nomfZerph2/m5wMAD4rIhXtEHghIGXnbpkgyagUX7Dn+NpH+eld6O/KCgY0e/DBbWwNv/q8JmFcWFywuD0ld2SP9bhbWF3uqXbnNuYMFYb8AweMTscms2uxgUsybN4EC3dJiOdWu3DF8Or+8w+2Fv4e8dsqNwzrB3jfYO6f0wsEuiN1hT5qHWP2zg5aNDOl7tlf2Fcg7vK+xeFexw66CLzhvHjfngSL01dr15Lxe86fJJ9emZNMGHfm8KbvvWdolNJ3X7EW9s5fP2UK4eyZtr7a8U1nYrtIldQ4OyJ2TGZeXhteP5jYva26XjV/tzba/XjnYNnS0Yf8r0M/282txHt4/FNltFWCdpZtZInThY6E96zLUHpf9k4SVQTHvt9RdnowyWpD4tJr3IfuTIfunJTYtul7RRDcUozMKaeoP9p/xNEPSGTqpPLOrV0Stn/5n+XinmWfWXxb5J928x9VTsr7uH7YFjtk5r4A0N517EdWwvPDMU++l26T5yqbiebI6VmHoqnzkverZduvvNM8V+GTznPRMsyKUj3XlzuqNLevcFnlHa021cksx2+vokHbSCgfox4PXjZnzRpZfBKLTx7cF20Ite3tqdH7/sG5Tzuf6m/P41my3RDhVehrUX12LN9z2RfW+uf+uQrr5CP31kWLy2vGso0I4yHm+s8fgqqy/fOnPjCANvlQHY8B1u0mCnUJdetFm7fks/f0POD/VKt9f5mU6xp1+G/6AXTX8ot/40KL3bu/Immxk4vtItvUPnQ2+Q9IDs/l+OS//2ghm41Rhzx2UisOur1n3hzidyfKBHugqmYntHl/QMHJeo3W7jIjq89MIG3rxcer9ferq9nXTbpePVHul/fyJ616ca6+Lli5/1e7BDW7SNZcCb+ph2bRnvob+4aUVBW8+Q2DoolwLRV9a1U7UfpQy8fATR1d/0+wtcv7JD9psNLR7HrNFjpgDPTcjxgR3SYaKdjZH3aq8M/uGWLCQtpP3wvkyY9vFV1T5298r+30zIrI6WLvVcsHBLzu4rvKwxhtRvbuUfvD3dtCFVa4OpkLf5m+dlsK+7WP58PzIs56/NhyMMY8uzILd+vz//kspoGFqnbkHuXDgu/T1+n5jvTz6RO6GoitL35fw/TN/n11lc/5p9fF+u/n5QenU/1t0rcbsX6v7Y4tOUO+mZIaq+9PfvG8a8em4X8wLw+GcRUfpR6Xial8vc/Qk56r1c3Noh+y/oiHoTYWnXR07Dfadk4p8qItC7Nj8ZEMNAQzJgxgmn3uqXHj0e2doh3aFxi9/u3v/L0eJLnPZX9ssnXhv9eF7K7V+zSe3Q43w/usMbT5kXPGaMYV765KLm1cuzB5dy453g2CuyHYXVhmQ11Oc2aT26Ydv5ucDAa1LQVssNtVLlTBwwwBSdEAzAAAysEAM35KiJwg6ZUf5AZ6X6Da7bAHWgDTzu4RW6hxuAE9iADRiAARhYNQz41pkbRxh43Hyr5uar5eAJA48H7FryRFrwBAM1YsBbAzAQBYe+NdK32Z+ZMPB4Jmx2xikfjMMADMBAWQy4Ydv5ucDAA+CyAGYQlB8EYeAxGORegAEYWAkGZuX80NHIZQoezn0i+3MbXKg1UOnj6ePLYQADD17K4YXvwgsMwAAMND0DvnXmxhEGHjdd09909RhkY+CtxMCda9aDZdKEq8ZiwFtnJ78mWXFToj5vnbYu2fMnb+F/6rax6taB+sLA45mQcQEMwAAMwAAMKAbcsO38XGDgqcrhQdeBh+cGqQ8MPFihvYABGFgZBh7OXZKzZlMivRnFq2bDobNyKWEDIeprZeqroXTHwGPQ1iDPoQ11X6Ep9xUMwEADM+BbZ24cYeA1MEx03gxGYAAGYAAGYAAGYAAGYAAGYAAGYAAGYKD2DLhh2/m5wMDDwOONAAzAAAzAAAzAAAzAAAzAAAzAAAzAAAzAgGLAt87cOMLAU5WDY117xxpN0RQGYAAGYAAGYAAGYAAGYAAGYAAGYKDRGHDDtvNzgYGHgYfDDgMwAAMwAAMwAAMwAAMwAAMwAAMwAAMwoBjwrTM3jjDwVOU0mhtMfnmDAQMwAAMwAAMwAAMwAAMwAAMwAAMwAAO1Z8AN287PBQYeBh4OOwzAAAzAAAzAAAzAAAzAAAzAAAzAAAzAgGLAt87cOMLAU5WDY117xxpN0RQGYAAGYAAGYAAGYAAGYAAGYAAGYKDRGHDDtvNzgYGHgYfDDgMwAAMwAAMwAAMwAAMwAAMwAAMwAAMwoBjwrTM3jjDwVOU0mhtMfnmDAQMwAAMwAAMwAAMwAAMwAAMwAAMwAAO1Z8AN287PBQYeBh4OOwzAAAzAAAzAAAzAAAzAAAzAAAzAAAzAgGLAt87cOMLAU5WDY117xxpN0RQGYAAGYAAGYAAGYAAGYAAGYAAGYKDRGHDDtvNzgYGHgYfDDgMwAAMwAAMwAAMwAAMwAAMwAAMwAAMwoBjwrTM3jjDwVOU0mhtMfnmDAQMwAAMwAAMwAAMwAAMwAAMwAAMwAAO1Z8AN287PBQYeBh4OOwzAAAzAAAzAAAzAAAzAAAzAAAzAAAzAgGLAt87cOMLAU5WDY117xxpN0RQGYAAGYAAGYAAGYAAGYAAGYAAGYKDRGHDDtvNzgYGHgYfDDgMwAAMwAAMwAAMwAAMwAAMwAAMwAAMwoBjwrTM3jjDwVOU0mhtMfnmDAQMwAAMwAAMwAAMwAAMwAAMwAAMwAAO1Z8AN287PBQYeBh4OOwzAAAzAAAzAAAzAAAzAAAzAAAzAAAzAgGLAt87cOMLAU5WDY117xxpN0RQGYAAGYAAGYAAGYAAGYAAGYAAGYKDRGHDDtvNzgYGHgYfDDgMwAAMwAAMwAAMwAAMwAAMwAAMwAAMwoBjwrTM3jjDwVOU0mhtMfnmDAQMwAAMwAAMwAAMwAAMwAAMwAAMwAAO1Z8AN287PBQYeBh4OOwzAAAzAAAzAAAzAAAzAAAzAAAzAAAzAgGLAt87cOMLAU5WDY117xxpN0RQGYAAGYAAGYAAGYAAGYAAGYAAGYKDRGHDDtvNzgYGHgYfDDgMwAAMwAAMwAAMwAAMwAAMwAAMwAAMwoBjwrTM3jjDwVOU0mhtMfnmDAQMwAAMwAAMwAAMwAAMwAAMwAAMwAAO1Z8AN287PBQYeBh4OOwzAAAzAAAzAAAzAAAzAAAzAAAzAAAzAgGLAt87cOMLAU5WDY117xxpN0RQGYAAGYAAGYAAGYAAGYAAGYAAGYKDRGHDDtvNzgYGHgYfDDgMwAAMwAAMwAAMwAAMwAAMwAAMwAAMwoBjwrTM3jjDwVOU0mhtMfnmDAQMwAAMwAAMwAAMwAAMwAAMwAAMwAAO1Z8AN287PBQYeBh4OOwzAAAzAAAzAAAzAAAzAAAzAAAzAAAzAgGLAt87cOMLAU5WDY117xxpN0RQGYAAGYAAGYAAGYAAGYAAGYAAGYKDRGHDDtvNzgYGHgYfDDgMwAAMwAAMwAAMwAAMwAAMwAAMwAAMwoBjwrTM3jjDwVOU0mhtMfnmDAQMwAAMwAAMwAAMwAAMwAAMwAAMwAAO1Z8AN287PBQYeBh4OOwzAAAzAAAzAAAzAAAzAAAzAAAzAAAzAgGLAt87cOMLAU5WDY117xxpN0RQGYAAGYAAGYAAGYAAGYAAGYAAGYKDRGHDDtvNzgYGHgYfDDgMwAAMwAAMwAAMwAAMwAAMwAAMwAAMwoBjwrTM3jjDwVOU0mhtMfnmDAQMwAAMwAAMwAAMwAAMwAAMwAAMwAAO1Z8AN287PBQYeBh4OOwzAAAzAAAzAAAzAAAzAAAzAAAzAAAzAgGLAt87cOMLAU5WDY117xxpN0RQGYAAGYAAGYAAGYAAGYAAGYAAGYKDRGHDDtvNzgYGHgYfDDgMwAAMwAAMwAAMwAAMwAAMwAAMwAAMwoBjwrTM3jjDwVOU0mhtMfnmDAQMwAAMwAAMwAAMwAAMwAAMwAAMwAAO1Z8AN287PBQYeBh4OOwzAAAzAAAzAAAzAAAzAAAzAAAzAAAzAgGLAt87cOMLAU5WDY117xxpN0RQGYAAGYAAGYAAGYAAGYAAGYAAGYKDRGHDDtvNzgYGHgYfDDgMwAAMwAAMwAAMwAAMwAAMwAAMwAAMwoBjwrTM3jjDwVOU0mhtMfnmDAQMwAAMwAAMwAAMwAAMwAAMwAAMwAAO1Z8AN287PBQYeBh4OOwzAAAzAAAzAAAzAAAzAAAzAAAzAAAzAgGLAt87cOMLAU5WDY117xxpN0RQGYAAGYAAGYAAGYAAGYAAGYAAGYKDRGHDDtvNzgYGHgYfDDgMwAAMwAAMwAAMwAAMwAAMwAAMwAAMwoBjwrTM3jjDwVOU0mhtMfnmDAQMwAAMwAAMwAAMwAAMwAAMwAAMwAAO1Z8AN287PBQYeBh4OOwzAAAzAAAzAAAzAAAzAAAzAAAzAAAzAgGLAt87cOMLAU5WDY117xxpN0RQGYAAGYAAGYAAGYAAGYAAGYAAGYKDRGHDDtvNzgYGHgYfDDgMwAAMwAAMwAAMwAAMwAAMwAAMwAAMwoBjwrTM3jjDwVOU0mhtMfnmDAQMwAAMwAAMwAAMwAAMwAAMwAAMwAAO1Z8AN287PBQYeBh4OOwzAAAzAAAzAAAzAAAzAAAzAAAzAAAzAgGLAt87cOMLAU5WDY117xxpN0RQGYAAGYAAGYAAGYAAGYAAGYAAGYKDRGHDDtvNzgYGHgYfDDgMwAAMwAAMwAAMwAAMwAAMwAAMwAAMwoBjwrTM3jjDwVOU0mhtMfnmDAQMwAAMwAAMwAAMwAAMwAAMwAAMwAAO1Z8AN287PBQYeBh4OOwzAAAzAAAzAAAzAAAzAAAzAAAzAAAzAgGLAt87cOMLAU5WDY117xxpN0RQGYOD/Z+9tY+s40ntPAv58gf2orwcwcM+AwTKjAIL1wcLGID/EZgxEjBa4RPzhYBwsZ3VniCtDKw+sOWuMdSJshtZFHI4XHtlzJ7y2vBzBnlCJPNq5lpl7NTkOIlPOGJQyDm2PQwFWlgmkcDyGaJjOs6iuerqru6v7vPCtKf4IEKfP6e7qql899fbvp6qwAWwAG8AGsAFsABvABrABbAAbwAZ2mw1UQ7ZLYoGAh4CHwo4NYAPYADaADWAD2AA2gA1gA9gANoANYAPYADbg2UAinVXjCAHPy5zdpgYTX95gYAPYADaADWAD2AA2gA1gA9gANoANYAPYADaw+TZQDdkuiQUCHgIeCjs2gA1gA9gANoANYAPYADaADWAD2AA2gA1gA9iAZwOJdFaNIwQ8L3NQrDdfsYYpTLEBbAAbwAawAWwAG8AGsAFsABvABrABbGC32UA1ZLskFgh4CHgo7NgANoANYAPYADaADWAD2AA2gA1gA9gANoANYAOeDSTSWTWOEPC8zNltajDx5Q0GNoANYAPYADaADWAD2AA2gA1gA9gANoANYAObbwPVkO2SWCDgIeChsGMD2AA2gA1gA9gANoANYAPYADaADWAD2AA2gA14NpBIZ9U42lEB7/PPP8c4PONAMd98xRymMMUGsAFsABvABrABbAAbwAawAWwAG8AGsIFebMDoVVX721EBb319HQEPAQ8bwAawAWwAG8AGsAFsABvABrABbAAbwAawAWygMjZg9Kqq/e2ogGdg4IWHCt6LCs612As2gA1gA9gANoANYAPYADaADWAD2AA2gA1slQ1U0fvO6Gc7LuD927/9GyIeKntlVPatqgAIl8YFG8AGsAFsABvABrABbAAbwAawAWwAG6i2DRjxzuhUVfzbcQFPoRj3RLzxqm3IVDTkDzaADWAD2AA2gA1gA9gANoANYAPYADaADdxrNmD0qCpOm1XNzHxWRsDzI8UxBCAAAQhAAAIQgAAEIAABCEAAAhCAAAQgYAkg4GEJEIAABCAAAQhAAAIQgAAEIAABCEAAAhCoMAEEvApnDlGDAAQgAAEIQAACEIAABCAAAQhAAAIQgAACHjYAAQhAAAIQgAAEIAABCEAAAhCAAAQgAIEKE0DAq3DmEDUIQAACEIAABCAAAQhAAAIQgAAEIAABCCDgYQMQgAAEIAABCEAAAhCAAAQgAAEIQAACEKgwAQS8CmcOUYMABCAAAQhAAAIQgAAEIAABCEAAAhCAAAIeNgABCEAAAhCAAAQgAAEIQAACEIAABCAAgQoTQMCrcOYQNQhAAAIQgAAEIAABCEAAAhCAAAQgAAEIIOBhAxCAAAQgAAEIQAACEIAABCAAAQhAAAIQqDABBLwKZw5RgwAEIAABCEAAAhCAAAQgAAEIQAACEIAAAh42AAEIQAACEIAABCAAAQhAAAIQgAAEIACBChNAwKtw5hA1CEAAAhCAAAQgAAEIQAACEIAABCAAAQgg4GEDEIAABCAAAQhAAAIQgAAEIAABCEAAAhCoMAEEvApnDlGDAAQgAAEIQAACEIAABCAAAQhAAAIQgAACHjYAAQhAAAIQgAAEIAABCEAAAhCAAAQgAIEKE0DAq3DmEDUIQAACEIAABCAAAQhAAAIQgAAEIAABCCDgYQMQgAAEIAABCEAAAhCAAAQgAAEIQAACEKgwgR0X8P7jn1+Qf9f8jtz35En+YbDtNmBsz9ggfxCAAAQgAAEIQAACEIAABCAAAQhAoKoEdlTAM8LJv5/6z9su2iAWIpb6NmBsEBGvqlUU8YIABCAAAQhAAAIQgAAEIAABCEBgRwU8PO8Q0nwhbSePjS3yBwEIQAACEIAABCAAAQhAAAIQgAAEqkhgRwW8nRRseDbiYdYGqlhAiRMEIAABCEAAAhCAAAQgAAEIQAACEEDAY905pjA7G6A6gAAEIAABCEAAAhCAAAQgAAEIQAACVSSAgIeAh4CHgFfFuok4QQACEIAABCAAAQhAAAIQgAAEIOAIIOAh4CHgIeBRIUIAAhCAAAQgAAEIQAACEIAABCBQYQIIeAh4CHgIeBWuoogaBCAAAQhAAAIQgAAEIAABCEAAAgh4CHgIeAh41IQQgAAEIAABCEAAAhCAAAQgAAEIVJgAAh4CHgIeAl6FqyiiBgEIQAACEIAABCAAAQhAAAIQgAACHgIeAh4CHjUhBCAAAQhAAAIQgAAEILALCPyXq38h//PUf5D/6eRvb+r/6Ev/aReknihCYG8TQMBDwEPAQ8Db27UgqYcABCAAAQhAAAIQgMAuIXD68n+JhDsjuHX6NyJfp2v0vLmWPwhAoNoEEPAQ8BDwEPCqXUsROwhAAAIQgAAEIAABCEAgIqACnvks+7vy0bux0Fd2nTlnRDwEvE6UOA+BnSeAgIeAh4CHgLfzNRExgAAEIAABCEAAAhCAAAQ6EkDA64iICyBwzxJAwEPAQ8BDwLtnKzgSBgEIQAACEIAABCAAgXuJAALevZSbpAUCvRFAwEPAQ8BDwOut1uBqCEAAAhCAAAQgAAEIQGDLCFy4/j/ECHVF/2a6a9E5/3ddA8//LXTMFNoty0oChsCmEkDAQ8BDwEPA29RKhcAgAAEIQAACEIAABCAAgf4J3H/69zZ1h1kj5HXz33+MuRMCENgOAgh4hQLejDz7ixW5sbIs5/78WUSuQk4nc2xeWnGme/dDOd7Dffft8LXbUeB4BgQgAAEIQAACEIAABCAAgTIC3YhtRdf4HnbmGuNd5/8WOsYDL5Abd6/J1G/vk1rjNbkVOH3P//T2KRkYGJDD5/Zk6iubvfe4gPcDOfcvjv2Xn8j3vpsXm1Q0Gn7vtrvwC3n7pyflvgsfyh3Ntn96JydS6X185pki4Knh8AkBCEAAAhCAAAQgAAEIQKA3Ar44918X3hCzo6z5r//x78eedPqb7jar9+iT9HcjznX625iAd0c+euNFeaLxkHxl30Ak+gzs+4o81HhCXnxrF4s/WyrgteXUgGMV+KyEaIaA16nY7Mj5e1zAOynDf6fCnMi7V/JikxXgPKFPvca++xN561cmT76QG+/MIOD14B1XJOANv/GOvPVPn8qdX1XTM29HSiAPhQAEIAABCEAAAhCAAAQg4BFQMU4/9ZTxnjO//e+v/V/6U+Rhp9eZT/3bFgHvzjWZ+t19kWi37zcelqPfnpKpP56SqW8flYd/w/5ea7woNz7VWPFpCTgB73eOWl6Gmff/4tuxK9HOAUPA2zn2JU++5wW8+777jtxQAJ/8LCzEvfRz+chdc+fDn4Sv6UHA2uteeUUC3vEP71rKKpJWjKmaCZ8QgAAEIAABCEAAAhCAAAR2ioAvyJnj//P/fSGKyj//+k4k4P3jbevZpoKef73GecsFvLvX5NQDxovsoDz1RsjT7o7cOHdUDg4MyL7HXonH2xq/vf3pBLzvtKuLAQGvknlz7wt4Tz4rsaD05SfybEA0Srz0PpU3zhd56fF7t8JkzDsj1CHgVbIOIFIQgAAEIAABCEAAAhCAQIUI+IKcHt/81/8viuG3f/J/R5/mu57zPzUZWy3gXTtzUAYG9skTP3VOGvrgzOdHM4ej645eqIBXWSZuO/cVAa8X9h9//HHh5Xfv3hXzX/RXdm/RPVX+fQ8IeCfl/r/+xOXBF/L25awQ502f/dX78oexwJd47uW98p6Vb1z7RG7d/SLJ27ufyrvX/kJ+K7rfEw1/9b78QRzmSblv9v14EcyP3vuB5+33bLxe391//Cvv92x8TyaCpBHIXv0beffXNhp3/uEv4vvu//Ofy7v/elfufumi+OVduXXz5/KN6XR497/6M3nDTGvV60Tk7t1P5Q2zDqCJ918mawHeeCd9b3Lurrz1l8m5nIDnhZEAs0dJmN+Tb1xblo9+7TH98gu588k7aX4+y00+zsaN7xCAAAQgAAEIQAACEIAABLabgC/I6fH/+l+/lYqG+a7nNusz9YDSL205Zda7a7yWrBtfeP01mfrNARk48ko8DhbxBKwvbsnlP/6aHLzfrglXe+Brcur1GxKUZO7ckNe+k1xrpu0+ce5GJg635JUj+rw70n7uqDzshT11pVsh0cWxKN6fmriMu3X/9slXfvcJeeXvg7EOkPHSHzhrf7ojF79upiEfllduZi56dyrybHz4+zqP0J03LJ87KocfqNm1CAdqcvDIKbmcSfKtt6bl6O9+RfZF6+/V5GBjSi5nnSgr4oH3p3/6p9JoNMR8Zv/efvvt6Jw5b46zf2X3Zq/dLd/3hIB333f/Rt51AlVOHPOmz9668aNYALvvySIBb0a+90lSMO9+/oX4Op4R++5/8qQkm2LclnPe5hn3v6NbtIpIanOMn8VxLF6rz4pkiUC2IjeSqIgKjaN//UlSiX3xhZg4xn+//lCOa3z+7B254Z3y0xILa574Fv+mwll8roOAd+F9+chw8kVC8/3zL+Tdt02anpXYO89E1MU5uj7jxdetF2A/18WMOIAABCAAAQhAAAIQgAAEILBDBEKC3Nif/R+p2Jjvoes28lvqAWVf3p2Srw4MSLdede3vGCHqqFyMhSQnYH37FXnlsZp85T+cklfeuCyXX39RnjhixaeDf3QtLeJ92o6m7O77jXE5de6iXH7rNXnx+GGpDQzIwe+0vWtVwJuWV77zkDz0h1NR2BfPnZLDkZB3UKbeLUucnisR8KJ4f0UOH39RXnvrsrz23NfkISNo7jsqF/9Z7y/77EbAE5FfviKHBwbkq6H0PTAl1/xH/PKiHDVTmvd9Rca/84pcfMvyfOrrT8hrngD40blx2TdQk4e/Ph3F/eK5Kfnab++TgX3j8sovvQArIOD9+Mc/jgU6I9KZ7/pnvO7Mb/6/74lXdq+GsRs/94aA9+Sz8j11wvt8Wb6tApQR2uJNLm7L6y8lXmRFAt6wCnBfrMi5V5+NBb/Rd1ZcpeGm4Xqedr4g96zGw1jL58vS0rjEYtiKvKQCm57LfMYCngnj18vyvfPfi+Nx30vvyI1IKPtCblz7USQmRmKWJ9bd+sWPo+tb/+jUu88/kWc9z7zfunBdzv13xyKOl0jfAp6LfyzSZUW5l67HayLcWvTi/N0fybPvvSPfyKS/H3Gum3t2YwEmzhCAAAQgAAEIQAACEIDAvUUgJML9YsVOIzTr3pk/8z10nZLYyim0t340LgMDD8mLS/q08s9b58w0Wt+TzAlYAwNy+AcZLzK5IxePG8HvYS9854125EX5yHNAMU+1U3S/6olyTsDLCXsisvSiPGzW5PvWZU/wK4p7iYA3sE/Gz6XjffenT0QebQ/n0hMKv0sBz6Tv+w+nWNjnZEXIj+TF3zHiXUaEyz468tz7qjz1Vqyk2is+bctTxkvy6xcTRyAEvCy9SnzfIwLeSbnviipnX8jbOj30SW/67O3rMpwSikIeeD+WN6KdaUXS01+N2PUTect5w1lPvuS7esbd9+Rfydufi8ivPnHTXj+VN2atUBZ77OXi4YuK9jgR8Py02HN/+Au3xU8gnFhAc9N6YzHRCHhFouF2CHie2Hlrced2/K1EiSQSEIAABCAAAQhAAAIQgMCeJpAV5p68OB3zMOf0z/yevVbPbamAlxPk9Knhz0IB7zczXmR6+9KL8pAR92acSBYJb/vkVH6WpMidi3LUv1ZUwHtCLnuz1WzQH8mLIwMy8HvdbKpRIuCNvBg7oGiU5e5lecJMSf12NxtTJALmQDSN1U4ftsenJB3CNZl6YED2GXHNbRyS8058+1QkHn7t9YwwF0fOHKj13/IAACAASURBVNyVy9/al5nKnFxw7cxXZWDgqeTZFRDwTOzKpsEyhTbJv2056sYravOuSaaoxoKat0PtR3/nr0dnxLCAgPfdZLfaMkAafuzhpmKaE8PMeRXPPrpmvfhe+icbYl4YLBHwfA++SHxM1tEri584D7j7r3ySvHn48q7c+PAdOf79xKswYr8dAt6TiTBq4n33Xz+R1+d/7NYTzKd/82wiHXYpM05CAAIQgAAEIAABCEAAAhDYBgK+KPfAcw351dpn0VN111n1wjO/m/P+9Rq9rRfwNsED73iRJ1zaQ+3OhaNuTTdf6Mocxzu6OgEvKNKpuJcVyZSa/1ki4AVFutD1fnj+sbv2d47K1B9PZf4v5sTBu1eekq8OHJSvNR6Wgd98StrOZ0dD/Oj7D8nAwLi8ll3HTi+IPt1ahDnB0OfoeUlWRMAzUS/biIJNLFKZvLVftkqIKQo3FtRUwNLpsHJbzqWmzxYIeJ6YdfdfV+TGP4X/3/5bt5Ze7PW3Ii89eVL+4IYtaWZKbbwW3ic/k9gzTxKPvKI0mN9jD7zsVFTPC1A+/7Qwfjd+qRtDmM04VlIbWJgcv3Pzb+QP1CPPS/OWTaE14uOf/Uzeup3xh/78trzxU296cMpDMi2+lfHq9tzWWjuhQwACEIAABCAAAQhAAAIQ6EzAF+T+n3d/Gt/g/64/mvOh37dSwJOr1uOr2zXwrv2RmRLreXf5m1hoQlKfaQHPevA9JE/96LJcNmu7hf7/Xr3PnEiX2nxCA98kAS8WCzVc89mHgBcMxw9Tj+/Ixa9boe1rP9J06jmR9nfMuU6ipIvfH06H+UVMr8kt9VqskICXpJSjvTOF1og/l5edx5nZdMHbKXblnWStuFgkCnjgeWu1qedcqTgUe/iZ3W9/IK/fNgbn1rjTsIwIp1NIc4JcWKQqFvD0GSLyLz8PpCkc3n3f/YEc/9sP5YbbzdbE8u4v/5tdV2+7BDzH/bfO/5W8fvNTb8OLkLhakI447/o7T3UAAQhAAAIQgAAEIAABCEBgpwn4glyvx0a4039z7+hL/yn+rr9nP8015tqu/3S6aC+70Prrq6nYVSRguWmx+/7IbtNg19zbJ6eudhPDe1DAc2vXHXxgnwxkN68QESuQ+puEhDhdszsHP/aatxtw6Dr3GwJeCZydO7W3BLwn/5tdg05Ebv3D+3LDcX/3rzPTRiMhKCDg6Rp25r5P/qYLgSwR1G7d+LndZVan08becrfl9cVI2ZPcDrkFglSxgHdSYi/DLz+R76kXXUE4efHxe/I9N5VXp9nep0KjWUDzvfQ043jdPumwC617frwGXxdC5f1//mFcseQ8/7pOT29C3s4VQ54MAQhAAAIQgAAEIAABCEDAErj/9O+lvOp6FfH6vb57/nel/R2zZtpBOfW2umyF77abTOyTp3TB+Ogy5w0WWkvOOJO89VS0ptsTP3Vhu11vH/p+euOI8BPvNQHPWwPvpt2V9uEMBzvFOMs4S0e9+EJrA2avFREEvACUnf9pjwl4J+Xbv9SdV7+w3niFQldIwPPuly/k1i/+m7dO27My/MbP5cbK+3LcE5h02qzcvRvt6GI3uLDCkoptdz+3cfJ3q82La4kYVSbg3fdT9TI0a8m9L9/2dpe9//s/kXP/tCJv/KUN69nFD+VZbyfd+56ckXP/4ozSbXRxnyd6yt1P4uvNTrXJDkDdCXjf+Aet3D+Vt/7cE00v/FzeuvYXHsuTcv8bH8Y74HTLpYxZN+d2vjgSAwhAAAIQgAAEIAABCEBgrxO4cP1/iFnnrujfCHRF5/zfzXXGu87/LXTcsweeyaBP23LqAbPz6cNy6q3Q4mt35MYPvia10G6w6oE3sE+OZjde0HB/85S0dfgoVsQK77J6R9o/uhg7f4huYnGPTKG9duZgagdf+93fodconpflqX0DMvDAKWnnZ9jGxckIffuC+SEiv3xFXvM9HBHwYm5VOthzAp4vcEUZEa1Bl4hjidATFvDu++5P5C1vqql8+YUYAe7uly5bs95l3hRUs/PLW048i57ztyueLbiptZ74l8QlHb9SAe/JZyX2dNMomfjFy8slcYjD+SKTBvlCbvxt4m33h27tvjiyLq13Vm47kS0J08Q5DjfLwhMXo7C+FIm865SRsnSCZnTNrz9MCaJFTDbj9zh9HEAAAhCAAAQgAAEIQAACEKggASPAqYBXFj0zTVYFvLLrzLm+BDxz4y8vyhO/bda3G5DaA1+TJ3RDhm8flYfvt2u2HfxGflOGeL24b5ySUw/U5PDxF+W1ty7LxXOnZPw3THh5z767b5+Sh4xIte8h+dofvyIX37osr/3gCXt9SqzbRR54wU0spmTqB207zo6mzg5IyuPu07Y89ZsD0W6yvj9izOf+h+Wo43P59Rflqa8/Ia/dVAv4SF5r1Gx+HXlCXnz9slx+4xWZ+vrDkdCa2uUXAU+hVepz7wl48dRVmw/F3l0FAp4R2L77I3n2FytyJxbFROSLL+TOyofy0oXspgs/kdhbOLtrrDc9VeKptWmxLiRMFQpksfj3rPzBlfflxq+9CBpx7Ncr8tbfJp5ux/9uRW4lyp4YMfLOv34i53JpmLHpVZHyy7vy0Yc/k1EV3jLCZHH8MptmfPmFvPv2SbnvJbOBxd1EBDVZ88VdufGLn0m8mUacts58Qsy6+a1SJZPIQAACEIAABCAAAQhAAAIQyBColIBn4vbFHbn2+pQcPXIwEoGMmDdw/0E5/PUpee3dIncwb5OKf27LVEPvrcnBxil57e9j17t06m9dlqnGQ/IVI+QNDMi+33hIvvad1+RG6jG7SMAr2hE2EiSd1+EDp+RaBof1pNsn4z/KeD7euizTX3845mPz4cXM/Xfkxrkn5PADVsgbGKjJwSNHZTrrRYmAl7a9inzbgwLe1glA3YhEXFNd/hUpk0QDAhCAAAQgAAEIQAACEIBAkEDlBLxgLDv96Al4nS7lPAQgEBNAwNsGzy5Eu+qKdn7exKWCAwhAAAIQgAAEIAABCEAAAhUkoAKemfba6V+n0Ha6ru8ptH3zQcDrGx037mkCCHgIeOKLWHv5eE/XBCQeAhCAAAQgAAEIQAACEKg8gVcWfrIlO9T+L8//b9uYdgS8bYTNo+4hAgh4CHgIeM4G7qFyTVIgAAEIQAACEIAABCAAgXuUwD/eviVmk4rN/N9eVAh428ubp90rBBDwEPAQ8BDw7pX6jHRAAAIQgAAEIAABCEAAApUngIBX+SwigpUkgICHgIeAh4BXycqJSEEAAhCAAAQgAAEIQAACEIAABCBgCSDgIeAh4CHgUR9CAAIQgAAEIAABCEAAAhCAAAQgUGECCHgIeAh4CHgVrqKIGgQgAAEIQAACEIAABCAAAQhAAAIIeAh4CHgIeNSEEIAABCAAAQhAAAIQgAAEIAABCFSYAAIeAh4CHgJehasoogYBCEAAAhCAAAQgAAEIQAACEIAAAh4CHgIeAh41IQQgAAEIQAACEIAABCAAAQhAAAIVJoCAh4CHgIeAV+EqiqhBAAIQgAAEIAABCEAAAhCAAAQgsKMC3r9rfgfxCAGxEjZgbJE/CEAAAhCAAAQgAAEIQAACEIAABCBQRQI7KuD9xz+/IP9+6j9XQsC5DyFtz+aDsUFji/xBAAIQgAAEIAABCEAAAhCAAAQgAIEqEthRAc8AMcIJnngn96x4ttPCqbE9xLsqVk3ECQIQgAAEIAABCEAAAhCAAAQgAAElsOMCnkaETwhAAAIQgAAEIAABCEAAAhCAAAQgAAEIQCBPAAEvz4RfIAABCEAAAhCAAAQgAAEIQAACEIAABCBQGQIIeJXJCiICAQhAAAIQgAAEIAABCEAAAhCAAAQgAIE8AQS8PBN+gQAEIAABCEAAAhCAAAQgAAEIQAACEIBAZQgg4FUmK4gIBCAAAQhAAAIQgAAEIAABCEAAAhCAAATyBBDw8kz4BQIQgAAEIAABCEAAAhCAAAQgAAEIQAAClSGAgFeZrCAiEIAABCAAAQhAAAIQgAAEIAABCEAAAhDIE0DAyzPhFwhAAAIQgAAEIAABCEAAAhCAAAQgAAEIVIYAAl5lsoKIQAACEIAABCAAAQhAAAIQgAAEIAABCEAgTwABL8+EXyAAAQhAAAIQgAAEIAABCEAAAhCAAAQgUBkCCHiVyQoiAgEIQAACEIAABCAAAQhAAAIQgAAEIACBPAEEvDwTfoEABCAAAQhAAAIQgAAEIAABCEAAAhCAQGUIIOBVJiuICAQgAAEIQAACEIAABCAAAQhAAAIQgAAE8gQQ8PJM+AUCEIAABCAAAQhAAAIQgAAEIAABCEAAApUhgIBXmawgIhCAAAQgAAEIQAACEIAABCAAAQhAAAIQyBNAwMsz4RcIQAACEIAABCAAAQhAAAIQgAAEIAABCFSGAAJeZbKCiEAAAhCAAAQgAAEIQAACEIAABCAAAQhAIE8AAS/PhF8gAAEIQAACEIAABCAAAQhAAAIQgAAEIFAZApUR8FZWVoR/GGAD2AA2gA1gA9gANoANYAPYADaADWAD2AA2gA3shA1URq0LRKQyAl4gbvwEAQhAAAIQgAAEIAABCEAAAhCAAAQgAIE9TwABb8+bAAAgAAEIQAACEIAABCAAAQhAAAIQgAAEqkwAAa/KuUPcIAABCEAAAhCAAAQgAAEIQAACEIAABPY8AQS8PW8CAIAABCAAAQhAAAIQgAAEIAABCEAAAhCoMgEEvCrnDnGDAAQgAAEIQAACEIAABCAAAQhAAAIQ2PMEEPD2vAkAAAIQgAAEIAABCEAAAhCAAAQgAAEIQKDKBBDwqpw7xA0CEIAABCAAAQhAAAIQgAAEIAABCEBgzxNAwNvzJgAACEAAAhCAAAQgAAEIQAACEIAABCAAgSoTQMCrcu4QNwhAAAIQgAAEIAABCEAAAhCAAAQgAIE9TwABb8+bAAAgAAEIQAACEIAABCAAAQhAAAIQgAAEqkwAAa/KuUPcIAABCEAAAhCAAAQgAAEIQAACEIAABPY8AQS8PW8CAIAABCAAAQhAAAIQgAAEIAABCEAAAhCoMgEEvCrnDnGDAAQgAAEIQAACEIAABCAAAQhAAAIQ2PMEEPD2vAkAAAIQgAAEIAABCEAAAhCAAAQgAAEIQKDKBBDwqpw7xA0CEIAABCAAAQhAAAIQgAAEIAABCEBgzxNAwNvzJgAACEAAAhCAAAQgAAEIQAACEIAABCAAgSoTQMCrcu4QNwhAAAIQgAAEIAABCEAAAhCAAAQgAIE9TwABb8+bAAAgAAEIQAACEIAABCAAAQhAAAIQgAAEqkwAAa/KuUPcIAABCEAAAhCAAAQgAAEIQAACEIAABPY8AQS8PW8CAIAABCAAAQhAAAIQgAAEIAABCEAAAhCoMgEEvCrnDnGDAAQgAAEIQAACEIAABCAAAQhAAAIQ2PMEEPD2vAkAAAIQgAAEIAABCEAAAhCAAAQgAAEIQKDKBBDwqpw7xA0CEIAABCAAAQhAAAIQgAAEIAABCEBgzxNAwNvzJgAACEAAAhCAAAQgAAEIQAACEIAABCAAgSoTQMCrcu4QNwhAAAIQgAAEIAABCEAAAhCAAAQgAIE9TwABb8+bAAAgAAEIQAACEIAABCAAAQhAAAIQgAAEqkwAAa/KuUPcIAABCEAAAhCAAAQgAAEIQAACEIAABPY8AQS8PW8CAIAABCAAAQhAAAIQgAAEIAABCEAAAhCoMgEEvCrnDnGDAAQgAAEIQAACEIAABCAAAQhAAAIQ2PMEEPD2vAkAAAIQgAAEIAABCEAAAhCAAAQgAAEIQKDKBBDwqpw7xA0CEIAABCAAAQhAAAIQgAAEIAABCEBgzxNAwNvzJgAACEAAAhCAAAQgAAEIQAACEIAABCAAgSoTQMCrcu4QNwhAAAIQgAAEIAABCEAAAhCAAAQgAIE9TwABb8+bAAAgAAEIQAACEIAABCAAAQhAAAIQgAAEqkwAAa/KuUPcIAABCEAAAhCAAAQgAAEIQAACEIAABPY8AQS8PW8CAIAABCAAAQhAAAIQgAAEIAABCEAAAhCoMgEEvCrnDnGDAAQgAAEIQAACEIAABCAAAQhAAAIQ2PMEEPD2vAkAAAIQgAAEIAABCEAAAhCAAAQgAAEIQKDKBBDwqpw7xA0CEIAABCAAAQhAAAIQgAAEIAABCEBgzxNAwNvzJgAACOxRArdmpVGrSe30wh4FQLJ3lsCCtIz91VqCBe5sTpinL5w2edGQ2Vs7H5fdGQNnz41ZWdmdCah4rOFb8QwqjN7K+YbUajVpXS28ZBNPbGO7crUVpWs39qGqX9+vyGyDNskWDFhsYgWxrUFVv5xtK45NfdieEfBWr03L2OBWD5S2seHcVDMoCezWnEzsr0n9kWlZWCu5jlNpAk4capy/V4cya7Lw3KjUa0MycWGXprGyAp7WIzWpNWZleT1tWtlvOjjI25oXTiQUmc5g5t+Jl7aRzZzLXqvf+xmgr6/I4oUZaU6Mysh+fU5dDjwyLpNnZmUhJZpovLupr3dzx66XdGZzne+bTYCO5kaJOnvup37Y6KP3xP17ge8W9it2sC+rbTQC3orMHR2S2uCoTF/b2QFF9ev73dy32ewKGRabTXS7wqt+OdsuEpv/nHtfwFtfkfZz4zIUDT67GRBuBPI9OCDbwU7PRnJix+9FwNvxLDARWH1/Ts4eG5Pp0Jvv3SDg1erSeL1cINXBQbGANyQjj47KaOj/Bev7tfBC/vyBQSO0Be791qWePGxW2tMynhLt9Fkjrl7OekH2Uo/u5o5dL+msRHHqORKl5a80tFVZunBWJg9Pb5t3Ih3N0gzp4uReEJi6wLBll+wFvgh4GzefbWxXevbA20YBb21Z2udb0vjmXLC/Uv36fjf3bTZuxekQYJHmsXu+Vb+c7R6W2ZjeuwLe+qosXmh5A0czGEXAyxoA37eIwD0v4G0Rt00O1jYeBVNXqi7gPTomY0ZEG5yUS7eLwXQW8Pqp97TD1M+9SVyXzzekbl6eDI5J68KirGa9CdfXZPnqjJx4yZ9E2ssAROO5G6c+9pLOhOluOiotf6UJ2X42dDRLM6SLky7P8MDrglU/l8C3H2pVuEfbaDzwtjE3tH9XUB9Vv77fzX2bzc5nWGw20e0Kr/rlbLtIbP5z7lEBTwu7GTiOygkzdQsPvM23HkIsJoCAV8xmG8+UCgjawavcGnhuoDbYktlX7do59ZPzUjThRAcHxR54/YhwWof2c6/N4LWrLRkx9e5wU+bLnQgzFtGLeKPxRMDLQKzE19LyVxrDXmygNKCuT9LR7BpVwYUITAVgNuln+G4SyG0PRttoBLxtRK/9OwS8bYS+VY/azf28rWKyO8KlX7V1+XTPCnhz3xyTyRfashyNerdrMLBdz9k6gyDkTSKAgLdJIDcWTKmAoB28qgp4xmN4fdktZFyX5nxYwtPBQaUEvPVFmR42Xs8Nmb3Zax72Uo/u5o5dL+nslWE1ri8tf6VR3H42dDRLM6SLky7PCgbMXQTAJaUE4FuKp8IntY1GwNvGTNL+XUF9VP36fjf3bTY7n2Gx2US3K7zql7PtIrH5z7lHBbwsqO0aDBQ9x+94rcnShZZMDA9FC8rXDzZk+krinrJ6bUYmjxxw084OyNixGVlYzabHfb+9JPMvTMr4I+76Wk2GhifCU9U0iNVFmTs9IaMH63ZB+/0jMvFcW1bWiypIP+4aiIho42jEDzddWdNUM2GenpOlz7zrM4er1+ekNZGsgdUx3v79t+dkIpqWZwQO/4R3/PGMjJlrjs5JFt/K1VlpTYyKXePLeGkekNGJlsxeTfIhCakoT/WKTuf1OvuZqsw+npPmYWsHQ8/5UwjN4m02n+IF/x3TxWxi0sGnvmmnMS/s2MvKztv88RjtH5HxYzOy6GlIen+2U5r6PZMOm89Lhd5kmu7u7TOV5PiL5aybJaQ/4/j6NizpcmlsYuzkrJTx3pANxzENHWRs6gNnyw+2pB0oU8o7n8+ZcEKPKvxN64P+PPDWrjSjOuzQc4uFTyg+0Uu8NZ49euCtLcvC+ZZMPJrUQfWD9qXPSqBOSZXbW+1oXUVbf9TlwJFJOdsO1R0uhdH14/HmHclzekjnWluaZjr1g9MSIrr4/KGoPp+8mK8g1uab0bmJC965LU5/V+UvaADKJF1m7eYreVtc/WBezh5L2EZrNj42KWffXMpP1w4+L/kxlcfJzyI3Z6URrQc5Iq2rWgG6eLrB4cqVs5l2+6yUmYSsurbb9QNM+oaGx2XyhXlZ8rJJZE3aT5u2+pBMX/cj5Y6vT8sh085981KunZO1eTvzINAGJiFtZvgZJm2PiVlLc2Ja2v5mNRtsx5M0eEf99Im82/VQ69SorTDrKL8wKWN+n+n0XHHb0E8c1ldl6c2zMvmYXx+NysTpS7KkJidpvhpX8xkvVTDckgW/jeixnEdhan8u7lcmeadc8m2NjU2qHNRc3ej1b/04Fx3rM+J22rswCj/HaD64xpl3mzss4Of3AzTtWi676ctm6yDXn7z0QZxxEk6Ti0/h0j4dzvfbrmTt2fR1jp1Nl80MvDR3r83reQ28ot2+03mTsiONX0kTm4qu5qepF7P/npiXqu9X/DbdjqP8cVkqfPOlD4a5MPSH0JgsGj917tukOJWVN2XiXlan7zPluyVz11MNj8bOfmb68TrGC/WPU7aeua+v/n83LHrJD5/FZpf391x77NlZGqRIP33jfPlryqzJr6Lyl0njwgsNO94dnJA5LUc9tg2p8pLJ16RPm01tprz3Ws7ywfGLRwABz4Ox8cOiBlcbp7Mye3pEjGjXPDMlrWNjbhH3upiB18qFCRmqDcnYsZZMnWlKQzuMw1OBHWD1WbYxbZ2ZkqlnJmXMLRY/cmYhL5LEAxHbQJk4TJ2ciAaW9cdb0ho3DV52IKxxn013kuIKYl4WTo9Ibf+YTD6TjkN9fEaWcoPhNVl8fswKlKZzdHIqldaho+EFZ9N5o4OOuoQ6eeba5ZfHpFarS/NK0omS9WW5dHzENupRp8Bwtvmgg/HRHDflnB882jh1Op+OeVwJvndJJqNBoetkeF5ga++ZHZPN76ZhbUZxbD7uRNr9EzLnD4LSwae+aUNa2Nk+b6dnZs/HgwHN0zNNmYg682nb0PCzeRD//uaCtIZrMnR4Uox9+vY+/vJSKq7Rl77sMx+M+WX5krGrKZk8bPmOHbPfzW+XPnb3dGHDtWDZ2wwbDsfb/pq3qcXnrd2GyrXyzuaj6ICvcIBQFgftPBbZfdm9IgtnjOhQXD473C2tqOPdzbM1nmnbLA9f7/HqQa++rT+er4Picts2Yk5dDjyeKZe1oXRd4yKwdnVKRqOynNTTphyY+mbo6aac6DqdazJ/0tjyqMx8kE3dksw86uqRwFTrhdMmL3w+W5/+rspfNhnR92W5ZNqlM5P2BUxtTCaj7+a3S7Ic3+OVQa+e1PYsEsSOznXcwTkOTjIdTT2xMi/NyJN0RFptf4CTtItts87j4AFpZNqy2v5mUHBP6ve0/SUvwCZkzvNaVQF29If5OnPph6NukNqUnIPu1VbUzubrBU2c/dy88LtgMtjw2q8+2/F09L1vWm8mZa1jn8i72z/UOrX15oJMPVKXaIBi+jemLVRxZzDkXdxHHD5blOkj3gtVZ0f2OX65TfjqGMzEeWW+6ZYqaEnKRKX3cm76SLOPF8RlsCGtZ8Yje8vZlNe3ilnF/dG6jD4X6I/6wL3jmH1q46k128+s1eL+s8nb8SgvumknzAPC/JKX0Rvoy3p1UNO9mPYZhdOktlIU/+LzfbcrNy/Jiag+S8pI3C8L7gy7JgtnRj2HAtNnbrmXFUPSfPqErX+8/quXlcHDuC1N9WOTvOm1Ps09ZHVBZkyb8XTDvtx40I63TN9v6uWF+EVHHI/CNr1gA7GeGeZimPxwc04m3JjNiFupMdl4U5rBMZkREJOxTFflze/rmvz0xj9qr6a/1jiftLAayaS96m48Ett6z/3/Pln0mh8+i37HroXlfVlmojGHX28rSfOpbd6YzOg4xD+dO07qvcipINIGtPyNSOt0QfmL09gWHTtYMVvj1XvbkJQXm0+xvZp6PtYqMi+Q/H5Vr+Usx4IfsgQQ8LJENvS9qMF1vw/WZejx2dSgIl4n6sERGRn03/DbSlo7U3nPigU5+62Ah9D6kpyNKpBJueSPN+IpbSPSzC5I5XfaUgM9AyNpWP1OY9zpCaRJ4jjUctP+Vi9ORp2BeoaD8eJrm8q0ZsXMTtmgbzHqT7fzQqUsyvSDxrOuKW1Pv9OKbOR4YGC3uuA60dnnF+WpxrDTeb3OftpKcFwmjo7I2Bnj+Zg+L7edsGcGB5kKfrVt1xSrh7wtMsGYr9qQ+h1J/7LgefXaeHAq7914azHlHaL3Fwl49cEhabya6RC8f9YNzDMDzr7t009R/tjy7rCJRdCGl2WmYUWRbNnbLBvOx1Z/CdiUxyfriaP5kM9nDceJO9m30bmyrs83n9rAFw0u/Guzx3pvI3nbl72k9LvGu5tnJ8+aTQ0Iyh6wInOnM95A5vL1VZk/aQaveW8na0d1qZs6Oj1KltU3rbdh7dEZSUksXlmeed+riMyz4vrG5E036RQxdmc6Ybl8jryN6zK0vy61waxXsqsLU3HbpvTHnbeC8leWRdruFLCJy+CR6byH+vqyzLkXNSPPh/wVww+OO6hqR5/ZFxC1WqDN1PgN1qU+nBFOYjuqSU50i21iTKav+Q20jdPyhRNOjJmWRW0bVi/JpCm7ubf6drBQ3z8k9YBYbr0yQ4JvJv2bFr4rtwVM2s9YL9FDXp70045nYu997bFP5N2ZPdQ6dWR4VMaeX8h4c67J0g/d5jw578Ze47Aic5FgVg88R2TlzVlv/VDH17ODuP8YXGe093Ie95Gens/1TZZfdWkO1EHxfafbaVZxc/jA8gAAIABJREFU33Ik7EGaBe/1W1L9ig9mZNSUAfOCV8uFu3f1+lL65XIgTPtTnl/0uw52g/0A7U/n+7Irr1se9VAddGteZt9MesxqT6k0aR1SUMfFfe/s+bgOaUhP7YrXh8i2YfKx8zIe9uod00zF/fWZ3Iya1Wv6ojm7i3xhBkQncvWs/dW+tCuoO2y7HKhPyx6l+eqVF//y0jb9SsuJfxmP9z4Y+s9MHXthhcZk2oalX77ZEHoub8pieERGA/a69v6M8zKfkDl/wzTP1rodj6itb2b/v5CFx7Bbm+537NptebfOIzUZezkz9jFZp23t4RnvZWTKKlJfSsufGw9GwlxWQNf8PjohE8MnrLdeKuTe2wZbXg6JaRNPXMikbX1F5p+2TgbZWTd9lbNUXPlSRAABr4hMX78XDTz195DqrgPQmhw6HXhL2W5Fg7b6mcwUy5L46Vt5v7Ogg79s4YqD+WzeTtHKDeo7dXqaMu9P2XAB6lv9dLwTYS10j+iUmm4EKlNxB0S66PHOjTmVVq04zRS0TAcwZqDTblOVq+Zd0SC70/k49OjAVmY1KRLh7KCraL2zVZk7agb8GXE2/Yj4mzakuQG/uyJ4Xiv+LvJA7/ftzAStv4c3XlBPorpMXYujGosTqTxLTosU2qd/Uf5YeWfjGF2paa1lOi0uGLXhWqpxTOxuwzacj677JWxTa9em7OD+cNqzVXnn81nDGZKRR0dlNPd/Qi4lY4xMbLReKrL7zOWprxu51wSk8e7m2fosfbOYikjPXzTPsyzVjsYCXlAi6gGXFix1Wmtq6qofI50anR2g+df4x1qHZcpmlP+DTZk7bwS+jPjoBr45IckP1zvezPSbYJVbsPx5z80fltmAK4MF5TYKS6ccZ17i5J+T/GLj6uzosyWZiYSVETlxMdNZjW7R+I0FPCJFRAWHjDdnR5uI39D73uOrcumbgXo/qr/q0nx9NhL4fGEstsmUcJukNX20WeErk0My/V76CdE3ZeIPqHttxwPBdvNTqE9Udp/WqXnRVO9ST4tQn06vSX+G4hALmAHP2fTd5pvj6/jFg+7hE3LJ89jM35f/JVjOtX4p7COZttt656XqR70v0y7FT9X+mCfcxucCB8o+VWe4qWJpGw/cXPpTml98qfYDBnvoy3r1S7AfEAduD4Jp0vwsrP+1PKXbwY51SEG7omOAcBsmsvicEdj99qNzPbv0QzPTZRMFvFpv9WkGc/qr5qtf33hXaNsU7nMmbbr/YrB3ht4DM4caVvj5Utzn7ae8KYvc+C6JVEh46mc8ora+Hf1/Zdi9TXvLP21VeddxbGocaTmvXpiIykthnzDJDhF1RCnp5xSWvzi/u39xoo8Otg1eXy6oU5ibvTrRd5zpp5xpXPgsJ4CAV86nx7PhBjfueBV0ptXAW+3A427OynhZA3l7SRauzEUu482JURmN1y1Jez20nzEDgLK38UUD4Q6dnsxgMk6Bxts/7zrw9ZQgEt8h8aCjsEPjX1swTdZUe1FHJN2x1kqpfCCbMIjXCeizk5WOafJN8zq8IYHrNOS8aJL7Q4OA5Gz6SBvSVGfbuyR43nujdeL8YvptunevOdT7Ux1t7/es55revvyqnYbjn+/fPjXU8KfyzsYxulobOd9G/WDUhp/xCuYm27D/uOS4qB5J3Ol9z0bNh3w+F4WTPKn4SMtCevBQfL1/Ru/NeFn6l5Qe9xJvfVYfAt7asixemZfZF8z07nEZ9dbDy7K0duQPbtIJsPbrn1+W2Wj6SxmDXtJpnqdp9cN0vxkBwHUc/TrO2oYfLy/eW5p++5zS8udFJX9YwkaFoA6ih506nH5RkH9O8ouNq5kWqdMIh2Ti9ZB4Z+5x8StYk1DW29ajJHVeB4V+/iXPj4/c1Ff/5ZeWcb/dsL+ZsNyLHb9/4eq2bgWPzQm/Sybjsynvg+ByFwXteMyo7KDLPlFZEMqj8XrhGw7RdszPkzjMLuNgbbRA8IwD0wPHtzEry7rchFlSo5N41205dy+L/fpDn6yfMZfzCRftW2XrTL1H1Ku/QEiJr3MH+oxUmx17Ao3J1JVlWSt6CZsNLPU94ZfE3hvQd+oH+OddGS0UX1LPLeorufgU9ndD5/ttV/TFaXE7mctH7f+U1bNFa3Bl0u9/jetZ9XSOTrq0pupL765gfeqdDx1q/67A7rpr08e9Tbj6YBiKl/ut3z5vLp+yzwiVN2WReaGUujWX3/2NR7T8+v17/zlab/rn+2PRZ34oC788+xFUDv75nsp70TRZ1053+1JR49FP+dM0+n0CP4163G3bEAt45f2pUD72Xs40cnx2IoCA14lQT+dDDa4JwP1e2pCkBbf4sVoQs6KXWQPhW25tCiPwmYVzHx2P1qFrPm6nqiQdoNDAL36CO9Brsg18QdyL4qXB6nk/zdrYm/iW/ncpGugz/ApO3wJkprZooxLsbGuc40rKFzqL8lRv6nRer7OftjLzOwX+eQ2rE58CW/GD8oS0oo61MsmeN2/2dV0OY1eN07OykOps2Qfp/Ymdlf+u0dP7kueq7ZUNbPWarH1qqOFPy7uAl9pPtmxpUHp+K21Yn5X6VDsIlIPP2tIynqfmTaobuOV5amAl4eglhZ/KOxCHwnv0hHas/HKk57r57CXeGs9e7GJNFl8Yd+uPGpbWQzFaj/OY9ShIbNPG19pR8TPydubSUNqB6iWdNh42r721BZ1oZ+s1xyK2V+dZlfOo2Y70+9wKyl+pKZSwce1ImdBggtZy0anO12jYPDwkY0fsVBBfJNdrkk8Xv5h1csYeheLvfiu1CU9Q8Ns1VxclL7/cYMBdY9Oa2Kf1TOhWGEqeubHw+2GSPLvmp7egHc9STn3vqU+UujP3RW0n27b5F+o1qbqipzho3VXW7vlPdHwfHJOxrnb47q2ca3rKyote46dZfyvv04WmgPtpS441vCz7lfmWW0+0Ztdczm34koQRPiqwT23ne+gHaBzLWPlx0OvTaXLx6UnAc/eU1iGhcNXWOvct47ztpp7VPn0ROx+COw63pQV5E98fSlN8Mnyg+VpQR4fjkQSVb9P7YJgElznSsMrKvl6T1OsmELWlnsqbsijLJ70m5qXMO9uMb9caP/83P/F6Praz4ItJ/44o1TIbLWnjs1A+neOXPMtrb4pY5DgkzLst72ZziXotM+XbzfDqVvTXDSpK+zlF5U/T4DsfpJD21jaYWzuVl+SadH+v0335cpaKKF9KCCDglcDp/ZRWeNlBb3njVGrAWhBTlY1ZWNauFzd2Zl6W/DULvAo+qUC1osvGy0+hXuNXkOZ8QdyD8fLC0/NxYyDxjjmH3ALw0aKy8QLlySYD6cXKvTBzh/kppTolJeuirI1G0MvRC9fmhS+wFeWp3tTpvF5nP8srMxeWv+hukI+3EUM6+NQ3TXOq8fKuKD2vu+K5XXLN2oRjzy+m1hvU+xM7s4EX/a6P1vNJvNT2+rFPDTX82XvZ8sLZFhv2nhcfltuUeQNrOge62UKepwZUHo5eFf7sJk/Cd5pfQ9Mxiq/Onukl3hrPbL2VDTP5rmuZjByfkYWbmbXpXIcosU17X3m51c6N33Fxach4GyWxMEe9pNPd6bzP1LPKCjXJlHrroeu+uzfxiShjw9ie9NtnlZY/l6TwRwkbl0fj2fU1MwFpuehU5+ttNq7j0jqdrG21GFgewl7v4ue3bxpQ9BmKv/ut1Ca8AUaq8+28IdRDxU2jir0YnF3Y705AL/HkTkU1+rIZ4ffDxDy8+3Y8H2/9pdc+kd4X/oxtJ7WRQvpavSbx0us1Dlp3taTdlUeZ2k9LWvG6eek22Y9hr+Vc05Ntz1NhBja+0vv8jaKCfTtvAwE/zOyxhheMx/pKtHt4vMFbrWiKezZU873APrWdT/Wxvfv1vFfW4zh6zvneHbnD+PqUPbn49CPgldYhoXDV1vxNgfw+d3I8c9WtzdlNPVskIOQIJD+E29KCvIlvC6UpPhk+COSbf2E4HskV+barD4ZJcJkjDaus7Os16b6N2lJP5U1ZFNm4iZ1eE3vpOeY9jkc0fsHy641Pkz6WprNXFnpfDzbtp7OIhXLYQHkXXRpC2+u4T9zDS7WCvmjKkIrKn6ahII29tg3mmZ3KS3JN2kuv0335cpZKIV9KCCDglcDp/VRRI1PeOJUacLAgLsiU2dnQqxz8uOoaGUkFahZKNm8pfGHKv8Mcq2t+urHYzE6PXLdbbPtTg7Kx6PW7CnZ24OI8TgIuyrpWQunbjPhNUDIgjtPfUyerOBXllZmurzYl3a94WPwsbUiThjJ9bafzevXaTd2JsS4nvJ1R9P7EzuwdRb9reHo+iddG7FNDDX/2Xra8cLTseQ35Vtiw90R3WFSP6JVmgG7WInIbrhQ29J3C0fBCn9o5KhNVQ/e533Q9yQdbwZ04S+5MBlqFZc6/W+OZrbf8a/xjtTW/jCfntZ5IbNOeKy+32rlJC3hRHV0r6ZTqtKCu0qlxdEJL5IHhhBp/qoerY6M3xdGxv5aaCWO70u9z87loOjp9ltiuS2PKYysQXDw98XrgZOCnJI+THW7rj+cXb7e3lrfp4XZD15Qq87qwL7qMQK8irUbVirPWq9VOo/Jt2IUdebHZ4/AGTxpa/nPj4ffDxMaj23Y8H2v9pdc+kd4X/tQ2qszjIj9Fu9c46NqD3Xoqe3zjnWvr0vjhUurFmk1R7+XcDOqMV0+ZMK7T3/z6Ma4zS6YbhymHf1X22X5F9urV93Th/RE5+372bOi7x88/re18wWA3Fja8foCmubw/mTwknCYXn8L6P3Te2VjP7YraWnqd1iSGgaNrU3b32dSLhMx1btp1ep3gzDWZr0k9658oyJv4khCL+GT4QPPVyzf/wnA8kivseb/t6oNhElzmSMtn72Mytb3kxUEm6NBXZeF7OWevyy3d0N94JGzrycP0fFKH9Muiz/xQFltY3k1q7UtsXb7EsQysi5eQyRxtpPyVplF5+/2H5NmxfXnLJJiztjyUtVVJXzxZgkrvK+6f58tZEheOygkg4JXz6fFsUSNT3jiVGnCoIOpvoYZpfcltY+03PCIq6gV3xjGpjBe+zRa0grhrHHqoBONFLnNTunrE7F++viAtI2aaKbNuOlnQRVkXFi17tqYpNf22SNh0kXALNHe7k6TN6yxjTZCundDDWxq9NfSpG6AE80i9HgK7WpaE5XfUtCHOdrSLftdg9XzSgG/EPjXU8GfPZcsPRu3BL2c6tavMjvww+jouqke8wOI1gZoy/6bd6Mbnaa/sIhwvyPShNsZ9Cnii3ijGUzC983b6OaFvvcRb41lUprLhl4Wtwmi+TJSXW+2k+HWuE9dqRRvSiKgnZbd1h6bE1uUNmf1gXpq1mqS8jV19aLzurCCTFYu2K/02tqXlTxMU/CyJp77dLlncWdYW7FTzHsppOo+T9SbD9uvi59cNqXSE42/XZ83kWeo+81y3mHx2IwgnXDbOL8n8SdfmefdGgpLxunvf7thZJj55tyWHGw6/PyZRBLptx5PYpo9CdbVeUdAn0tOhT22jwguxewvM+y8L+4iDPifYZ8lFLMM33iW5nt/tXb3NguJQQT2nwnjRINPjmGpr9GVNqt+Ui3zXPyiTbL8iFEBIUAxdZ3/L8NMLNd+C/STPM8kv63pPl/VLOE399C37b1fUKz7VXiiD0KeupVaw2L9IYkd+vzAUlP9bup7VMwV5o6dL7Tm+KH2geeTnm3dFOB7JBaG2q2eGSXC5I9s+16RoA4bCMVk/5U1ZdMxL/4Vff+ORsK0nydfzfh3SL4u+8kNZbGF5j1LrnhPV7W6c2HXZMwHoZiV+G5NgNLtGxJsK5cpfaRpdWeulbYgFvC7sNdN+9FPOUsnkSyEBBLwIzYrMHR2S2nBT5jPTUQvJBU8UFYzyxinUUMTBBwuivoVL1sGKrl9flfbpEakP2p3CUh0gDWdwTKavORd5fchqW1rDdakbISy3S1FB3DW8XipBszD183Z9oZGn52UlO21kfVUWz5+Qs97upBrFsk87KJqUmZfNLoxFbwgSQWHk+JwsZ5+9uiDTRwy3EZm6lp5Wp+LnoWfa6U0dIm66/kJ3Qkenysx4eI2Y9QGNLQbWnVu9PisnXurSP08bgFpDZj5OE1x+1U4TM2/c/YZUPl6QdnZaoWkq3jwRvZ33BWBtiFN25rnIZ3/XGOh9qeeqPfVsnxpq+FOf5cc7vlKfuWM2XFTvFNUjccyjA3WDP/SgXfMyxTO6ortw0qHqNxXGurNrvSv1+dmCTD1i66L6Iydk9r1MvWMuXl+T5aszGZvuJd4az24FPH37eEha/lZZ5o3pqw0ZcnVnlmWnchuqw9eutuRQVJZPyFym/MnHs9LYX7ceDsGOVIpk+ovrDE4ePyH1XH1txMS61B5tStN4XfveeVEo25d+87jS8pdOVeab5mt6MyK9SG2/fmRaFrJmtb4sc8ftEhO9eCnk8nh9Reaftu1VXsQraBc1gkUDzltz0jDtbKieMzZ44URU/+v0+Dg4c6DC5TdPyInBTL1tzkceFKPSfNp4UoXfrqfCy37ZcPh9MnHx6K4dz0Zav/fRJ9JbA59qt6Y/cOJCZiOTdd3kJDuY6SMOawsyFa1nF3iOKT9vzsiluB8Q4LuSeMc3zvvx7Keca5kzy2UspPs6mb5lun40njDWI7zxw8DGV+sr0n6uJXNxOgLAvZ+Uvd9/WL3elsVA33zppdHIE715Jd1n84LzDgP8zNm++gEd+pO35mXG2+I9lCbz6H76ln23K/FLv4bMXM9WmoZDW6ZPz0mywYe+TKhJqM/st5c5AcGjnj3M1bPRBQV5E9/szvfUVrp7CgSQcDziB4Z3UO+ZYRJe7kjtLtQWlI7J+ihv+qxacV4ar+9adifpPsYjRbau6dfzqTpE49cri37yQ5/VU7+/t/Ju0+qcJB6clpnnTR89+zJViRR9bqD8laaxn7YheUkd7LvEY+H8C+t+ytnatWkZHazL2MtLRXD4XUQQ8IwZqLFH3gxJ89W7hRQ1MuWNkzVw33vDe7LGLVPZLJ93AszgARk71pKpZyZl7GBdasMtmX3JToXwO0AmxOXXJ9zC7XU5cGRSWmempDkxIkO1uoyemZWzuUVCzV0FcS+IVxxzPZ99++V1fs0GCVHcz9hdIEf2GzEsPX8+Dq/sQN9IuQbI78ambjMDOyPUmuu8Z1sG5tkFuw5qI2HWHTvYkKZZl+5kQw5EFcxZu9tglx2LTpWZiW+ct7Ukn0z+jg/buPcyBVkFU5M2y7olk2Zdu+GmzDxn7STVkLrpmEPDdkOUqTMtmTxywAoNwy1Z8NaE0oY4a2dFv2te6PnUc/u2Tw214DO2DZf+kw2ZVoFYbTRTtuKQ9PxW2bCGn6t3iuqROGbuwDTEViDLCbHRFRqO3aRh9NFRyf+fLZiurYO5DQh4Jg6fLclsJKY4oTvabMfFw9sxOz0A6CLe37rkBhoaz24FPBEzAIpEcrN5xURTps40ZcKUrcGGzL4a9mbsVG7Ddbjp9OlGQ/osrXNNXTPbU90R5756K5l6LGubRmyfb8abBIXe+G5f+kWkrPzFCQof6Nv12v4xtznTtGerYbatY2NyIHoRZdq0hcDUwvCzzK/BPPbaq7SIV9AuxsGrDefLz9rVqXgx/qHhCdueaPtt2phHplL1bBxkFEct7wF7V48ZYxd9ekPZaaGmrPYTfv9MojTGtmIHkoXtuA/EO47bTW3blWlJn8i7PXWobVTr1dlIcI3z6eSE2H5KTYaO5l8E9hWHm3PxplH1g6MSbaZzpil2nTc/Hwr46o60tbr4Il4/5VxScbHlbsql2dil9i2zbbfE3oBmg4kRlwZT14268tj91M2YvbdenP2tLgceceVF6+xoLdhuPbwL+Gk73Ec/wO9PjkZtyZQ0H7f9JZ9RKE2RwfXVtwzXfbYfW96uJDZRk9imDUtti+P1z1xx8F7CJfnq2kuzA7JrL9Ptd6oo5b4E69micUZ8d3F9Gl+SO1APMu23t2TyZCJQhuORBBJu0/3+Q5cMkyBzR6ExmW3DysZkpl+1IK1I+O+yvKmNPzMrs6bPGJdRl5emzSjY0Tqu07ocjxTauku9nvfLhznVL4uebVpZbGF514zWpSFM/7zXJS2iMELlT9u1svLXIY0JM+2Xdt8HPnve9J+TsWmnPlc/5UzLXm2/3+dTqnwqAQQ8Q0KFnf0n5FLgLZ/C6vxZ1MgUdBxcgGqsWSEkOl1YENdk6ULLDjyjyndEJk7PydJniddDKLzV63PSikQ7K5YZIe/sFSNaFg2EC+JeGC+XKD0fGGCK8bQzcddOQySMjcr4sbMy/0HgzWBH8Br3sqlJLhDdnOExI1xaUcF2mmekXfaGeKUtZ+PBoW24W2+aIUZRnocj3aky07tsPmnn1+0y/NiknH1zKf1mXG8o/MzYSTS4mYm8VoIN6e1FmT09IaNOLDSCatRpfrmd85jU+7N2VvS7RlHPZxtwc753+9RQiz9X2tNuMGQ6cmMyo+vlqI322pCbR22GDRfWOz3YlBm8OTvO89RwnHjmrosE7Pg4LzBYklqmis4X8w6dWf1gXmZO+nZlO55m0DOTs+ku4h3XKxpPf6AbikH6t5X22USYjsrEWWmbarBgPcFO5basDl+54j0rEg1bMhd5QGg6e2WcDEyCnqWe5+1sQZ22rekvKn/pLMl/izx3zIsSbatmJPs+duXqrLRikcDa1PixKcc3H2TZL4V57Il4iWjj8i62w2zIHfL21oKtZ81Lt6gsDsnIY5MydSHgweQFHQ8IMtNU7CW6JlDAO88Lo+xwY+FvkEncB+miHQ8mItPWmUFqF32iUFDaRkVtm/FM8vtMj0xIqzCf+ozD6pLMv5C8pIt2xs619yV8jUdvVE7SLyF7LecRi9VFmTudCJWmzZx8wbb/yiXf1pg2cUXaqTSYftKoTBTsYh/ibn7TZ/j9CrMO79lj4zLaY3lJP6OA3wb7AUtvnpVJrz8ZvfzM7JAbSlMctz77ln23K7dMX3Y8FqLtLuwT0jq/kOvjRXF0+Ro5CJi6ypWrRdNVL1pEP05c/iBczxbkTXy7O9/li/L4NvMC8eSY6+sPiZn5oyOMcDziO8MeeHq6V4Z6X+Cz7z5vL+XNt/GoXfXLtymjcxLlZyB+5qdexiOltu6V71Ad0jeLXvLDZxFKr54Pte2B8WOovMfBxi9bN7AsUj/lT9NQNLYx+bCBPrCZCdZUxw7TpzVtVaQjxCmPD/opZ/YlZ11GX1iMw+EgT2CPCHj5hPNLlkAyEPYXoMxeVc3vLu4FrvLVjDOx6o3AbrbP3lLK1RCAAAT2HoHqtOOdBqF7L29sipVLL1PT9yor0g2ByhDoQtCpTFzvpYiogBd84bYJCXUCei+zsvp9aichrt9wua9/Agh4/bO7t+7U6TdmUd7dljI39cYs3M7fPUpgN9vnPZolJAsCEIDAphGoUDuuQpXvBbZp6dy1AekGCrqz4q5NCBGHwN4igIC3I/mtHu0hb8PNiJBu/jF5Uf1KNyPUcBgIeGEuO/krAt5O0q/Qs1cvTERTeXajCGYXv96Ai3KF8oGohAnsZvsMp4hfIQABCEBACVSpHUfA01zxPm/PyYSZQml2Os5uAuZdxiEEIFAxAgh4O5AhbhOL2oTMbWhproKo66ZTofVqC27ZyM8IeBuhtzX3IuBtDdcKhrogZ781G9zJa/XqtIxF66dkdrWtYCpyUXILONdzuy3mruSHShO4R+2z0syJHAQgAIEKEKhYO75XBbyFlwp2C7+9INNH7HqNjVd73V6kAvZFFCCwlwkg4G177utGESPP9z+nbeVCS6avLMta9oXJ2pLMuc3hDj3T7mmzrn5BIOD1S27r7kPA2zq2FQtZF4H1dvLydjat1UakOb+RHXi3M7krMndsQprRTrDmjfAuFB63E9eueNa9ZJ+7AjiRhAAEILCDBKrbju9ZAe+0v7HXlES70HsbNJhNAFayg8kdtCAeDQEIdEEAAa8LSJtxiXnRMSnNY27jlOFW4Y7y3TxN2yGzccz4sZZMnfF39q5J/fGZaOPKbsLa6DUIeBsluPn3I+BtPtPKhrj63pxMpXby6m+XsJ1P4IrMPe52Jny8JfMf73yMiMHGCdw79rlxFoQAAQhA4N4mUN12XAdOe24NvNuLMndmUsYfOSB13al8/4iMTrRk9upuecF7b5caUgeBngkg4PWMrL8bFmTK7QY+dmxGFja6NN3asrRfbsrEoyNuJ2XjsHJARnM7lPcX217uQsDrhdb2XIuAtz2ceQoEIAABCEAAAhCAAAQgAAEIQAACEIAABPoigIDXFzZuggAEIAABCEAAAhCAAAQgAAEIQAACEIDA9hBAwNsezjwFAhCAAAQgAAEIQAACEIAABCAAAQhAAAJ9EUDA6wsbN0EAAhCAAAQgAAEIQAACEIAABCAAAQhAYHsIIOBtD2eeAgEIQAACEIAABCAAAQhAAAIQgAAEIACBvggg4PWFjZsgAAEIQAACEIAABCAAAQhAAAIQgAAEILA9BBDwtoczT4EABCAAAQhAAAIQgAAEIAABCEAAAhCAQF8EEPD6wsZNEIAABCAAAQhAAAIQgAAEIAABCEAAAhDYHgIIeNvDmadAAAIQgAAEIAABCEAAAhCAAAQgAAEIQKAvAgh4fWHb7psWpFWrSa0xKyvb/Wiet0sJOJuptWRhl6agm2ivnG9IrVaT1tVurk6u0fsa5ylRCRWOILDNBK62ovJbO30v11LbzJTHQQACEIAABCAAAQjcswQQ8DYla9dk/mQtGohMXlwtCXFFZhvmunq54HB9Wg6lBDsEvBKonAoSQMALYnE/IuCV0anAuVtzMrG/JvVHpmVhrQLxIQpbQ2ATBLzVD+bl7LExOTBo2+D6wTGZfKEtK+tbE2VChQAEIAABCEAAAhCAwE4RQMDbJPJnsClEAAAgAElEQVSrFyetJ8HJeSkcb65ekkkjzNVqcuj5xcInL/1wNLpm9IdL7prtE/BW35+LBkPTPXo0FSaGEztEAAGvDDwCXhmdCpwrEfDWbrZl9nRDJl/fJO/JtWVpn29J45tz2+DhvCpLF87K5OHpe9oztmsL2qCAt3JhQuqmTd0/JpPPTMnUM5Mytt8JeY+clUVEvK6zggshAAEIQAACEIAABKpPAAFvs/JIxbnBliwUDBrW5ptW5DMDjkdnROW5dBTUS++QTF/XM9sn4C2ctoOfXqckakz5rAoBBLyynEDAK6NT7XObnne3ZqWR8njeyvTvjXLZNcGNCnjnGzL2/IKs+m3u+rLMPl6P2tpyj/iuY8mFEIAABCAAAQhAAAIQqAQBBLxNy4aQ8OYHvibtp82goiGNaBptQ2Zv+efdcVAIRMALkOKnUgJ7QyhQMadXwVnvYw28UiOq5MlNzzsEvJ3L5w0KeKs3l8Me723W1tu5TOXJEIAABCAAAQhAAAJbRQABbxPJ5qe+eoGvL0jLrNHTmJV5N0W2OR+YbHu1ZacEpabiIuB5JDnsigACXhmmTReByh7GuU0lsOl5h4C3qfnTU2AbFPCKnqXe7o3NmmZd9CB+hwAEIAABCEAAAhCAwDYSQMDbTNi5zSe8wN25aF07d1wP7Ly3+PyhwNSftIC30j4rk0cOWKGvNiQjE9PSDnnzmcffXpL5FyZl/BG9viZDwxPSurCYmnakU2fN+nzZ/568m24tyOzpCRk9aKcw1Uz8HpuUs1cDm3usurgND8XPHBoel8kX5mUpcLnoQNtwW1+R9guTMuaeM3S4KbPXk5uW32zJhIa7f0QmTs/Lsj/NymWNigFRGlfa3mLoQzJ2clYWNcj1ZZk/PSEjbn2liOGby14Ge4dry7JwviUTj47IkONZtrC6Ze88Mm/5cajLgSOTcrZdstZXdP14HK/kOT0IeGttaRpx+cFpCa3MGLZJm14dKE9cUFCOw/qKZeDZXf3gqEycnpWFkK12GsgXnE/ln5cF0eH6qixeaMlEHAdTVloy9/6a6H3deOAtPmfLZPG16l17SKbfy0SiRxvvFK9O5zNPT76uLsqcXy5NmXjOLPSvnsNZj+B0nbN8oenWFhuS6Wsu2PU1Wb46K62J0dj+aoMHZOzY2XR9dHtOJkw5MPYVKINRaO/ZjXvqT7edR1X6+eYaTXu2fjLfU3nTbfnT+iRQ54V2/F654te7rmxeKSmbCX0R0fKYr19rgZ2i7cYMSbmO69E3l1L1tohne/GSC96DtU365iXJlFCRtXlpmrQfnUvOdcvOPSJVd308J83Dti4fei69q2zE7jGtD716raBcR8Fnbda1JTPXAy++vCTLZwvSGq5JbbAp85/5JziGAAQgAAEIQAACEIDA7iaAgLeZ+be+KNMPmgHapFzKjJasd55b10698XKCyZLMPGruLx5Mt883pD54QBonp2TqTFMaKpQNNmQuJ4zooNEOmFpn0ot8j5xZiKcfLV8y4U3J5GE7wBw7Zr+b3y593B2k5YsnZCQaDCfPa+nugBmxcu29aRlzuwYaMaxp4nam6YluEzJ3M/NcHXA/MxutcTR0eFJMmpqPqzg5ItPX12ThudGEkb+o+eP5RepVFGi9OiuNwSEZO9ZKcx2elsXPFmT6kbrUDzaieJo0WWGuLnkPDxVErFCq6dJ8qgfiEA+C2yYOdTnweDPKiyRdQ9K8kh+0rl2dktGIYZ730NNNORHlRauLxfJ1F+VRmfkgw1zUJmtSS3mF2usWTttp4anp4DcvyQkzgK7VJBIUzeLyZ1qJ6Dw4KlNXM+kpG8ibRxWcj/Mvu+nK+rLMHXXCsBGrXHmJRN3BcWmeHI/ilxJ+sknX7x/PyJhh2ZgNb3KgAujhGfEl3X5sXNNTFK9O5zXKqc+bxq5sfsRl7aQVo+uPt6Q1Xl7nLF6cdC8LbBgq6GtcajFfryym6iO1r6Ldtz0RKhZA8wLe6tUZVy6soHrIlRNTR83ELwh6KH+rCzJj6p2nG3bX7wdt+TbhTb28kIha68ty6fhI2p7jeqUuo88l9WiKe+rLslyK6rhJa0u1MZmMvpuyccmzmzVZfH7Mezlj64Ipl1+mTA0dnUu9jFARPdn0KHmweoXXak3JOXw7b+/E1npg5x4R113vXZJJZ2ORwBrX92uycGbUpicSd039qnXBkDSfPmFf3sTXu4Bjm9U6eUqaE6PRTrNJfJN0xkfx+ncj0srWMfFFHEAAAhCAAAQgAAEIQGB3EkDA2+R8s4JGTdLTY93AKN7gQge0GcFkZc4upp7b4MINZgfrUh9uSdsXB9dXpf2MG9DmdrZdkLPf8rzINK3rS3I2EuryQqMdkNVEB+l6S6fPtWtTVrwbPiFzWcFvdVFmLngeGbfdYG9wTKav+YmxT1m+4IRAI575Hjsq4A3WZfR0etC8bIRNI7IMj8jIYENmffFPPTJq/sYg9lkqQtQHR9MDvnggWJOR4RGpPz6bHjRfbdn05kTYFZk7HfCIXF+V+ZNG7MrHwTKvS31wRFqpzBVZfbNp05W1iZhhQ2bez4hhqwsyfUQ9ILsR8ER0F+Xc4DgSr+oytL8utdh+1RqcYO3HzYjYkXg3Iicu+HKWvWf1mhNuByfl0m0Np1igi6/oUcBbfN4KLiNPz8uKb0MiEttX1nMrflj2YFlmovIyJjNZ25aE3djLXnrj/OnNxtUec/ngotTpfDbm4uVHcz7jLebZeOFLg/EJmRgek6mAp9nKhZZMB35Xm/V32l67Yu045HUsQQE0L+Bp2soZ9F7+Ys/eAoE2tqXT7bT3W8zPvDjQ2HX61Jcq4XJpyqGpx+pHpmUhWzUaUdoJiSN+Xa/rpubib+22vn9I6rW8eGo9a/02qHd2tu4al4mjIzJ2xnh0ptMfp+fxGVnKeMPFdYGpt1MCnraPh2RKvT012PUVWfwgC8ad1Pww7cp7mTpR7+cTAhCAAAQgAAEIQAACu5gAAt4mZ556Q9TPeIKVDrA8D6aQYKL3+gNfGz0d9AWm6JkLPpiR0TIPoUAa1TMjK9T1J+B5AkfOgyv/cJ2SmZt2GV+qHjn1tOdZLOA1pZ0dn6lXY4Eos/yq87jKrImkYkAydS+OROT1FYmCOY9Ic82yzDrPpbmMLuKFkDrU/M2KM8p87IehfYnVA64h/nM6MvzAeY0FpuelIqVf1EYzU+0iPoNNmTs/mRcfnd35nj9q14eeC03GtQ9bfnks8rpJCV4FAp1GrycPPE1L4ZRNIxBYgTObF/HzMgerFybycY6uWZW5o8YzbULmPEGyY/7EUx/TNq72WBSvTucz0Y6F2cL8+GzeTp/O2bjWOXXpeSdPnZrpC0rqnTyYL7taLtL1Qb8CXpZA8l2fk2Or9YofX71NbenwjCxlxKnoEjf1N19nawDZT+UaEvDUgzttS6kQVOxMcVyVS980Nph5IROlqy7N12dlslaTdBxdveKL76kHpb8UsdO6q56pN+zdndOz9ENbF6QFPPUEzKQnHaXMtzX3IgvPuwwYvkIAAhCAAAQgAAEI3EMEEPA2OzN1wOcNjHTwkxoI66DRG/hY7728h1a8flLO28tFfr0tLSPgjc96U7G8hN1ekoUrc9F0MTMNaTReEyzvaacDsqyw54WWP9S0+Gsp5a9yv6ggFZjS5d/jpnelhFB9jieEJrfooG887X2nF7hdCbODdxVE0h6T7iZ9XgHX9jOhqYfu3rVlWbwyL7MvTEnr2LiMeuvhZeNgmYfy3YZln+OfV/GwjGGZUKBQ/E/l54fpfjO83Tpmvlhn2fnxUs8Z36vHf4Y7Vq7+dOLNFPBcXvtxzcZC8z2bF9nr4u8qnGSmyYrjkhaA+7fxTvHqdD6OrzuwtlOWH5rvBdP2Q1Mv/YeYdfDea8v8+bMy9cykjD/qrYeXEcTsWoJpwVJEBdCsWLNBAa+H8lfmgad1d6GdhMRKn0/uuKRc6ouYYP2WBGTbiXrKO03twq/H7G+mPDvGXpukaU6Leu4ZPbDT9sJ/bhzTm7MybtqlsvQUlPvY6/H4rCx6wngcdvbATXMvFKqz1/MdAhCAAAQgAAEIQAACu5AAAt6mZ1p+QBxcJyxeW0wFE+etkJumaCJYPJi10S8YFJq1m77l1h8yA6nBAzL66LhMPmPWqrLTbrNCnQ7Isr+XYnKDsDLBJLnfxdUfTCYnkyMVefzBn/6Wmm6lt+S565no08UxOxDXgW8wvfq8jBCh4VpWWeFjTRZfGI83r4gWn3901K7Bdsx6m2TjEA5HnyKSz5NuGBbYRBJs7siy8KbaOXHKDs4d35iF8/pJeblpHqhN5x7hfgjEv2AgH4dQcD6Uf/pbUFRwAeo12byInxc4sOU4LYZZb8KsZ2wgfYHwVETxBY5O8ep0Pv2YbvJDr8nasUtDgXhtnrP23lkZd5u6RGuzDY/K6IRZs82t8xbbiouVE6hSYmdQADXXu+dnw/A2swjnXe/lL86HkmdF67qZOrToP3BvOi/0m0tXyDO2y3pUbSBl366uSqYoO9HO1Z/2niSPrads1m57Z2frpoKXJt2kp6Bcy2dLMqNrWNbMuqAtmb1a4urswgnbhLLnEwIQgAAEIAABCEAAArubAALeFuSfDrAijzud2hkQrOw0OyeYlHorFA9mbfRDg0KzeLhZB6wuY2fmZSnjxaBxzApXebGoC0Bu8DT+qrcGWOFtLq4lwkB0q4pnz7STkPS3Cgt4K6/btfhGjs/Iws3MPN+CQWbfAl4pw5BNJCiDR05gUa8cO8hPPKPstGv33XkeJYKBCVHFoJa0Q9MN44cGbKBoIK/3FJwP2XH8m2c6Gox+6jU9Dfhz0yXd1PGUiGmeEEifPtj/VHv2bLxTvDqd94P388Ob0J++JM6zRNyxF7g0FAlTt+bsxhjDJ2Tm6rKspfK76F6dap8IvDY9WSHJxKAojGQ32lDe9VP+uhHw/E19ok0u4g0o3GY//qYXGcLpry5dJQJep3pUbaCVsm/n9ale2s4TPPb6dmXbfneespmXRf2wK627umkXCsq1MrO78eqmQXZtwMXMWnrRtTkBU0PgEwIQgAAEIAABCEAAAvcOAQS8rchLfyrU9eloh0MVRVKPc9NEzTldOywecKUuLB7M2stCg8IFmTK7AuqALhWeiK7RtSkCnkuj70mUeZz3VddFSgbx3snk0GMT/6iCR2UFvBWZezywFpVLgOZxVngoHQQbKeO09fxJ8srlba1EKNNp1SGhIAaaPfDXxXKDfG+Kt7h8jjx/ouP8dEi7FlfaSy37lFgw8cPuMJDvZQ08I0QYT6kyISReE/F8iVdPLuKeYGfOOR6ptfyie/q3cRVnsjaiUel0Xq+zn2qPBR5S0UU6Hbs3AU8ZB+srXUYgIP7pWoL2PsczOy05ildxnVfMQNObiM4+j6LyF9tjKL4XzdqPtcBu037IvRyH6mp3f5f1aNFSC1Zgt2XPTv31OTibjDzy7HHKE1L6Y1dad12bshvweAJ1jpSb7p5eAy93lcjassw/7XYCPn4p2SE4cCk/QQACEIAABCAAAQhA4F4lgIC3JTnrBkuDUzLzw9HICy4RX7wHxusnzcjMyZL11Eq8UWxogUGhil2BQamsL7ldNTdpDTz1Msws5O+lNHVo18KqSXrRev+SNVk4bab4ZjxzNE2VFfAC+RAnq3jjhNJBcFDA07Xm6pndjuOHiRnA2w04QovlJ9dlj6yw25DZD+alWcvkkctn43VnxYK8CKsCTdlaVCrApPJfPVBD9ioiajPZgb6GlSpfKqAXbTzg2X+RUJblot/t86xd2jiFNxzQ+KbSqIFEnwU27gSNtGej3qjrxdWk23irUJ8XGV2Y8WYnvQl4eVFZ4yiJ7YXy0k2ZrRnx1uVTOC39CHj9lb8yAU/c2mq1rtb3TBgUH5XEUTf6KKtH1xak9aB7MZPyekwE5cb5JZk37UkmzpHwZ7zu3rebHqWm4GobExT8+6y7tH0bbMp8yGtOknCz5TrIr6+XEsGQ+BECEIAABCAAAQhAAAK7kgAC3hZlm66XNfqoEebyQod9rO4eOC7jZkfTwDRbe13xYDZ1PjX4Ui+tRnpTh/VVaZ8ekfqg3YUzJXx460sVDvgLeOn0q/qRaVlYzVy0uigzr3uT+HT63eCYTF/LXiyyfOGEjNRqUvc3OTBBVl7AUy+WQ9LKbJO7/GpDhhzzrGDRu4Ansna1FXl21oZPyNzHGd4fz0pjf70vAU/cNNHJ4yekntuZ1HgD1qX2aFOaxtPQ96DTKKwtyNSwsfkROXEhP6V69dq0jBnP0OEpWUjNMFavtUPSupK2idV2K7KHaP2xjHgbFPDiaaF1GXt+QVZ9oSNj/9m80GQUfjoB6tDzMzJthBR/jUb/pn5tXL3Xag2ZyeSrsSErynYv4MVlJlTWVtvSGq5L3eRHLq/L6xz1wDv0TFtS2ejbXkjAi3ffnZTp58zO0GEBtGwKrXpjpj3IDPz+yl/8rNTOrpqZpo42dWVdGj9cTNuSuWR9RdrPtWTull7f6VOnmY/l8tfcWVqPri/L3HG7LEIjs5t29FQVAL95Qk4MBmwk8moelebTxkPV986Lnuy8hzez7lKRuiYjx+dk2S+HZh9vr05MC3jLsnAlOy1bRD6blxNmDcKQx+bagkw/Upf6kYLdgjtlC+chAAEIQAACEIAABCCwCwgg4G1RJunuhZHoEBI63HNVgDDXBafZRteVD6bjAWhKwBNZPu8G/IMHZOxYK9olcuxgXWrDLZl9yU4zzAp4scdJbcjec7Ih09e6gWTW3NMNM4Zk5LFJaZ0xO7C69Ysywsva1SkZjYSDmgwNT0jTrCn1zKRE8TPi3SNTspD12qi8gGeFNSM+RptXRAv6N2VieEhqgw2ZfbVlp+Nlpm32I+CJZHhHz5qS5sSIDNWGZOL1WbszccYmOuZk7E1Zk1pAgPHtutC77OacTLjNDeoHx6JNU6bOOA6Gzf4JmbuZj4mZ4mgFKrNovdkMwWy2ckDqg2My85Jllx7oJ+uh5ew4FAdnX8a21P57FvBiAcrkcXYKcTpNfdm48TZ83k4VNDYUldszLZk8PCS14abMPGfLbS/xXn59wm2qUpcDR2y5tHZSl9Ezs3K20buAJ58tSCsSapPyq2E2zjvbC9iPIbR2xXiHWuEwL8Ipw5I6T3cEjjY3aEb1RvOCnQpthO1ey58pS+2n7QuN+sGGNA3vk3MST6720lrbP2I3pDG2OTEqB5z4ORdfrPEv/rQbn5hyYMtG8/FpSV5vhMu1qUfts0yeLaRFU+9R9sVRKD8NeOtVG7VJGe88E0Q/7DrVXcZOph6xbBN2ri4w9YCrE9Pl2uX9/hEZN+2Wa0ds+kekdTUlGdvU6xT82lCX7ZUHjUMIQAACEIAABCAAAQjsEgIIeFuVUd5gqXSwrdP9at7un7k4lQxmo2vd+ZxYsyZLF1pWQIqEkxGZOD0nS5+VCB/GC6Q9LQ0j9Bkh7eCYzLyfi1DhDytXZ6UVD2zN/aMycXpWFkIeKrcWZPb0hIy6Z0Wi12OTMnUh4OlinrgLBDwTzZX2WZk8csCKUZF4elbaZoDvBplZe+g0CLbn89Odo2dd8Z5VG5KRiZbMXTcebEU2UZh17kQiZgS9MD0PsdlQnmrwq0sy/8KkjBvxMhI063LgkQlpvtyWlYwnjt5iPlPsXHrmjSeaDtAzQrAK4DkBzwS2uihzpydkxBcTX7DP1/uyeeHHpfDYrc8YrS9Zkpbo/l5tPLopU24jG5qJPFv7jffq9TlpReKuFR6NkHf2ijFK9QjrbQptFM2Vtpz1hKUozMjQO9RX6imWnSKfAl4extr7s9I0oqar15pvJl6bKRvqovxFj/1sSWZP6mYJQzLy9Hx6nTXjaZeyZyNcltRtqbRkvkRee41YkDtgvMYyl2TrUSN+jR+bcmU7c7H31YqjBV5qoh7fAe88F0av7DrVXVGwjp2+nImEvNNzsmiyLFiuV2XxfEsmHjUvI4y9unbk5Iy0i+oc9cB75KwsdiqTHi8OIQABCEAAAhCAAAQgsJsIIODtptwirhCAwM4TcAJeUODc+dj1GINEwOvFi6zHh3A5BCAAAQhAAAIQgAAEIAABCGyQAALeBgFyOwQgsJcIqIdi1mNtlzJQT+GC3ap3aaqINgQgAAEIQAACEIAABCAAgXuOAALePZelJAgCENgyArqLamANsS175hYGrLsGh3e93cIHEzQEIAABCEAAAhCAAAQgAAEI9EQAAa8nXFwMAQjsXQJmV02zwcSITF/fLRQW5Oy3ZmXxdj6+q1fdjsBmB9rApiL5O/gFAhCAAAQgAAEIQAACEIAABHaKAALeTpHnuRCAwO4gcG1axo417U6wtZqMnC7eBbR6CdLNTNwmIm6352RzkRFpzvewhWr1EkiMIAABCEAAAhCAAAQgAAEI7AkCCHh7IptJJAQg0DeBa1N2R+H9YzL58oKs7rJdLlffm5OpY+Pebs8b2EG1b4jcCAEIQAACEIAABCAAAQhAAAIbIYCAtxF63AsBCEAAAhCAAAQgAAEIQAACEIAABCAAgS0mgIC3xYAJHgIQgAAEIAABCEAAAhCAAAQgAAEIQAACGyGAgLcRetwLAQhAAAIQgAAEIAABCEAAAhCAAAQgAIEtJoCAt8WACR4CEIAABCAAAQhAAAIQgAAEIAABCEAAAhshgIC3EXrcCwEIQAACEIAABCAAAQhAAAIQgAAEIACBLSaAgLfFgAkeAhCAAAQgAAEIQAACEIAABCAAAQhAAAIbIYCAtxF63AsBCEAAAhCAAAQgAAEIQAACEIAABCAAgS0mgIC3xYAJHgIQgAAEIAABCEAAAhCAAAQgAAEIQAACGyGAgLcRetwLAQhAAAIQgAAEIAABCEAAAhCAAAQgAIEtJoCAt8WACR4CEIAABCAAAQhAAAIQgAAEIAABCEAAAhshgIC3EXrcCwEIQAACEIAABCAAAQhAAAIQgAAEIACBLSaAgLfFgAkeAhCAAAQgAAEIQAACEIAABCAAAQhAAAIbIYCAtxF63AsBCEAAAhCAAAQgAAEIQAACEIAABCAAgS0mgIC3xYAJHgIQgAAEIAABCEAAAhCAAAQgAAEIQAACGyGAgLcRetwLAQhAAAIQgAAEIAABCEAAAhCAAAQgAIEtJoCAt8WACR4CEIAABCAAAQhAAAIQgAAEIAABCEAAAhshgIC3EXrcCwEIQAACEIAABCAAAQhAAAIQgAAEIACBLSaAgLfFgAkeAhCAAAQgAAEIQAACEIAABCAAAQhAAAIbIYCAtxF63AsBCEAAAhCAAAQgAAEIQAACEIAABCAAgS0mgIC3xYAJHgIQgAAEIAABCEAAAhCAAAQgAAEIQAACGyGAgLcRetwLAQhAAAIQgAAEIAABCEAAAhCAAAQgAIEtJoCAt8WACR4CEIAABCAAAQhAAAIQgAAEIAABCEAAAhshgIC3EXrcCwEIQAACEIAABCAAAQhAAAIQgAAEIACBLSaAgLfFgAkeAhCAAAQgAAEIQAACEIAABCAAAQhAAAIbIYCAtxF63AsBCEAAAhCAAAQgAAEIQAACEIAABCAAgS0mgIC3xYAJHgIQgAAEIAABCEAAAhCAAAQgAAEIQAACGyGAgLcRetwLAQhAAAIQgAAEIAABCEAAAhCAAAQgAIEtJoCAt8WACR4CEIAABCAAAQhAAAIQgAAEIAABCEAAAhshgIC3EXrcuw0ErkvryZPy+5dXtuFZPAICEIAABCAAAQhAAAIQgAAEIAABCFSPAALehvJkRc49e1Lue/KktBY7BWSFqPuePCdvd7p0m87fuvwnUdxN/NP/fyLndkgvs3HyGSHgbZM58BgIQAACEIAABCAAAQhAAAIQgAAEKkoAAW9DGZMIePc9+1dyqySsRCzzxamSG7bhVF4scw9dPGcFvQ5pKoziyl/J7z/ZnwiYjxMCXiFnTkAAAhCAAAQgAAEIQAACEIAABCCwJwgg4G0om52A9+yfdBCs7HW//6zxeNsFAl7ExHkMvnK9d0IbEPDyD0PAyzPhFwhAAAIQgAAEIAABCEAAAhD4/9l7+yc7ijNB94cN24H/A8f84JV3I4x38MfEzA/33pWD2FnHehye3UVLhGci7twbxhu3JYFsIYQQIBkIbAyy5A/2IvDwoQ8jMCCPByHJxhjxKSRZDBiJsbA+LMs2yOijBUhIQrSk3Hir++3OzKrqrqyqUyfr1FMRTZ3Mk/nmm09mqfs81DkHAhDoEgEEXqXVHhN4Dzwz+lbaPNmV3NH2oHkwectqWwSeMem74QrCQuAVBEUzCEAAAhCAAAQgAAEIQAACEIAABCAwNQEE3tSMJmmhAu/Xk8iuIm3G7nazPosu9aUN41JsLJ73xQ4Tb9Ed/Ty7VP+MWUwp6JIx05/vlz/WRG72Z+pN5JLxvPc23XROWXfgpXlN/RmEGQCoggAEIAABCEAAAhCAAAQgAAEIQAACLSCAwKu0SBNyzpgs0WSMGRdvOXe0jX3e3ITk0j6Lzb+x7+gbiyNvw/W/YGLrAyLtrDv7xsSbEzNjnmlZ5jdKz6nQWNacnYj/+qD3bbJjIs6aZzonPwe/bIz51wcLfImIkwkFCEAAAhCAAAQgAAEIQAACEIAABCDQGgIIvEpLZQu8bEGXCK+xu8zScmq0f6ZoS8SeJevypNxYvX8HWnqs9ESnbuPJsqJj5Qm8dAqjdy5ad+Glc8rKweKSEZMqCEAAAhCAAAQgAAEIQAACEIAABCAwSAQQeJVW0xV4o3fb2W85HZVPKtdScmpS0ZUlruzYo4knMS0BNj4dXwCOPzHxIJXPxFNjj9wcCo816byMGb2Lb/StvqNvtZ24ezCdk5uD3uno3HGYypsKCEAAAhCAAAQgAAEIQAACEIAABCAwOAQQeJXW0hN4ZkxO2XfcWXItJacmlWyeuEpHb/sAACAASURBVMqRYmkZZouxye9US+Xjs/DGLDyW10/Djo7nvjXYz8Evq7Bz71Ic4z72mYEqSHUczhCAAAQgAAEIQAACEIAABCAAAQhAYJAIIPAqrWZa4Mnnsf2ba0WceQLOZLzFNkd0jabk9c9pmwgvSxKGTCcty9zefmy/7La2Slm5JnU5dxBan9+XzsnjYA0jD0fbL/Y+W89rRBECEIAABCAAAQhAAAIQgAAEIAABCLSYAAKv0uJlCDwzWidfNuG/zTNITo2LwLEEs6SYPOW3C5hPOh+rcxLXE25Fx8rKNVPg6Z10IW+htXIce5jcGWh9EUa6BTUQgAAEIAABCEAAAhCAAAQgAAEIQKC9BBB4ldYuS+CpVHPfKirDZAqzMVHmvEV0THal67LeEpuWYMmUDj9jbtl0eNLZZeajbwO+1pN3o0HNg9/xvvFW6lNjZd01N1pnf7Pu6PhuvHROXqxCY006bZ6EAAQgAAEIQAACEIAABCAAAQhAAAKtIoDAq7RcOQLPiHRKy7a0nBobfEzYjX6hw+hn2KU+1y3rrjYr9/Tn003c1WY1cx5OCDT7c/PS4tHpZAm+iXzTY9mxx0WkN0+p95n45dRn4HkxJIfx+H6ilCEAAQhAAAIQgAAEIAABCEAAAhCAwAAQQOANwCIyBQhAAAIQgAAEIAABCEAAAhCAAAQgAIHBJYDAK7m2N/1ik5m4A827g23s21F5vlkutz39bMnVpBsEIAABCEAAAhCAAAQgAAEIQAACEIiXAAIv3rUhMwhAAAIQgAAEIAABCEAAAhCAAAQgAAEIGAQemwACEIAABCAAAQhAAAIQgAAEIAABCEAAAhETQOBFvDikBgEIQAACEIAABCAAAQhAAAIQgAAEIAABBB57AAIQgAAEIAABCEAAAhCAAAQgAAEIQAACERNA4EW8OKQGAQhAAAIQgAAEIAABCEAAAhCAAAQgAAEEHnsAAhCAAAQgAAEIQAACEIAABCAAAQhAAAIRE0DgRbw4pAYBCEAAAhCAAAQgAAEIQAACEIAABCAAAQQeewACEIAABCAAAQhAAAIQgAAEIAABCEAAAhETQOBFvDikBgEIQAACEIAABCAAAQhAAAIQgAAEIAABBB57AAIQgAAEIAABCEAAAhCAAAQgAAEIQAACERNA4EW8OKQGAQhAAAIQgAAEIAABCEAAAhCAAAQgAAEEHnsAAhCAAAQgAAEIQAACEIAABCAAAQhAAAIRE0DgRbw4pAYBCEAAAhCAAAQgAAEIQAACEIAABCAAAQQeewACEIAABCAAAQhAAAIQgAAEIAABCEAAAhETQOBFvDikBgEIQAACEIAABCAAAQhAAAIQgAAEIAABBB57AAIQgAAEIAABCEAAAhCAAAQgAAEIQAACERNA4EW8OKQGAQhAAAIQgAAEIAABCEAAAhCAAAQgAAEEHnsAAhCAAAQgAAEIQAACEIAABCAAAQhAAAIRE0DgRbw4pAYBCEAAAhCAAAQgAAEIQAACEIAABCAAAQQeewACEIAABCAAAQhAAAIQgAAEIAABCEAAAhETQOBFvDikBgEIQAACEIAABCAAAQhAAAIQgAAEIAABBB57AAIQgAAEIAABCEAAAhCAAAQgAAEIQAACERNA4EW8OKQGAQhAAAIQgAAEIAABCEAAAhCAAAQgAAEEHnsAAhCAAAQgAAEIQAACEIAABCAAAQhAAAIRE0DgRbw4pAYBCEAAAhCAAAQgAAEIQAACEIAABCAAAQQeewACEIAABCAAAQhAAAIQgAAEIAABCEAAAhETQOBFvDikBgEIQAACEIAABCAAAQhAAAIQgAAEIAABBB57AAIQgAAEIAABCEAAAhCAAAQgAAEIQAACERNA4EW8OKQGAQhAAAIQgAAEIAABCEAAAhCAAAQgAAEEHnsAAhCAAAQgAAEIQAACEIAABCAAAQhAAAIRE0DgRbw4pAYBCEAAAhCAAAQgAAEIQAACEIAABCAAAQQeewACEIAABCAAAQhAAAIQgAAEIAABCEAAAhETQOBFvDikBgEIQAACEIAABCAAAQhAAAIQgAAEIAABBB57AAIQgAAEIAABCFQgcPr0afPAAw+YOXPmmKGhITNv3jyzceNGc+bMmQpR6QoBCEAAAhCAAAQgAIEJAgi8CRY8ggAEIAABCEAAAkEERNLdfvvtZtasWebuu+82mzdvTs5SXrVqlTl79mxQPBpDAAIQgAAEIAABCEAgiwACL4sKdRCAAAQgAAEIQKAAgQMHDpgrr7zS/PjHPzbnz59Pesj5hz/8oZk/f755/fXXC0ShCQQgAAEIQAACEIAABCYngMCbnA/PQgACEIAABCAAgVwC+/fvN3Pnzk3uvLMbrV+/PqmX5zkgAAEIQAACEIAABCBQlQACrypB+kMAAhCAAAQg0FkCJ06cMLfeequ5/vrrze9///vkLjy5K0/Ky5YtM/L5eBwQgAAEIAABCEAAAhCoSgCBV5Ug/SEAAQhAAAIQ6DSBd99913zve99LvsBCvsRCfuRz8U6ePNlpLkweAhCAAAQgAAEIQKA+Agi8+lgSCQIQgAAEIACBjhEQSffd7343+Qba+++/32zdujX5EovZs2ebe++9l2+i7dh+YLoQgAAEIAABCECgVwQQeL0iS1wIQAACEIAABAaewBNPPGEuv/xys337dmeuIvKk/umnn3bqKUAAAhCAAAQgAAEIQKAMAQReGWr0gQAEIAABCECg8wTOnj1r7rjjDvONb3zDyNto7eP48ePmhhtuSO7Gs+v18RtvvGGK/vz61782e/bsKdy+aNyutBN+u3fvLszv0P//N2bQf95cu6Awj1B+XdlXRecJv+L/1mUxhV95frArz072ovDjgEBsBBB4sa0I+UAAAhCAAAQg0AoCVQSeCL6iPy+//HIi8Iq2p53LVuTdwYMHC/M+Mv/D5siVHxjon2PrFhXmEcqP/Vdt/8EPfnXtAa5ddy+FchV+HBCIjQACL7YVIR8IQAACEIAABFpDQN5CK593J2+ZtQ8pS708X/UQgScCiqMcAbmTQl64FT26IPBOPn5zURzJnXoh/AoH7kjD0P3XESyFpwm/wqhSDWGXQhJUIfw4IBAbAQRebCtCPhCAAAQgAAEItIbAiRMnzLe+9a3kSyxWrFiRiDz5Mos5c+aYpUuX1vJNtAi8atsh9EUsAs/lHcrP7U0JftX2APzK84NdeXbSU/hxQCA2Agi82FaEfCAAAQhAAAIQaBWBU6dOmTVr1iTSbmhoyMybN888/PDDRurrOBB41SiGvohF4Lm8Q/m5vSnBr9oegF95frArz056Cj8OCMRGAIEX24qQDwQgAAEIQAACELAIIPAsGCUehr6IReC5kEP5ub0pwa/aHoBfeX6wK89Oego/DgjERgCBF9uKkA8EIAABCEAAAhCwCCDwLBglHoa+iEXguZBD+bm9KcGv2h6AX3l+sCvPTnoKPw4IxEYAgRfbipAPBCAAAQhAAAIQsAgg8CwYJR6GvohF4LmQQ/m5vSnBr9oegF95frArz056Cj8OCMRGAIEX24qQDwQgAAEIQAACELAIIPAsGCUehr6IReC5kEP5ub0pwa/aHoBfeX6wK89Oego/DgjERgCBF9uKkA8EIAABCEAAAhCwCCDwLBglHoa+iEXguZBD+bm9KcGv2h6AX3l+sCvPTnoKPw4IxEYAgRfbipAPBCAAAQhAAAIQsAgg8CwYJR6GvohF4LmQQ/m5vSnBr9oegF95frArz056Cj8OCMRGAIEX24qQDwQgAAEIQAACELAIIPAsGCUehr6IReC5kEP5ub0pwa/aHoBfeX6wK89Oego/DgjERgCBF9uKkA8EIAABCEAAAhCwCCDwLBglHoa+iEXguZBD+bm9KcGv2h6AX3l+sCvPTnoKPw4IxEYAgRfbipAPBCAAAQhAAAIQsAgg8CwYJR6GvohF4LmQQ/m5vSnBr9oegF95frArz056Cj8OCMRGAIEX24qQDwQgAAEIQAACELAIIPAsGCUehr6IReC5kEP5ub0pwa/aHoBfeX6wK89Oego/DgjERgCBF9uKkA8EIAABCEAAAhCwCCDwLBglHoa+iEXguZBD+bm9KcGv2h6AX3l+sCvPTnoKPw4IxEYAgRfbipAPBCAAAQhAAAIQsAgg8CwYJR6GvohF4LmQQ/m5vSnBr9oegF95frArz056Cj8OCMRGAIEX24qQDwQgAAEIQAACELAIIPAsGCUehr6IReC5kEP5ub0pwa/aHoBfeX6wK89Oego/DgjERgCBF9uKkA8EIAABCEAAAhCwCCDwLBglHoa+iEXguZBD+bm9KcGv2h6AX3l+sCvPTnoKPw4IxEYAgRfbipAPBCAAAQhAAAIQsAgg8CwYJR6GvohF4LmQQ/m5vSnBr9oegF95frArz056Cj8OCMRGAIEX24qQDwQgAAEIQAACELAIIPAsGCUehr6IReC5kEP5ub0pwa/aHoBfeX6wK89Oego/DgjERgCBF9uKkA8EIAABCEAAAhCwCCDwLBglHoa+iEXguZBD+bm9KcGv2h6AX3l+sCvPTnoKPw4IxEYAgRfbipAPBCAAAQhAAAIQsAgg8CwYJR6GvohF4LmQQ/m5vSnBr9oegF95frArz056Cj8OCMRGAIEX24qQDwQgAAEIQAACELAIIPAsGCUehr6IReC5kEP5ub0pwa/aHoBfeX6wK89Oego/DgjERgCBF9uKkA8EIAABCEAAAhCwCCDwLBglHoa+iEXguZBD+bm9KcGv2h6AX3l+sCvPTnoKPw4IxEYAgRfbipAPBCAAAQhAAAIQsAgg8CwYJR6GvohF4LmQQ/m5vSnBr9oegF95frArz056Cj8OCMRGAIEX24qQDwQgAAEIQAACELAIIPAsGCUehr6IReC5kEP5ub0pwa/aHoBfeX6wK89Oego/DgjERgCBF9uKkA8EIAABCEAAAhCwCCDwLBglHoa+iEXguZBD+bm9KcGv2h6AX3l+sCvPTnoKPw4IxEYAgRfbipAPBCAAAQhAAAIQsAgg8CwYJR6GvohF4LmQQ/m5vSnBr9oegF95frArz056Cj8OCMRGAIEX24qQDwQgAAEIQAACELAIIPAsGCUehr6IReC5kEP5ub0pwa/aHoBfeX6wK89Oego/DgjERgCBF9uKkA8EIAABCEAAAhCwCCDwLBglHoa+iEXguZBD+bm9KcGv2h6AX3l+sCvPTnoKPw4IxEYAgRfbipAPBCAAAQhAAAIQsAgg8CwYJR6GvohF4LmQQ/m5vSnBr9oegF95frArz056Cj8OCMRGAIEX24qQDwQgAAEIQAACrSFw9913m6Ghodwfeb7qgcCrRjD0RSwCz+Udys/tTQl+1fYA/Mrzg115dtJT+HFAIDYCCLzYVoR8IAABCEAAAhBoDYGnn37arFixIvVz0003mZkzZ5qf/vSnleeCwKuGMPRFLALP5R3Kz+1NCX7V9gD8yvODXXl20lP4cUAgNgIIvNhWhHwgAAEIQAACEGg1gcOHD5vrr78+kXojIyOV54LAq4Yw9EUsAs/lHcrP7U0JftX2APzK84NdeXbSU/hxQCA2Agi82FaEfCAAAQhAAAIQaC0BEXbLly9PBJ6IvDoOBF41iqEvYhF4Lu9Qfm5vSvCrtgfgV54f7Mqzk57CjwMCsRFA4NW8Im++c9o88uIfzTVrd5i/+f6z5t9eu8F8dOHojzyWOnlO2khbDghAAAIQgAAEBofA9u3bzeWXX26efPLJ2iaFwKuGMvRFLALP5R3Kz+1NCX7V9gD8yvODXXl20lP4cUAgNgIIvBpWZOTceXP0xHvmhX1HzQ+e2Wdmr3nJ/O3/et781Td/MS7vVOJJnTwnbaSt9JG+EoMDAhCAAAQgAIH2Ejh58qS55ZZbzDe/+U1z4sSJ2iaCwKuGMvRFLALP5R3Kz+1NCX7V9gD8yvODXXl20lP4cUAgNgIIvIorcvbceTP87hmzee8Rc/1PXjXTb3vKfOy6jSlxpwJPz9JG2kof6SsxJBYHBCAAAQhAAALtJLBjx47a774TEgi8avsh9EUsAs/lHcrP7U0JftX2APzK84NdeXbSU/hxQCA2Agi8iisi4u3Z3YfNV1a9aP7jbZvMxxf/bEp5pxJP2kof6SsxJBYHBCAAAQhAAALtI3D27Flz1113mYULF5pDhw5NOQF5YVD0RwTenj17Crffvf8Ppgs/Ifx2795dmN/hqy4wR678wED/vLl2QWEesv9C+BVdl660g1/xf+uy9gT8yvMLZXdg3z7ThZ+sfZZVJ/w4IBAbAQRexRX55f5hc/P6XcnbZf/d9T8tLO9U4kkfeVutxJBYvT8Om4cvm2am3fpS74dihEYIvHQr69kIaAaBAAQgMAkBkXYi7+655x5z/vzUd9QfP37cFP2RFxEi8Iq2//z3ngn+e0T/LmnLeebq7YV5iHw6ePBg4fZduAPv2LpFhXmE8iu6T7vSDn7F/63L2hPwK88vlN2jy24xy770xYH+WXHV7KB/+yb5tc9TEOgLAQReBezvjZwz9zy/31y89Gnz70vIO/0jWfpKDIklMXt7IPB6y7f56Ai85pl3fcS9e/ea6dOnm2nTpmX+zJ0715w6darrmJh/xwi88sorZtasWeall+r/H2Qi8ERAFT2+cPtzAy/wrnig+J0RcmeFiIGiRxcE3snHby6KI7lTL4Rf4cAdaRi6/zqCpfA04VcYVaphKLsN318y0PJO5OSqq69IccqrEH4cEIiNAAKvwoo8+dohM/ehX5kLv/4z59tmVcwVPcu300oMiSUxe3tUFHh/ethcNu0y8/CfepvlQEmpHjMbKFa93VZE7zGBY8eOmRkzZpht27b1eCTCQyA+AuvWrTPz5883r7/+eu3JIfA2pIQkAq/aW3wReLVfprkBQyVKbqCOPgG/8gsfyg6B57JG4Lk8KMVBAIFXYR2+9dPXzIzlL6T+qCwq7vx2Ekti9vZA4PWWb0Z0BF4GFKoGkcCSJUsMd98N4soyp6kIyFtmf/CDH5gbbrgh6E6vqeLq8wg8BF7dn8mHwNOrq/fnUInS+4zaNQL8yq9XKDsEnssagefyoBQHAQRehXX4u3/cav6vWzfVJvAklsTs7YHA6y3fjOgIvAwoVA0aAe6+G7QVZT4hBE6fPm1EYMuPPK77QOAh8BB4dV9VzcULlSjNZdaOkeBXfp1C2SHwXNYIPJcHpTgIIPAqrMNf3PxE0LfO+nfc+WX5VlqJWedxeO1lzmdUXbb2pZwvsRgTe/ZnWjlfdJHx/LRp5rK1h5N0x9/Gmciqic/F0ufdOb1kbrHHmXaLGf/EoBdvcfId/Ywt63k3kDFmNNYtLxrjztV7m28S16uTWGP5Sn/n8OYx7bKHzUsJy3Quydyt+UzEmpyZM57JEatjPCZiSq/Rtsq2CPukzWUPm9HVskce5aex7Gd4DIEQAo888kjy9lkReRwQgEC9BBB4CDwEXr3XVJPRQiVKk7m1YSz4lV+lUHYIPJc1As/lQSkOAgi8CuvgC7i6yhVScrqOiiVbOFlCyZZzKqvsujExJuLKkT5J27QIG5dYdvss+TQ2li2M0nkak9Q5+ThTswoTMnDSmCECLyPvCTmYwbPQnNPMrEkkD0fHsONbUtJm4a1BIfY5888a08+LMgSmIqB334nE44AABOongMBD4CHw6r+umooYKlGayqst48Cv/EqFskPguawReC4PSnEQQOBVWIe6hJ0fp0JKE109yTPxxJjwsoRQriwbk23O3V85cbMknN4pNs0fyxZeSWLpu8Byc5qYyNij9HySJ/zccwRW+g489+42e7jUHHNiJu3sOeYws2Nn5jx2p90tt95ibJGaSDcrfiqvJJh/R59flkZZdamsqIDAlATkSyvkyyu4+25KVDSAQCkCCDwEHgKv1KUTRadQiRJF0hElAb/yixHKDoHnskbguTwoxUEAgVdhHXzxVle5QkrjXX3JM/5EStqMCjBH0o03zhA8OTIqT7i5MitvrPQ4efHGUxt/kBdztH78rrwc2ZYSeL74Gx9H74abuEMuL8fUXW05zKzQYw9HOYznnPS7xbzk9ffH9csaN6m3RF/5vDQiZwhkE9DP/sp+lloIQKAqAQQeAg+BV/Uq6l//UInSv0zjHBl+5dcllB0Cz2WNwHN5UIqDAAKvwjrE/Bl4vryZmKYnyyYRVpl3aHkySeMWkkhjY41+rt3E5+SNl/079ayyjpM+1yzw8kSf3K/mfAbeGEfrs+/G55HUTYi+UUk49VtoZW62eE0eJwxssZeebyH2Etxba3usNFdqIFCMwN69e8306dON3IXHAQEI9IYAAg+Bh8DrzbXVRNRQidJETm0aA37lVyuUHQLPZY3Ac3lQioMAAq/COsT8LbT5csYTeNaXQKRR+G1VAqVlVDGJlJZP6TFHa/LipdvnxRytd+9mS+ftSy1TWOAFfE5fjvRMz8WYifFH2eudkeMyL8nPkoPyNR63TjP225Q1blJv3YEn9RNtbSmoPThDIJwAb58NZ0YPCIQSQOAh8BB4oVdNPO1DJUo8mceRCfzKr0MoOwSeyxqB5/KgFAcBBF6FdfjWT18zM5a/YOp666zEkpi1HHkiSu+CG7+7LUPSaQLeHVtJdY6MmhBD2nn07EqkScZyu1miyXsiVQwReNOMCjENk4ixaXa9J/60ob712PrG3NG+rkwbb24/yGFmN5l4PDaftQ+by6yxRsXeLeZhuQtwfO1GexVjPzaCCkA9TwzMIwiUIsDbZ0thoxMEgggg8BB4CLygSyaqxqESJarkI0gGfuUXIZQdAs9ljcBzeVCKgwACr8I6PPnaITP3oV+ZC7/+M/Nvr03/cVlU7ElfiSGxJGY9x5gssyXQuITy7thKST3JYFQk+bJI68fvbBtLtrBESsTRNJPu74qwwnIs9w5CX8Rl8NB5OwJP3yprS72JumkOzzFG3l1uIttcUejnMvkKC8vLLvNFncS4zFx2mZtXslIBd+CNr18q/uQ58SwEsgjw7bNZVKiDQP0EEHjpv7GueODlwqBDX8Qemf9hU7cwiy3eycdv7hm/woE70jB0/3UES+Fpwq8wqlTDUHYIPBeh8OOAQGwEEHgVVuS9kXPmnuf3m4uXPm3+/fU/LX0nnvSVGBJLYtZ3qLTSz5uTt5Dm3QU3JqOsz3TzJZvmpXetyWe+aZvCAk+CWOJMPzdO4+gYKppGn3fl3kQbeTSatyvMJurduN4cRbyN5eL3t+eo88yWij7jac43xmqudjw3J21hncckp59Twnha+m3AQez1c/Y8aWmNzkMIFCbA598VRkVDCFQigMBD4NUtABF4lS7JoM6hEiUoeAcaw6/8IoeyQ+C5rBF4Lg9KcRBA4FVch1/uHzY3r99l/uqbvzD/roTEkz7SV2JILI54CSQSzr/bLt50czMblHnkTpAnIAABCAwYAQQeAg+B196LOlSitHemvckcfuW5hrJD4LmsEXguD0pxEEDgVVyH4XfPmGd3HzZfWfWi+Y+3bTIfX/yzwnfiSVvpI30lhsTiiJVA3p2Lseabl9foXYhT3gWY1516CEAAAhBonAACD4GHwGv8sqttwFCJUtvAAxIIfuUXMpQdAs9ljcBzeVCKgwACr+I6nD13PhFvm/ceMdf/5FUz/banzMeu2zilxJM20lb6SF+RdxKLo/8EXro1522qzuff9T/PMhlkvw24TCT6QAACEIBAUwQQeAg8BF5TV1v944RKlPozaHdE+JVfv1B2CDyXNQLP5UEpDgIIvBrWYeTceXP0xHvmhX1HzQ+e2Wdmr3nJ/O3/ej55a6z/RRbydll5TtpIW+kjfSUGRxwE7M+r08/om9byt85OzGmyzxOMgz9ZQAACEICASwCBh8BD4LnXRJtKoRKlTXNrIlf4laccyg6B57JG4Lk8KMVBAIFX8zq8+c5p88iLfzTXrN1h/ub7zzrfTivfNit18py0kbYcEIAABCAAAQhAYDICCDwEHgJvsisk7udCJUrcs2k+O/iVZx7KDoHnskbguTwoxUEAgRfHOpAFBCAAAQhAAAIQyCSAwEPgIfAyL41WVIZKlFZMqsEk4Vcedig7BJ7LGoHn8qAUBwEEXhzrQBYQgAAEIAABCEAgkwACD4GHwMu8NFpRGSpRWjGpBpOEX3nYoewQeC5rBJ7Lg1IcBBB4cawDWUAAAhCAAAQgAIFMAgg8BB4CL/PSaEVlqERpxaQaTBJ+5WGHskPguawReC4PSnEQQODFsQ5kAQEIQAACEIAABDIJIPAQeAi8zEujFZWhEqUVk2owSfiVhx3KDoHnskbguTwoxUEAgRfHOpAFBCAAAQhAAAIQyCSAwEPgIfAyL41WVIZKlFZMqsEk4Vcedig7BJ7LGoHn8qAUBwEEXhzrQBYQgAAEIAABCEAgkwACD4GHwMu8NFpRGSpRWjGpBpOEX3nYoewQeC5rBJ7Lg1IcBBB4cawDWUAAAhCAAAQgAIFMAgg8BB4CL/PSaEVlqERpxaQaTBJ+5WGHskPguawReC4PSnEQQODFsQ5kAQEIQAACEIAABDIJIPAQeAi8zEujFZWhEqUVk2owSfiVhx3KDoHnskbguTwoxUEAgRfHOpAFBCAAAQhAAAIQyCSAwEPgIfAyL41WVIZKlFZMqsEk4Vcedig7BJ7LGoHn8qAUBwEEXhzrQBYQgAAEIAABCEAgkwACD4GHwMu8NFpRGSpRWjGpBpOEX3nYoewQeC5rBJ7Lg1IcBBB4cawDWUAAAhCAAAQgAIFMAgg8BB4CL/PSaEVlqERpxaQaTBJ+5WGHskPguawReC4PSnEQQODFsQ5kAQEIQAACEIAABDIJIPAQeAi8zEujFZWhEqUVk2owSfiVhx3KDoHnskbguTwoxUEAgRfHOpAFBCAAAQhAAAIQyCSAwEPgIfAyL41WVIZKlFZMqsEkA4kxPQAAIABJREFU4Vcedig7BJ7LGoHn8qAUBwEEXhzrQBYQgAAEIAABCLSUwPnz58327dvN4sWLzcyZM83ll19u7rzzTnP06NFaZoTAQ+Ah8Gq5lPoSJFSi9CXJiAeFX/nFCWWHwHNZI/BcHpTiIIDAi2MdyAICEIAABCAAgRYSEHm3YcMGM2vWLLNs2TKzefNm88///M9m7ty55sYbbzTvvPNO5Vkh8BB4CLzKl1HfAoRKlL4lGunA8Cu/MKHsEHguawSey4NSHAQQeHGsA1lAAAIQgAAEINBCAocOHTILFy40K1asMCMjI+Mz2LFjh5k9e7bZunXreF3ZBwg8BB4Cr+zV0/9+oRKl/xnHlQH8yq9HKDsEnssagefyoBQHAQReHOtAFhCAAAQgAAEItJDApk2bzIIFC4yIvF4dCDwEHgKvV1dX7+OGSpTeZ9SuEeBXfr1C2SHwXNYIPJcHpTgIIPDiWAeygAAEIAABCECghQTuvvtus2TJErN//37z3e9+N3kr7Zw5c8yaNWvMqVOnapkRAg+Bh8Cr5VLqS5BQidKXJCMeFH7lFyeUHQLPZY3Ac3lQioMAAi+OdSALCEAAAhCAAARaRuC9994zS5cuTT7v7mtf+5q56667krfMyttp9TPxTp8+XXlWCDwEHgKv8mXUtwChEqVviUY6MPzKL0woOwSeyxqB5/KgFAcBBF4c60AWEIAABCAAAQi0jIDIObn7bmhoyDz22GNGvtBCDjk/8cQTicTbsmVL5Vkh8BB4CLzKl1HfAoRKlL4lGunA8Cu/MKHsEHguawSey4NSHAQQeHGsA1lAAAIQgAAEINAyAirw5s+fb15//XUn++Hh4eTLLe677z6nXgvywqDojwi8PXv2FG7/uWWbzEcXpqXXINV95d4XCvMQfrt37y7c/vBVF5i6hVls8d5cu6Awj1B+Rfd1V9rBr/i/dVl7An7l+YWyW3vrTWbZl7440D/3zB0K+rdPf2dzhkAsBBB4sawEeUAAAhCAAAQg0CoCZ8+eNXfccYe59tprzVtvveXkLmWpl8/IyzqOHz9uiv7IizAReEXbf/57zwy8wJu5enthHiLvDh48WLj9kfkfHniBd2zdosI8QvkV3addaQe/4v/WZe0J+JXnF8ru0WW3DLS8Ezm54qrZQf/2Zf3+pg4C/SSAwOsnfcaGAAQgAAEIQKDVBB555JHkM/DkSyzsQ76VVr6dVp6veojAEwFV9PjC7c8NvMC74oGXi+JI7rYQMVD06ILAO/n4zUVxBPMrHLgjDeWuspD91xEshacJv8KoUg1D2fEWWheh8OOAQGwEEHixrQj5QAACEIAABCDQGgJyZ9xXv/pVI19cMTIykuQtn4Enn4k3e/Zss2PHjspzQeCl3w6MwPtApbsEEXiVL8vCAUIlSuHAHWkIv/ILHcoOgeeyRuC5PCjFQQCBV/M6nHv7DXN620pz/EdD5q1v/6U5Mu+DE39gzftgUifPSRtpywEBCEAAAhCAQHsJiLQTeTdz5kyzbNkys3nz5qQs30JrS70qM0TgIfDq/gw9BF6VKzKsb6hECYs++K3hV36NQ9kh8FzWCDyXB6U4CCDwaliH82ffN+eOv2nO7HrcnHz8G+ade2eYt5b+lRn++p9NyLsrR/9PqdTJc9JG2kof6SsxOCAAAQhAAAIQaB8BkXjyrbPXXHNN8o208+bNMz/5yU/MmTNnapkMAg+Bh8Cr5VLqS5BQidKXJCMeFH7lFyeUHQLPZY3Ac3lQioMAAq/iOiTy7p2D5r2d68yJB/+nGb7pY+bIvA+lxF3qD695H0raSh/pe+6dg0i8imtBdwhAAAIQgMAgEkDgIfBSf0eO/Y/hsvXcgdfcvxShEqW5zNoxEvzKr1MoOwSeyxqB5/KgFAcBBF7FdUjuvHv1MfP27Reb4cUfMUfnXzC1vBv7o0vaSh/pe+bVx5I78SqmQ3cIQAACEIAABAaMAAIPgVdW1OX1Q+A1949EqERpLrN2jAS/8usUyg6B57JG4Lk8KMVBAIFXcR3kLbDJnXeLPmKOXFVc3o3/QXXVBWZ40UeSGBKr18epU6eSb8tbsmRJr4cifgsIyLcjzpgxwxw7dqwF2ZIiBCAAgW4SQOAh8Mb/bqx4553GQeA1929JqERpLrN2jAS/8usUyg6B57JG4Lk8KMVBAIFXYR3kSyjkc+zkbbMhd97pH096Tu7Eu+ljSaxef7EFAq/Cgg9g1zICT2SfSD/py9FtAroXpk2bZuSH/zHQ7f3A7HtHAIGHwNO/Ges6I/B6d736kUMlit+/62X4ld8BoewQeC5rBJ7Lg1IcBBB4FdYh+bbZe2cU+8y7qf6P6bwPmeP3zki+nbZCSlN2rSrw9u7day655BIj514eIgIGRQY0xazMeiDwylCjjxBQeafXqf7bMnfuXCOPOSAAgfoIIPAQeHWJO42DwKvv+pwqUqhEmSpe156HX/kVD2WHwHNZI/BcHpTiIIDAq7AOx380lHyjrP4xVPUs304rMXt56ItsfdEdOlZTMgqBF7oy5dqXEXjlRqLXoBHI2jvy78P06dPNtm3bBm26zAcCfSWAwEPgVf0b0++PwGvukg6VKM1l1o6R4Fd+nULZIfBc1gg8lwelOAgg8Cqsw1vf/ksz/PU/K/ylFf4fT35ZYknMXh4IvF7SzY7dlPTMHn3y2iwJM3kPnoXAKAGR7P7ddnpXHgKPXQKBegkg8BB4/t+MVcsIvHqv0cmihUqUyWJ18Tn4lV/1UHYIPJc1As/lQSkOAgi8CutwZN4Ha5N343+IzftghYzSXUXQ6OdTyXnlypWZX2KhYs9ua9+ll/W8tJX4cugdc3oHjsbR5+3M9EW+trG/REFe+Gu9nu3n7TjyWGNJP3uucheQ5KKHPO/XyXOary8ctF5zEFEh7LJykblrOzlrrKmYaW7+OWtsiWXna6+N1MvcdX55TCQ3fz2k7M9J8rfnY89JxtL4dix5LIzkF53E0/5+nskk+M9AEMjaO7J3m3iL/UAAZBIQCCCAwEPgjf+dONVHshR8HoEXcAFWbBoqUSoON3Dd4Vd+SUPZIfBc1sKPAwKxEUDgVViRuv+Y0ngVUnK6ijyx5YwtlGyxosLIrlNJ499hk/cCXfqKtLHbqwhSoSXJ6Vi2/PHzlHZSZ+fjTMwqaJ4y9mQxJQcVXFb38XzsHLPyltgyRhbPInMuKjX8sXXN/DHsuShTnYMy+fKXv+ww0dg2J3nsz0m4Sww9pI09nsb34/h8NC+7ncbk3H4Cug/0OtW9quX2z5AZQCAeAgg8BJ7+jVjXGYHX3PUdKlGay6wdI8Gv/DqFskPguawReC4PSnEQQOBVWIe6/ojy41RIabxrnmjzX3RLB3nBnfWiWwWMiiFpmxdX+tsiSNpmvaCXdraMknaaky168nKS9vahff38/dxlDraE0hh+O83ZzkXb+nPMi+nPMY+ZxtWzjj3VXKS9PYY8tvvkMZF+Mi97nfyy5mKfNZ7uAy3bjORxFl87TzsmjweDgO4FkbfyY+/DwZghs4BAHAQQeAg8/2/FqmUEXnPXdqhEaS6zdowEv/LrFMoOgeeyRuC5PCjFQQCBV2EdYn4LrQgVX5TJVH1JpC/AVc7YOPy28lyejPIlksaxBU7eWFnj5MXTuHrOi6n1KpnyZJsv8PyyjiNniWXLr7wc/XZ5zOzY8jivnT8Xaat1koOdk/1c1pr6HPxcNSfloGJGzspSx9ay9JHHWfstL76Ow7m9BPy9pPsgS+S2d5ZkDoE4CCDwEHhVhZ3fH4HX3LUdKlGay6wdI8Gv/DqFskPguawReC4PSnEQQOBVWIeYv8TCFmf2FH1ZpqImS/b4bSVOnmTKk1l2HjqWLYXsx9JWj7x4+ryeVSb5+Wu9SqYs2SAxNCftn9dO2toyStnY+duPbamWx0znoGcZ247hP9a5+O01d63Xufv18rw/P3tO8rz2tSWM1un4fln6yXMIPF2BwT9n7QGZtV4X9rU8+DSYIQR6TwCBh8DzBVzVMgKv99etjhAqUbQf51EC8Cu/E0LZIfBc1gg8lwelOAgg8Cqsw/EfDZm3lv5VbV9kIbEkZh1HnlDxX2DrC/Es2eO3lbzyZFSecJN6FTuTjeXPOS+e3y4vptardMrLu6zAkzyK5pg3tj+Xou2kn85PPufOloX2c1lrKnV2e+Fjl+310vx0LGXpl6Vd3n7z42tMzu0m4F839mzy9oLdhscQgEAYAQQeAq+qsPP7I/DCrsEqrUMlSpWxBrEv/Mqvaig7BJ7LGoHn8qAUBwEEXoV1OL1tpTl+7wxzZN6Hqku8eR9KYknMOg7/TiuNqS+8RdTIkSXp/La2CMqTTHkyyxZCk42lY+o5L54+r2eVSXaO8pzWq3TSefvt5Hm5003r/X46juZuy66iciqPmcbWc97Y+rx9Vq7SRwSplPXQOHadPqf9ZD5y+HPwn5c2wkYYKUuNr2WNo6I2CTz2Hz++/RyP20sgaw/IbPQ6ydp77Z0tmUOg/wQQeAg8X8BVLSPwmruuQyVKc5m1YyT4lV+nUHYIPJc1As/lQSkOAgi8Cutw7u03zMnHv2GGb/qYOTr/gtIST/pKDIklMes49IW0LZy0TmSM/QJb5ZZdpy/Q7TrJS+tteSP10s5vq/W22PFlkM511apVSWwtFxU/mo8KOO2v9Zqnzt3mofO2BZ70lz55dXZ/HcOen/SXXOx8tJ3mojlmnbPGltxXrlyZyBGNb7/FVeehY+p4eXPQdhJLxrPnJGU7th1L89c6LWscn4PW2/GljmMwCPh7RWYldf6+G4zZMgsI9JcAAg+BV1XY+f0ReM1d06ESpbnM2jES/MqvUyg7BJ7LGoHn8qAUBwEEXsV1OLPrcXPiwf9phhd9xBy5qoTEu+qCpK/EkFh1Hiqt5AW1/IiY2blzZ+qOLRlTpYy2lbMtaOy89EW63aaowJM4KpwmG8vOZzIBpO1sKWXPx56DttVxRTgJD+Hi97fnqPOUOj8Xn7G0zRNZOq6dk81VH0su2lbPmp8+58cQ/iredJ5PP/10kovG0Od1HDlnzUliaR+Z7759+5J565gaX8saJ2/ePjN7fB63m4DuR3u/yP7ggAAE6iWAwEPg+QKuahmBV+81Olm0UIkyWawuPge/8qseyg6B57JG4Lk8KMVBAIFXcR3OHX/TnHn1MfP27Reb4cUfCboTL7nzbvFHkr4SQ2JxxEtAhFWWpIotYxVsKv1iy498IAABCEAgjAACD4FXVdj5/RF4YddgldahEqXKWIPYF37lVzWUHQLPZY3Ac3lQioMAAq/iOpw/+745985B897OdaN34t30sWKfiTfvQ8nbZuXOO+krMSQWR5wE9E67rLcJx5YxAi+2FSEfCEAAAtUIIPAQeL6Aq1pG4FW7JkN6h0qUkNhdaAu/8qscyg6B57JG4Lk8KMVBAIFXwzokEk/uxNv1ePI5du/cOyP5dtrhr/9Z6nPxpE6+bVbayGfeSR+58w55V8NC1BRi+fLlydt87XAi7tryVlAEnr1yPIYABCDQfgIIPAReVWHn90fgNffvQqhEaS6zdowEv/LrFMoOgeeyRuC5PCjFQQCBV/M6yJdQJN9O+6Mh89a3/9IcmffBCYk374NJ3fEfDSVt6vrCipqn0Plw8lZZ/UwvPbfhrbO6cAg8JcEZAhCAwGAQQOAh8HwBV7WMwGvu34ZQidJcZu0YCX7l1ymUHQLPZY3Ac3lQioMAAi+OdSALCEAAAhCAAAQgkEkAgYfAqyrs/P4IvMxLrSeVoRKlJ0m0OCj8yi9eKDsEnssagefyoBQHAQReHOtAFhCAAAQgAAEIQCCTAAIPgecLuKplBF7mpdaTylCJ0pMkWhwUfuUXL5QdAs9ljcBzeVCKgwACL451IAsIQAACEIAABCCQSQCBh8CrKuz8/gi8zEutJ5WhEqUnSbQ4KPzKL14oOwSeyxqB5/KgFAcBBF4c60AWEIAABCAAAQhAIJMAAg+B5wu4qmUEXual1pPKUInSkyRaHBR+5RcvlB0Cz2WNwHN5UIqDAAIvjnUgCwhAAAIQgAAEIJBJAIGHwKsq7Pz+CLzMS60nlaESpSdJtDgo/MovXig7BJ7LGoHn8qAUBwEEXhzrQBYQgAAEIAABCEAgkwACD4HnC7iqZQRe5qXWk8pQidKTJFocFH7lFy+UHQLPZY3Ac3lQioMAAi+OdSALCEAAAhCAAARaSuCVV14xM2fONENDQ87P3Llzzf79+yvPCoGHwKsq7Pz+CLzKl2XhAKESpXDgjjSEX/mFDmWHwHNZI/BcHpTiIIDAi2MdyAICEIAABCAAgZYSWL9+vfnqV79qfvCDH5gVK1aM/6xZs8YcPny48qwQeAg8X8BVLSPwKl+WhQOESpTCgTvSEH7lFzqUHQLPZY3Ac3lQioMAAi+OdSALCEAAAhCAAARaSOD8+fOJuLvtttvMqVOnejIDBB4Cr6qw8/sj8HpyqWYGDZUomUE6XAm/8osfyg6B57JG4Lk8KMVBAIEXxzqQBQQgAAEIQAACLSQg0k7k3d13392z7BF4CDxfwFUtI/B6drmmAodKlFSAjlfAr/wGCGWHwHNZI/BcHpTiIIDAi2MdyAICEIAABCAAgRYSGB4eNgsXLjQPPvigWb58uZk1a5aZM2eOkbfP1nVHHgIPgVdV2Pn9EXjN/WMTKlGay6wdI8Gv/DqFskPguawReC4PSnEQQODFsQ5kAQEIQAACEIBACwn85je/MVdccUUi7e6//36zdevW5G48EXlLly41J0+erDwrBB4CzxdwVcsIvMqXZeEAoRKlcOCONIRf+YUOZYfAc1kj8FwelOIggMCLYx3IAgIQgAAEIACBFhJ46aWXzJVXXmm2b98+nr18Lt6zzz6bfDPtE088MV5f9gECD4FXVdj5/RF4Za/G8H6hEiV8hMHuAb/y6xvKDoHnskbguTwoxUEAgRfHOpAFBCAAAQhAAAIDRODYsWPmuuuuM3fccYc5e/ZsambywqDojwi8PXv2FG7/uWWbzEcXpqXXINV95d4XCvMQfrt37y7c/vBVFxhfeA1a+c21CwrzCOVXdF93pR38iv9bl7Un4FeeXyi7tbfeZJZ96YsD/XPP3KGgf/tSv7ypgECfCSDw+rwADA8BCEAAAhCAQLsJjIyMGLnrzj70yy2WLFliTp8+bT+VPD5+/Lgp+iMvwkTgFW3/+e89M/ACb+bq7YV5iLw7ePBg4fZH5n944AXesXWLCvMI5Vd0n3alXQi/Q+seM/v+x6UD/zO8Ywf7L+B3QNlrJWTvyRiPLrtloOWdyMkVV80O2nupX95UQKDPBBB4fV4AhocABCAAAQhAoL0E5NtnFyxYYA4dOuRMQspSf9999zn1ZQoi8ERAFT2+cPtzAy/wrnjg5aI4krst5MVp0aMLAo+30BbdDdXbyV1lRfff8Sd+YXZ9/MKB/3lv377CYEP4FQ7akYah7HgLrbsxhB8HBGIjgMCLbUXIBwIQgAAEIACB1hDYsmVL8s2zGzZsGL8LT+7Ge+yxx8zs2bPNjh07Ks8FgZd+OzAC7wOV7hJE4FW+LAsHCJEoCLw01hB+6d7drgllh8Bz9wsCz+VBKQ4CCLw41oEsIAABCEAAAhBoIQH5lln5tln51tkVK1Yk30J71113JV9gIWV5e23VA4GHwKv7M/gQeFWvyuL9QyQKAi/NNYRfune3a0LZIfDc/YLAc3lQioMAAi+OdSALCEAAAhCAAARaSkA+727NmjVmzpw5ZmhoyFxzzTVGvn22DnknSBB4CDwEXkv/cTAm6C3cCLz0OodKqHSE7taEskPguXsFgefyoBQHAQReHOtAFhCAAAQgAAEIQCCTAAIPgYfAy7w0WlEZIlEQeOklDeGX7t3tmlB2CDx3vyDwXB6U4iCAwItjHcgCAhCAAAQgAAEIZBJA4CHwEHiZl0YrKkMkCgIvvaQh/NK9u10Tyg6B5+4XBJ7Lg1IcBBB4cawDWUAAAhCAAAQgAIFMAgg8BB4CL/PSaEVliERB4KWXNIRfune3a0LZIfDc/YLAc3lQioMAAi+OdSALCEAAAhCAAAQgkEkAgYfAQ+BlXhqtqAyRKAi89JKG8Ev37nZNKDsEnrtfEHguD0pxEEDgxbEOZAEBCEAAAhCAAAQyCSDwEHgIvMxLoxWVIRIFgZde0hB+6d7drgllh8Bz9wsCz+VBKQ4CCLw41oEsIAABCEAAAhCAQCYBBB4CD4GXeWm0ojJEoiDw0ksawi/du9s1oewQeO5+QeC5PCjFQQCBV/M6HDp5yPxkz0/MDZtvMJeuu9R8cvUnzUWrLkp+5LHUyXPSRtpyQAACEIAABCAAgckIIPAQeAi8ya6QuJ8LkSgIvPRahvBL9+52TSg7BJ67XxB4Lg9KcRBA4NWwDu+fe98cPXXUPP/68+bOV+40czbNMZc+dqm5+KGLx+WdSjypk+ekjbSVPtJXYnBAAAIQgAAEIAABnwACD4GHwPOvivaUQyQKAi+9riH80r27XRPKDoHn7hcEnsuDUhwEEHgV10HE2+FTh82m328yizYvMp9b+znzqdWfSok7FXh6ljbSVvpIX4mBxKu4GHSHAAQgAAEIDCABBB4CD4HX3gs7RKIg8NLrHMIv3bvbNaHsEHjufkHguTwoxUEAgVdxHY6cOmKe+sNT5h9++g/msw991nzmh5+ZUt6pxJO20kf6SgyJxQEBCEAAAhCAAARsAgg8BB4Cz74i2vU4RKIg8NJrG8Iv3bvbNaHsEHjufkHguTwoxUEAgVdxHeQtsHIXnYi4T6/+dGF5pxJP+khfiSGxen2cOnXKzJ071yxZsqTXQxEfAhCAAAQgAIEaCCDwEHgIvBoupD6FCJEoCLz0IoXwS/fudk0oOwSeu18QeC4PSnEQQOBVWAf5Egr5HDt5K2zInXcq7/QsfSWGxOr1F1sg8CosOF0hAIFxAnv37jXTp08306ZNG/955JFHxp/nAQQgUB8BBB4CD4FX3/XUdKQQiYLAS69OCL90727XhLJD4Ln7BYHn8qAUBwEEXoV1kG+SlS+jKPKZdyrr8s4SQ2JJzF4eVQWevGi/5JJLjJx7ecgdgoNyl2AIs5C2veRPbAhMRWDjxo3OvwPbtm1LRB4SbypyPA+BcAIIPAQeAi/8uomlR4hEQeClVy2EX7p3t2tC2SHw3P2CwHN5UIqDAAKvwjrcsPkGc+m6S4PfNpsn8SSWxOzlgcDrJd3s2CFSLqRt9mjUQqB/BES6y1v05d8ZDghAoD4CCDwEHgKvvuup6UghEgWBl16dEH7p3t2uCWWHwHP3CwLP5UEpDgIIvArrIMLt4ocurk3gSSyJ2csDgddLutmxQ6RcSNvs0aiFQP8IyN13CLz+8WfkwSWAwEPgIfDae32HSBQEXnqdQ/ile3e7JpQdAs/dLwg8lwelOAgg8CqswydXf7I2ead35UnMOg95QW1/RtXKlSszv8RCxZ7d1n4La9bz0lbfLqdveRUBZX8ulj5vz+nYsWNmxowZ43nJY6mTQ9+GZ+dhP2/HkccaS/rZc5UcJBc95Hm/Tp7TfOV5+9B6zUOkhLDLykXmru3krLGmYmaPN1VbmZvksHPnzmQe/lzycrDHsPlInllrY7fnMQRCCei/A6H9aA8BCExOAIGHwEPgTX6NxPxsiERB4KVXMoRfune3a0LZIfDc/YLAc3lQioMAAq/COqh0q/tcISWnq7yYtoWTLYnkOT1UVtl1Ksb8u2mkbdZn4ElfkUJ2exFZttCS8XQsWx75eUo7qbPz0Vz9s+bpCyk/ZojAy8pb5VcWzyJzzmLmz0XKeXxlfImxePFi5+2JuqZT5eDzyFqHrHyog0BRArJH7eujaD/aQQACUxNA4CHwEHhTXyextgiRKAi89CqG8Ev37nZNKDsEnrtfEHguD0pxEEDgVViHusWdxquQ0njXPBGkwsuWY/LYLmsQlTx6R5nU58X1BZG0Vblkx5bHtmySdpqTL/XsfpqTf9a+fls/96ICT3O2c9ExZQxbUOTF9OeYx0zj2ue8tpKPf9ed9CuSg89Cx0O4KAnOZQnIXhd5Lj/+NVg2Jv0gAIE0AQQeAg+Bl74u2lITIlEQeOlVDeGX7t3tmlB2CDx3vyDwXB6U4iCAwKuwDjG/hVbkjC/KZKoqqPTFtgowW9IpEr+t1OcJJomnMbW/nKVO88gbK2ucvHh2bHmcF1PrVcTliS5fbvllezxfeOXl6LfLY2bH1sd5bf2Y2r5IDtJX10D7yTmPid2GxxAoSkD2EyKvKC3aQSCMAAIPgYfAC7tmYmodIlEQeOmVC+GX7t3tmlB2CDx3vyDwXB6U4iCAwKuwDjF/iYUtzuwp+rJsMmHlt5U4eYIpTyTZeehYeseOf5a2euTF0+f1rKLOF5BaL/JKjjxZpTlp/7x2EsOWaMrGn4OW7Tv18pjpHOxzXtssCVc0B2GpefnnrLv67Hx4DIEQApNdPyFxaAsBCLgEEHgIPASee020qRQiURB46ZUN4Zfu3e2aUHYIPHe/IPBcHpTiIIDAq7AON2y+IfnWWH3ra9WzCEGJWceRJXwkrkoflWUqulRg2WP7beW5PMGUJ9ykXu/+mmwse1x5nBfPb5cXU+tV4OXlLfUisXT+kwkIW+CF5Jg3tj8XKee1zVvPIpzy+maNTx0EqhDwr6cqsegLAQhMEEDgIfAQeBPXQ9sehUgUBF56dUP4pXt3uyaUHQLP3S8IPJcHpTgIIPAqrMNP9vzEzNk0x3xq9acqfxutxJBYErOOI09E6QtsET9yZEk6HV/bqtyS+jzBlCeSpF4F3mRj6Zh6zounz+tZRZ2dozyn9bbAs0Wd9pfn5a407e/303aau31nnfS1y9rWP+cx89tJOa9tnoQrkkPeXsganzoIVCHAXqtCj74QyCeAwEPgIfDyr4/YnwmRKAi89GrOYfFrAAAgAElEQVSG8Ev37nZNKDsEnrtfEHguD0pxEEDgVViHQycPmTtfudN8bu3nzGd++JnSEk/6SgyJJTHrOLKEk9aJsFKBJ2OpqLPrVGTZddJW60Uc2UeecJN6FXjSXl7gy/h+/1WrViWxNWYRMSVtNR8VcNpf63Ucnbst3HTetsCT/tInr87ur2PY85P+koudj7bTXDTHrHNeW+nrjyP9tb3/nJ1D1tylr8x/48aNWWlQB4FJCci+k2tb9pYeej0V2efahzMEIFCMAAIPgYfAK3atxNgqRKIg8NIrGMIv3bvbNaHsEHjufkHguTwoxUEAgVdxHZ5//XmzaPMi89mHPms+vfrTwRJP+khfiSGx6jxU3IiMkh+5A23nzp2JCMoTc9pWznkvxKVe22mbogJP5qcv9DVG1lgqpuQ5W5r5fLSdLcykjdZrfnadjivSS3hMdmeetpU48uPn4jOW9r5Mk7Glrx3Ln4ddzmordVlxpV/RHGSNNAc5+3Oxc+AxBKYi4O8n2VP+dThVDJ6HAASKEUDgIfAQeMWulRhbhUgUBF56BUP4pXt3uyaUHQLP3S8IPJcHpTgIIPAqrsORU0fMU394yvzDT/8hEXEhd+JJW5F30ldiSCyOeAlMJtHizZrMIAABCECg7QQQeAg8BF57r+IQiYLAS69zCL90727XhLJD4Ln7BYHn8qAUBwEEXsV1eP/c++bwqcNm0+83JXfRyVthi3wmnrSRtnLnnfSVGBKLI04Cepebf+dinNmSFQQgAAEIDBIBBB4CD4HX3is6RKIg8NLrHMIv3bvbNaHsEHjufkHguTwoxUEAgVfDOoh4O3rqaPIWWPkcO/kyiksfu9Rc/NDFqbfUSp08J22krbxtVvoi72pYiJpCLF++PHmbrx1OxB1vObWJ8BgCEIAABJoigMBD4CHwmrra6h8nRKIg8NL8Q/ile3e7JpQdAs/dLwg8lwelOAgg8GpeB/kSCvkm2Rs232AuXXep+eTqT45LPHksdfKctKnrCytqnkLnw9mfQaefGZf3+XOdhwUACEAAAhBwCJw5c8bcfvvtyRe9nD592nmubAGBh8BD4JW9evrfL0SiIPDS6xXCL9272zWh7BB47n5B4Lk8KMVBAIEXxzqQBQQgAAEIQAACLSdw/vx5s2HDBjNz5kwE3sK0dPtojXVXPPBy4d0S+iL2yPwPm7qFWWzxTj5+c8/4FQ7ckYYh+w+Bl94UIfzSvbtdE8oOgefuF+HHAYHYCCDwYlsR8oEABCAAAQhAoJUE9u3bZ+bPnz/+be/cgdc7iYfA+0AlyYjAa+6fmBCJgsBLr0sIv3TvbteEskPgufsFgefyoBQHAQReHOtAFhCAAAQgAAEItJjAkSNHzOLFi81jjz1m7r77bu7Aq/Fuu6w79xB4CLy2/HMRIlEQeOlVDeGX7t3tmlB2CDx3vyDwXB6U4iCAwItjHcgCAhCAAAQgAIGWEhgZGTErVqwwS5cuNSdPnkTg9VjeidBD4CHw2vLPRYhEQeClVzWEX7p3t2tC2SHw3P2CwHN5UIqDAAIvjnUgCwhAAAIQgAAEWkrgySefNFdffbX54x//mMyAO/B699ZZvRsPgYfAa8s/FyESBYGXXtUQfune3a4JZYfAc/cLAs/lQSkOAgi8ONaBLCAAAQhAAAIQaCEB+dy7BQsWmO3bt49nj8BD4MX2pRV+PnwG3vjl2vMHIRIFgZdejhB+6d7drgllh8Bz9wsCz+VBKQ4CCLw41oEsIAABCEAAAhBoGYETJ06Yb37zm8nbZ+VttHoUEXjywqDoz8svv2z27NlTuP3nlm0yeqfaoJ6/cu8LhXkIv927dxduf/iqCyp9QYQvy2Isv7l2QWEeofyK7uuutAvhd+Chh82uj1848D9/2LKF/RfwO6DstRKy92SMtbfeZJZ96YsD/XPP3KGgvae/1zlDIBYCCLxYVoI8IAABCEAAAhBoFYGXXnrJDA0NTfojMi/rOH78uCn6Iy/CROAVbf/57z0z8AJv5urthXmIvDt48GDh9kfmf3jgBd6xdYsK8wjlV3SfdqVdCL9D6x4beHkngnJ4xw72X8DvgLLXSsjekzEeXXbLQMs7kZMrrpodtPeyfn9TB4F+EkDg9ZM+Y0MAAhCAAAQg0FoCx44dM7/85S9TP7fddlvyjbSbN2828hbbqocIPBFQRY8v3P7cwAs8PgOPz8Arej30u53c2SRypMjBW2jTlEL4pXt3uyaUHW+hdfeL8OOAQGwEEHixrQj5QAACEIAABCDQagJF3kIbMkEEXvoz9RB4CLyQa6ifbUMkCgIvvVIh/NK9u10Tyg6B5+4XBJ7Lg1IcBBB4cawDWUAAAhCAAAQgMCAEEHhp4Vb3Z/Eh8BB4bfnnIkSiIPDSqxrCL9272zWh7BB47n5B4Lk8KMVBAIEXxzqQBQQgAAEIQAACA0IAgYfAi/GLK+yc+Bba5v6xCZEoCLz0uoTwS/fudk0oOwSeu18QeC4PSnEQQODFsQ5kAQEIQAACEIAABDIJ8BbatBDkDjzuwMu8WCKsDJEoCLz0AobwS/fudk0oOwSeu18QeC4PSnEQQODFsQ5kAQEIQAACEIAABDIJIPAQePbdc3U85g68zEutJ5UhEgWBl16CEH7p3t2uCWWHwHP3CwLP5UEpDgIIvDjWgSwgAAEIQAACEIBAJgEEHgKvDmlnx0DgZV5qPakMkSgIvPQShPBL9+52TSg7BJ67XxB4Lg9KcRBA4MWxDmQBAQhAAAIQgAAEMgkg8BB4tnyr4zECL/NS60lliERB4KWXIIRfune3a0LZIfDc/YLAc3lQioMAAi+OdSALCEAAAhCAAAQgkEkAgYfAq0Pa2TEQeJmXWk8qQyQKAi+9BCH80r27XRPKDoHn7hcEnsuDUhwEEHhxrANZQAACEIAABCAAgUwCCDwEni3f6niMwMu81HpSGSJREHjpJQjhl+7d7ZpQdgg8d78g8FwelOIggMCLYx3IAgIQgAAEIAABCGQSQOAh8OqQdnYMBF7mpdaTyhCJgsBLL0EIv3TvbteEskPgufsFgefyoBQHAQReHOtAFhCAAAQgAAEIQCCTAAIPgWfLtzoeI/AyL7WeVIZIFAReeglC+KV7d7smlB0Cz90vCDyXB6U4CCDw4lgHsoAABCAAAQhAAAKZBBB4CLw6pJ0dA4GXean1pDJEoiDw0ksQwi/du9s1oewQeO5+QeC5PCjFQQCBF8c6kAUEIAABCEAAAhDIJIDAQ+DZ8q2Oxwi8zEutJ5UhEgWBl16CEH7p3t2uCWWHwHP3CwLP5UEpDgIIvDjWgSwgAAEIQAACEIBAJgEEHgKvDmlnx0DgZV5qPakMkSgIvPQShPBL9+52TSg7BJ67XxB4Lg9KcRBA4MWxDmQBAQhAAAIQgAAEMgkg8BB4tnyr4zECL/NS60lliERB4KWXIIRfune3a0LZIfDc/YLAc3lQioMAAq/mdXj/0CHz1o9/bN64fpH57X/772bXhZ8wuz5+4ejPhZ9I6uQ5aSNtOSAAAQhAAAIQgMBkBBB4CLw6pJ0dA4E32RVX73MhEgWBl2Yfwi/du9s1oewQeO5+QeC5PCjFQQCBV8c6jIyYkeFhc3LbNnP0nnvNH7821+y/9FKzZ/pnJ+TdmMSTOnlO2khb6SN9zchIHZkQAwIQgAAEIACBASOAwEPg2fKtjscIvOb+kQiRKAi89LqE8Ev37nZNKDsEnrtfEHguD0pxEEDgVVyH82fPmpFjx8y7W7eaP914o9n71//ZvPYf/jwl7sbvwhsTedJG2kof6SsxJBYHBCAAAQhAAAIQsAkg8BB4dUg7OwYCz77Cevs4RKIg8NJrEcIv3bvbNaHsEHjufkHguTwoxUEAgVdxHRJ59/xm84dZs8ze//TX5jef+vSU8k5lnrSVPtL33ec3JxKvYjp0hwAEIAABCEBgwAgg8BB4tnyr4zECr7l/JEIkCgIvvS4h/NK9u10Tyg6B5+4XBJ7Lg1IcBBB4Fdfh5IsvmkPfutXsmT7dvPbnFxWWdyrxpI/0lRgSq9fHqVOnzNy5c82SJUt6PRTxB4DAsWPHzIwZM8y2bdsGYDZMAQIQgEA7CSDwEHh1SDs7BgKvuX8LQiQKAi+9LiH80r27XRPKDoHn7hcEnsuDUhwEEHgV1uH8e++Z4ZUrzb7/8nnz2kWfDJZ34xLvok8mMSSWxOzlgcDrJd3Bi43AG7w1rWtGujemTZtm7B8RvvIcBwQgUB8BBB4Cz5ZvdTxG4NV3fU4VKUSiIPDSNEP4pXt3uyaUHQLP3S8IPJcHpTgIIPAqrMOJp54yr1+9wPzmM3/hftusfuts0fOFn0hiSCyJ2cujqsDbu3evueSSS4yce3nIHYKDcpdgU8x6sR4qabgDrxd02x2TvdHu9SP7dhFA4CHw6pB2dgwEXnP/BoRIFAReel1C+KV7d7smlB0Cz90vCDyXB6U4CCDwKqzDm0uXmt/93d+XvvNO78DTs8SSmL08EHi9pJsdG4GXzYXadhNA4LV7/ci+XQQQeAg8W77V8RiB19y/ASESBYGXXpcQfune3a4JZYfAc/cLAs/lQSkOAgi8Cutw4P/5f5MvoVABV/UsX2ghMXt5IPB6STc7NgIvmwu17SbQ5n3dbvJk30UCCDwEXh3Szo6BwGvuX5IQiYLAS69LCL90727XhLJD4Ln7BYHn8qAUBwEEXoV12P1//J/mtU9/prY78CSWxKzzeOSRR5zPp1q5cmXml1io2LM/y8p+C2vW89JW4suhb3mVF/XTp08fH1Oft+ekd+7oWPZnZslbNbVez/bzdhx5rLGknz1XyUFy0UOe9+vkOc3Xf4uo1msO8sUfwi4rF5m7tpOzxpqKmebmn4uMbc9bx9f10ufsnPQ5HUvK8pM1luSth8aaiq+259wdArJ3mng7fXeIMlMI5BNA4CHwbPlWx2MEXv71VvczIRIFgZemH8Iv3bvbNaHsEHjufkHguTwoxUEAgVdhHarecZfXv0JKTlcRNLZwsoWSLXRU4th1Km5EXNlCJ+9Fu/QVYWS3F+ljCy1JTseyxZ6fp7STOjsfZ2JWQfOUcSaLKbkUFXhZeasczOJZZM5FRUfRsXXeX/7yl8eFoWKRXCWOHnnMJafFixePr6/GtOejdVPx1bE4d4eA7lXZG/pT5JrtDiFmCoH6CCDwEHh1SDs7BgKvvutzqkghEgWBl6YZwi/du9s1oewQeO5+QeC5PCjFQQCBV2Ed8gRc1foKKY13zRNtKmTsF9p5skzFjy+DsmSUxLDlliSiwtAfyxZE0k5z8gWc3W98Yt4D7eu39XOXORQReJqznYsO6c8xL6a0s+eYtxYaV88hY+fNW2P5Z8nJZuTPRdv73PLG8dtpf87dJaB7wt773aXBzCFQLwEEHgLPlm91PEbg1XuNThYtRKIg8NIkQ/ile3e7JpQdAs/dLwg8lwelOAgg8CqsQ1VRl9e/QkrjXUVAZb2QVkmkMkcFjS3pNIjfVurzZJQviDSG1GseeWNljZMXT+PqOS+m1quIy5NtKh10/n5Zx5GzxLIlZV6Ofrs8ZnZseRwytj8/P5Yy1Tuj5KzrIG3zcvfjaln56Dhar3y1nnO3CUy2h7tNhtlDoBoBBB4Crw5pZ8dA4FW7JkN6h0gUBF6abAi/dO9u14SyQ+C5+wWB5/KgFAcBBF6FdYj5M/BscWZPUcWOPC/HZC+4/bbaPu8OPI1pj2fnoWPZUsl+bPeXx3bZjmk/VpE0lWAqKvDy2smYtphTNnb+9mNb9Mm8s5jZ85DHRceWtnnzlueEm+RiyzWpQ+D5xCnXTUD3pb336h6DeBDoIgEEHgLPlm91PEbgNfcvSYhEQeCl1yWEX7p3t2tC2SHw3P2CwHN5UIqDAAKvwjrE/C208gLaFjY6TRVPKsf0BbcvwKS931bq8mSUxNOYOpacbXE02Vh2H+2XFc9vlxdT61Uk5OWtUlHnHyLR8ubs55g3tt8uZGydn+atsfJi2OsgbfNy9+P6ZR1H65Wv1nPuNgH2RbfXv8uzP336tHn44YfN1772NTM0NGSuueYa88ILL5hz587VggWBh8CrQ9rZMRB4tVyahYKESBQEXhppCL90727XhLJD4Ln7BYHn8qAUBwEEXoV1eHPpUvO7v/v72r6FVmJJzDqOPJGjwkrlWJak0/G1rS2J8mRUnhCyxdFkY+mYes6Lp8/rWYWBnaM8p/UqmLLmIu3keblbTfv7/XQczd2+s0762mVt65/zmPntQsbWtpq3xpKy/1l/2tYWusI3K3e/v/b1x9F65avjc+42AX//dJsGs+8KgZGREbNixQpz+eWXm/vvv99s3rzZLFu2zMyaNcts2LDBnD9/vjIKBB4Cz5ZvdTxG4FW+LAsHCJEoCLw01hB+6d7drgllh8Bz9wsCz+VBKQ4CCLwK63DiqafM61cvML/5zF+YXRd+orzIu/ATSQyJJTHrOLKEk9aJsBKBo4fKLbtOBY1dJ+213hc30s5vK+2lzhZH8gLff3untFu1alUSW3MqKsc0n6kEk87dllY6b1vgyfgydl6d3V/Htucn/SUXOx9t5zPTudrn0LHtcSSOzskeS9ZA5mPnqXX2mmlfu05z98fRenscex48HnwCGzduTPabzjRr/+hznCEwyAReffVVM3v2bLN+/frxaYrUW758ubn++uvN22+/PV5f9gECD4FXh7SzYyDwyl6N4f1CJAoCL803hF+6d7drQtkh8Nz9gsBzeVCKgwACr8I6nH/vPTO8cqXZ918+b1676JOlBZ70lRgSS2LWdai0EnkjP3Jn1s6dOxORY0saGU+FjLaVc56cUclkt5F4fkyJK3W2OJI6faE/2Vh2PrY089louyKCSdvquJKX8BAufn97jjpPqfNz8RlLW3++krMdTx5Pdtht88bWufh5S1yp0znKWcr+Ouh6ZY1l55Y3jtZPNRc7Fo8Hi4C/d3SvDdYsmQ0Epibw5JNPmkWLFpnXX3/daSx341177bXmrbfecurLFBB4CDxbvtXxGIFX5kos1ydEoiDw0oxD+KV7d7smlB0Cz90vCDyXB6U4CCDwKq7DyRdfNIe+davZM326ee3PLwqWeNJH+koMicURLwERFllyromM6x5bBV4TuTMGBCAAgS4ReP/9981zzz2X/L6Qu8vPnj1befoIPAReHdLOjoHAq3xZFg4QIlEQeGmsIfzSvbtdE8oOgefuFwSey4NSHAQQeBXXYeTYMfPu85vNH2bNMnv/01+b33zq04UlnrSVPtJXYkgsjjgJ6J12Ir6aPnoxNgKv6VVkPAhAoAsE5G208iUW8nPHHXckXwZVx7wReAg8W77V8RiBV8eVWSxGiERB4KWZhvBL9+52TSg7BJ67XxB4Lg9KcRBA4FVch/Nnzybi7d2tW82fbrzR7P3r/2xe+w9/PqXEkzbSVvpIX5F3Eouj/wTkc4vkbb72IcLLf/us/Xxdj5saG4FX14oRBwIQgMAEgVdeecVs3bp1/Estli5dak6ePDnRoOQjBB4Crw5pZ8dA4JW8GEt0C5EoCLw04BB+6d7drgllh8Bz9wsCz+VBKQ4CCLw61mFkxIwMD5uT27aZo/fca/74tblm/6WXmj3TP5sSeVInz0kbaSt9pK8ZGakjE2LUQCDrs72aeutsU2Mj8GrYKISAAAQgMAkB+TbamTNnmieeeGKSVsWeQuAh8Gz5VsdjBF6xa6+OViESBYGXJh7CL9272zWh7BB47n5B4Lk8KMVBAIFX8zq8f+iQeevHPzZvXL/I/Pa//Xf322kv/ERSJ89JG2nLAQEIQAACEIDA4BEYHh42CxcuNHfffXfm5OSFQdEfEXh79uwp3P5zyzaZjy5MS69BqvvKvS8U5iH8du/eXbj94asuMHVIsphjvLl2QWEeIfy2791ubn3u1oH/eejlh3rC78BDD6f+5/+uj184cHV/2LKlJ/yK/pvalXYh164wWXvrTWbZl7440D/3zB0K2nuZv8CphEAfCSDw+gifoSEAAQhAAAIQaDeBNWvWmBtvvNG8/fbbzkQOHTpkFixYYO677z6nXgvHjx83RX9U4BVt//nvPTPwAm/m6u2F+Ym8O3jwYOH2R+Z/eOAF3rF1iwrzCOH364O/Nhetumjgfx7d9WhP+B1a99jAybosATm8Y0dP+BX9N7Ir7UKuXWHy6LJbBlreiZxccdXsoL2nv7M5QyAWAgi8WFaCPCAAAQhAAAIQaB2BTZs2JW+VlbfM6nH+/HmzYcMGM2vWLLNlyxatLn0WgScCqujxhdufG3iBd8UDLxfFkdxtIS9Oix5dEHi9egvtgXcODLy8E0H589/9vOh2Ctp/vIU2jVXuDAu5ftMRulsTyo630Lp7RfhxQCA2Agi82FaEfCAAAQhAAAIQaA0B+ZIK+bKKOXPmmPvvvz/5Egt526zIuxUrVpiRGj7jFoGXfjswAu8Dle4SROBVu0sQgVftbb3v7dtX+N/4UAlVOHAHGoayQ+C5mwKB5/KgFAcBBF4c60AWEIAABCAAAQi0lMCpU6eMvJVWJN7Q0JC55pprzM9//nNz5syZWmaEwEPg1f15egg8BF7WW1ubqkPg1fKrYcogCLz05/mtuvqKKblpAwSekuAcEwEEXkyrQS4QgAAEIAABCEDAI4DAQ+Ah8KoJt7o/l4878LgDz/tnOsoiAg+BF+XGJKlKBBB4lfDRGQIQgAAEIAABCPSWAAIPgYfAQ+A1dXdcE+NwB15vf2dodAQeAk/3AufBIYDAG5y1ZCYQgAAEIAABCAwgAQQeAg+Bh8BrQqw1NQYCr5lfVAg8BF4zO41RmiSAwGuSNmNBAAIQgAAEIACBQAIIPAQeAg+B15Rca2IcBF7gL4GSzRF4CLySW4duERNA4EW8OKQGAQhAAAIQgAAEEHgIPAQeAq8JsdbUGAi8Zn6vIfAQeM3sNEZpkgACr0najAUBCEAAAhCAAAQCCSDwEHgIPAReU3KtiXEQeIG/BEo2R+Ah8EpuHbpFTACBF/HikBoEIAABCEAAAhBA4CHwEHgIvCbEWlNjIPCa+b2GwEPgNbPTGKVJAgi8JmkzFgQgAAEIQAACEAgkgMBD4CHwEHhNybUmxkHgBf4SKNkcgYfAK7l16BYxAQRexItDahCAAAQgAAEIQACBh8BD4CHwmhBrTY2BwGvm9xoCD4HXzE5jlCYJIPCapM1YEIAABCAAAQhAIJAAAg+Bh8BD4DUl15oYB4EX+EugZHMEHgKv5NahW8QEEHgRLw6pQQACEIAABCAAAQQeAg+Bh8BrQqw1NQYCr5nfawg8BF4zO41RmiSAwGuSNmNBAAIQgAAEIACBQAIIPAQeAg+B15Rca2IcBF7gL4GSzRF4CLySW4duERNA4EW8OKQGAQhAAAIQgAAEEHgIPAQeAq8JsdbUGAi8Zn6vIfAQeM3sNEZpkgACr0najAUBCEAAAhCAAAQCCSDwEHgIPAReU3KtiXEQeIG/BEo2R+Ah8EpuHbpFTACBF/HikBoEIAABCEAAAhBA4CHwEHgIvCbEWlNjIPCa+b2GwEPgNbPTGKVJAgi8JmkzFgQgAAEIQAACEAgkgMBD4CHwEHhNybUmxkHgBf4SKNkcgYfAK7l16BYxAQRexItDahCAAAQgAAEIQACBh8BD4CHwmhBrTY2BwGvm9xoCD4HXzE5jlCYJIPCapM1YEIAABCAAAQhAIJAAAg+Bh8BD4DUl15oYB4EX+EugZHMEHgKv5NahW8QEEHgRLw6pQQACEIAABCAAAQQeAg+Bh8BrQqw1NQYCr5nfawg8BF4zO41RmiSAwGuSNmNBAAIQgAAEIACBQAIIPAQeAg+B15Rca2IcBF7gL4GSzRF4CLySW4duERNA4NW8OO++9Z7Z9cIbZtPqXeahb/zSLJ+1ydwxc/RHHkudPCdtpC0HBCAAAQhAAAIQmIwAAg+Bh8BD4DUh1poaA4E32b/49T2HwEPg1bebiBQLAQReDStx7uw5c/KdM+bAvx412zfsNxuX70hE3Yqrnx+XdyrxpE4knrSRttJH+koMDghAAAIQgAAEIOATQOAh8BB4CLym5FoT4yDw/H/le1NG4CHwerOziNpPAgi8ivRFvL379hnz218dNk+u2mVWXbfZLJ/1VErcqcDTs7SRttJH+koMJF7FxaA7BCAAAQhAYAAJIPAQeAg8BF4TYq2pMRB4zfyiQuAh8JrZaYzSJAEEXkXacvfc/leOmH/69r+Y++Y/b+68fGp5pxJP2kof6SsxJBYHBCAAAQhAAAIQsAkg8BB4CDwEXlNyrYlxEHj2v/C9e4zAQ+D1bncRuV8EEHgVyR949WhyF10i72YXl3fjEm/2qMSTO/EkVq+PU6dOmblz55olS5b0eijit4CA7AP2QvZCPfLII8m1ItcMBwQgAIF+EkDgIfAQeAi8JsRaU2Mg8Jr5jYLAQ+A1s9MYpUkCCLwKtOVLKLav35+8FTbkzjuVd3qWvvJ2WonV6y+2QOBVWPAB7IrAy19UBF4+G31GGE2bNm38BxmsZDhDoF4CCDwEHgIPgdeUXGtiHARevb8j8qIh8BB4eXuD+vYSQOBVWDv5Jln5Mooin3mnsi7vLDEklsTs5VFV4O3du9dccsklRs69PAZJLDXFrMx6DBLnMvOXPseOHTMzZsww27Ztc0Ig8BwcqYLPR/b59OnTuaMzRYoKCFQngMBD4CHwEHhNiLWmxkDgVf+9UCQCAg+BV2Sf0KZdBBB4FdZr0+pdyTfK5km50Hr5dlqJ2csDgddLutmxEXjZXGKpReCVW4mdO3ca/+3FIvVEhgpTDgh0iYBcC2vWrDFXXnmlGRoaMnPmzEnK/jVSlgkCD4GHwEPgNSXXmhgHgVf2t0FYPwQeAi9sx9C6DQQQeBVWSYTbiqufn/IbZ4uKPIklMXt5IPB6STc7NgIvm0sstan4DA4AACAASURBVAi8+lZC7mKUu/Bkz3NAoCsETp48aZYuXZpIu/vvv9/88pe/NHK+/PLLk3p5vuqBwEPgIfAQeE2ItabGQOBV/a1QrD8CD4FXbKfQqk0EEHgVVmv5rE21yTuVfBKzzsP/jKqVK1dmfomFir28z7PKel7aSnw59K2Y+jY6jaPP23NSYaJt7Dt2RABovZ7t5+048lhjST97rr5EyBMLmq88bx9arznIF38Iu6xcZO7aTs4aaypm9nj6OI+jjC/x7MOer4zrz1nbSj52fnY7HU/b2jnrPJSxHUP6+YffTlht3LgxMy8/96x94sfXsqyNvo07j7229XOSOdi5+/3t5yUn4S5//MhcdP52fx2H8ygB2TP2/oILBLpAYMuWLYms27p1qzPdzZs3m1mzZhl5vuqBwEPgIfAQeE3JtSbGQeBV/a1QrD8CD4FXbKfQqk0EEHgVVkulW93nCik5XUU02MLJljO2hFBZZdep+PDFkS1P7MGkrwgOu72KIxVB0l7HsoWNn6e0kzo7H3ss+7HmKWNPFjNPLGg+do5ZeUtsGSOLZ5E5q3Cyc896LHOWtosXLx4XdjpHexypk7a21JOynZ/E17zt+YlUk3nLIX3kRw7dH1kx7P7KzOadVaccfaHj55nVN0ko5z/SXhjJj51X1lylzm6TNZbytdvJ0BrP5pHVPyfNTlbL2tr7tJMQmHTnCPzoRz8y1113nRkeHnbm/qc//clcffXVZv369U59mQICD4GHwEPgNSHWmhoDgVfmN0F4HwQeAi9819AjdgIIvAorVLe403gVUhrvqpJDzvahskKljTwnj+2ytldZYYuNvLjS3xYdEkOFkB1bHvsv8DUnWwhJO7uf5uSfta/f1s9d5uCLJInlt9Oc7Vx0TBnDnmNeTGlnzzGPmca1z/4Y+pyfp9bbZ3+cIn1kPPnRedvzs2P7j7Wf1vtlrReONve8nKRd0bE1Rt4a2ew1D/vs56p7yN7n0t7PXWNI/6nG0LZdOufx6hID5goBm8COHTvM7NmzzaZN1e+sR+Ah8BB4CLym5FoT4yDw7N8WvXuMwEPg9W53EblfBBB4FcjH/BZaeTGdJRlU1IiEkCNPXshzflup8yVREmQSCWjLjryxssbxJYuO45/zYmq9Sp482aYySOWNX7bHk1i2ZMrL0W+Xx8yOrY/zYvrz0faSt761U886Fz8P7WOfdTw523Oz28hjXSMdQ866v/Jyk34+d8lJ+9lj2O2yxrJzm4xn1pyz4tk5aP7KTfPKyzVrDO3TxbPN12fYRR7MGQJCQD8Xb+HChebQoUOVoSDwEHgIPAReE2KtqTEQeJV/LRQKgMBD4BXaKDRqFQEEXoXlivlLLETI2JJCp6kvtuV5OSYTVn5bbZ/1dlAVQTqOnu08dCxbAtmPNSfpmxdP4+o5T75ovcgWOWxBpH3lrDmpeMhrJ21tcaNs7Pztx0WFk52LPM6btz8fzdsfR+5207nY7P1xtCxtNG/tp8/pWdsoS6m3Y2suWf19nhpLx7TP9p16OnbWWcbL2oPS1l4jzVPGyMtd2ihbP3/pk3UN+WNk5diVOmVn78OuzJ15QiCPwOnTp80//uM/Jp+Lt3379rxmQfUIPAQeAg+B15Rca2IcBF7Qr4DSjRF4CLzSm4eO0RJA4FVYmk2rdyXfGqtvfa16FiEoMes48uSDiicRKXLoC3BfXshzflupy5MnEk9jJoHH/iN1KkEmG8vuI4/z4vnt8mJqvYqbvLx9+eQLJ3s8X9wUzTFvbDu2Ps6LqfOR/LLWRfr7c8nbAzqWnHU8aSuiy98HeTykn66rP64d3+9fJCe7f9bjyXja8f2xNZadu9TZbLWNnO1Yfj3CamK/6T6wGfEYAl0l8O6775rvfOc7ibx79tlnzfnz53NRyAuroj8i8Pbs2VO4/eeWbTIfXZiWXoNU95V7XyjMQ/jt3r27cPvDV11g6hZmscV7c+2CwjxC+G3fu91ctCou2daLfB56+aGe8Dvw0MOmCYHW7zH+sGVLT/gV/Te1K+1Crl1hsvbWm8yyL6Wl1yDV3TN3KGjv5f4S5wkI9IkAAq8C+F0vvGE2Lt9hls96qvK30UoMiSUx6zjy5IXKFpEYcuTJIHlO20osPfLkiYogbadnW5ZMNpa213NePH1ez3nyRetFwsiRNRepl+dtceX303E0d1vcSF+7rG39cx4zv52UZd5ZMe311FykrX34c7H72O3sxzZnv7+0y4qhjFTc5OUj/SW+fWddVjw7nyKPdS0lX/vw88gay89d+mudtLcPia9z9Ouz1shu04XHsrZZfLowd+YIgSwCb775prnxxhuT60JeNE11HD9+3BT9kXgi8Iq2//z3nhl4gTdz9fbCPETeHTx4sHD7I/M/PPAC79i6RYV5hPD79cFfd0LgPbrr0Z7wO7TusU4IvOEdO3rC7/l/es2svPb5gf8p+rsg5NqVmI8uu2XgBd6Kq2YH7b2pfpfzPASaJoDAq0D83bfeM9vX7zerrtts7ry8vMSTvhJDYknMOg6VGbZo0DoRVrb8USFi16nUsOskL6335Ym089tKe6mzX+SLJPHf0ijtVq1alcTWuUt8O3et98+ajy9ftF7z1LnbMXXetsCT+NInr87ur2PY85P+koudj7bTXPw52GXhVWR9pJ2dS95cpJ0t0GSsvG+hled0fM1f49q5axt73tLeX1ets8fPWgcZV8aRvIocmpMdV3O367TdVLlrTjIv+5B+9hz1Oam32Wt9l866p3WfdGnuzBUCWQQOHDhg5PPu5Ece132IwBMBVfT4wu3PDbzAu+KBqSWp8pI7S+TFadGjCwLv5OM3F8WR3K1SlN+Bdw50QuD9/Hc/7wm/40/8ohMCr1dvod2+YX/lmyqqvqOq1/3lpo+iR+i/fRu+v2TgBd6qq68oii/5t69wYxpCoCECCLyKoA+8etQ8uWqXuW/+8+bO2eEST/pIX4khseo8VEyIWJEfkRs7d+5MpIQvK/QFubb1ZYydlwgMbadyROL5MaWP1PkSRMWKxsgay85nMlmi7XyRoPWan+SidTqu5CU8hIvf356j5id1fi4+Y2nrz1fGtuPZOdlclZcws9vr+HZbf9yQudj5Za2b1MmYcpZDRZzUyY+U5Tk7TlY7aSNtbammc9AxNKbPVdtlnWX/yGfgSWzpN1mMMrnrvGUN/DlKPlIfkm/WHNpe519LugZ6VoZtnyf5Q6AIgSNHjpjFixcnP/K4FwcCL/12YATeByrdJYjAq/Y2XwTehZVEIwJvU2nRiMCr9hZfBF4v/kohZpMEEHgVaZ9854zZ/8oR80/f/pdRiRdwJ57ceSfyTvpKDInFES+BPKFTZ8YiPgZJfqhkE+FT16ECT84cEIAABPpJQD7jbs2aNWZoaCj5t3vFihXG/3nllVcqp4jAQ+DV/Rl6CDwEXj8/Bw+Bh8Dr1+fqIfAq/0lCgD4TQOBVXIBzZ8+Zd98+Y377q8PJXXTyVtgin4knbaSt3HknfSWGxOKIk4De8dZruTZoAk/mk3UXW5VVRuBVoUdfCECgTgLytsIbbrghEXgi8bJ+1q9fX3lIBB4CD4FXTbjV/UUW3IHHHXi9fqtsXnzuwOMOvMp/VBCg1QQQeDUsn4g3uXvuwL8eNfLZC/JlFPKNsiuufj51e7TUyXPSRtpKH+mLvKthIWoKsXz58uQz2exwIqKaeNtkWwWe3J0od9vZh9RlvX3WblPmMQKvDDX6QAACbSaAwEPgIfAQeP28Y67usbkDjzvwuAOvzX+VkHs/CSDwaqYvX0Ih3yS7afWuRNQtnzXxD7Q8Fnknz0mbur6wouYpdD6ciCf9LC89130XWR7ktgo8/7PmhFuvhCcCL2/3UA8BCAwqAQQeAg+Bh8CrW6L1Mx4Cb+L1Yd6ddnn13IHHHXiD+rcO8ypGAIFXjBOtIAABCEAAAhCAQF8IIPAQeAg8BF4/hVvdYyPwEHjcgdeXPycYdAAIIPAGYBGZAgQgAAEIQAACg0sAgYfAQ+Ah8OqWaP2Mh8BD4CHwBvdvFmbWWwIIvN7yJToEIAABCEAAAhCoRACBh8BD4CHw+inc6h4bgYfAQ+BV+rOAzh0mgMDr8OIzdQhAAAIQgAAE4ieAwEPgIfAQeHVLtH7GQ+Ah8BB48f/tQYZxEkDgxbkuZAUBCEAAAhCAAAQSAgg8BB4CD4HXT+FW99gIPAQeAo8/cCBQjgACrxw3ekEAAhCAAAQgAIFGCCDwEHgIPARe3RKtn/EQeAg8BF4jfz4wyAASQOAN4KIyJQhAAAIQgAAEBocAAg+Bh8BD4PVTuNU9NgIPgYfAG5y/UZhJswQQeM3yZjQIQAACEIAABCAQRACBh8BD4CHw6pZo/YyHwEPgIfCC/gygMQTGCSDwxlHwAAIQgAAEIAABCMRHAIGHwEPgIfD6KdzqHhuBh8BD4MX3twYZtYMAAq8d60SWEIAABCAAAQh0lAACD4GHwEPg1S3R+hkPgYfAQ+B19A8apl2ZAAKvMkICQAACEIAABCAAgd4RQOAh8BB4CLx+Cre6x0bgIfAQeL37m4HIg00AgTfY68vsIAABCEAAAhBoOQEEHgIPgYfAq1ui9TMeAg+Bh8Br+R8mpN83Agi8vqFnYAhAAAIQgAAEIDA1AQQeAg+Bh8Drp3Cre2wEHgIPgTf1735aQCCLAAIviwp1EIAABCAAAQhAIBICCDwEHgIPgVe3ROtnPAQeAg+BF8kfGKTROgIIvNYtGQlDAAIQgAAEINAlAgg8BB4CD4HXT+FW99gIPAQeAq9Lf8Uw1zoJIPDqpEksCEAAAhCAAAQgUDMBBB4CD4GHwKtbovUzHgIPgYfAq/kPBcJ1hgACrzNLzUQhAAEIQAACEGgjAQQeAg+Bh8Drp3Cre2wEHgIPgdfGv0bIOQYCCLwYVoEcIAABCEAAAhCAQA4BBB4CD4GHwKtbovUzHgIPgYfAy/mFTzUEpiCAwJsCEE9DAAIQgAAEIACBfhJA4CHwEHgIvH4Kt7rHRuAh8BB4/fyrgrHbTACB1+bVI3cIQAACEIAABAaeAAIPgYfAQ+DVLdH6GQ+Bh8BD4A38ny5MsEcEEHg9AktYCEAAAhCAAAQgUAcBBB4CD4GHwOuncKt7bAQeAg+BV8dfB8ToIgEEXhdXnTlDAAIQgAAEINAaAgg8BB4CD4FXt0TrZzwEHgIPgdeaP0FINDICCLzIFoR0IAABCEAAAhCAgE0AgYfAQ+Ah8Pop3OoeG4GHwEPg2b/leQyB4gQQeMVZ0RICEIAABCAAAQg0TgCBh8BD4CHw6pZo/YyHwEPgIfAa/1OCAQeEAAJvQBaSaUAAAhCAAAQgMJgEEHgIPAQeAq+fwq3usRF4CDwE3mD+vcKsek8AgVcz4xPDR82rTz1hHr/rdrN6wRzznb/7W6P/QMnj1QuuMI/f9f2kjbTlgAAEIAABCEBgcAjs27fPXH/99Wb//v21TQqBh8BD4CHw6pZo/YyHwEPg6evjps+rrr6i8O/mN954o3BbGkKgKQIIvBpInx0ZMe++dczs/9W/mC1rHzT/vORm88Nrvmru/P/+73F5p/84Sd0Pr5mTtJG20kf6SgwOCEAAAhCAAATaS+C3v/2tWbhwoZk7dy4Cb2Faun20xrorHni58EaRF2HHjx8v3P7I/A+buoVZbPFOPn5zYR4h/A68c8BctCou2daLfH7+u5/3hN/xJ35h+inWmhobgYfA09fGTZ8ReIX/6aJhpAQQeBUXRsTbiWPDZs/2reZny79r/nH2l813/v6/psSd/4+TtJG20kf6SgwkXsXFoDsEIAABCECgDwROnz5t1q9fb+bMmWOGhoYQeDWKuv/d3v2/2lHmdwD/qT90/4nib/2hf4I/FkrpdtGf1t2lUMpujF9aydokQlkoC7WGRqul2VBTi27VtSalWozN+iWUFK1kG0XEFhOwEjCsbo00WyMhmqc8V4753JNz45l5zpn7mXNeA4f73HPnmfnM65mczLyZmbNV6CfA+5WmkFGA1xYyCvB+vSloFOAJ8KbPjYf6XYC3DQdJVrlQAQFeI2e9eu7Mz14rT/5gdznw3W+XB771ja8M7yYfUHXe2qf2rcuoyzIRIECAAAEC4xKo4V0N7u69997y4osvCvAEeE3h2hBX6wnwBHhDXW03az0CPAHe5Hx46J8CvHEdX6n2WgEB3rUmnd6pt8DWq+g2wrtb5g/vJh9WD9zyRYhXl1GXlXU6c+ZMuemmm0r9aSJAgAABAgSuCrz00kvl6NGj5dKlS+XUqVMCPAGeAG/Fb6N1BZ4r8P761v4hXEvfAzuPX/3P5ytaXW5/r4t67sF9c1+IMjmXHdtPAd5X7DT+nF5AgNcwRPVLKF45/MTGrbBdrryb/qCrfevttK8efqIs44stXnvttXLDDTdsetXn81y8eHHurRfgzU1lRgIrL1A/D2688cZNnynxM6br58vKg9nAtRIQ4C332XeTW2rdQusW2mU8227eZQrwBHgtIVxLXwHe7zSFjAK8tTokW8mNFeA1DGv9ttn6hRXzPPNuOrSb/r0uoy6rLnNRUw3o6ol0PdGevnJu3759GyffNdyL0/nz58vNN99cpt8X4EUlbQIEZgls9fkxa17vEVhVAQGeAG+IW2Bb1+EWWrfQzrq1daj33ELb/+o9AZ4Ab1WPn2zXfAICvPmcZs517OCDG98oOx3G9f29fnPtsYMPzVxXnzdrSFfDuHpSPWt6+umnrwn3tjoBF+DNEvQeAQJRoH7muPouimivo8C8AV69tWne1+uvv15Onz499/y/uf/lMrlSbVV//sHfvjK3R/V755135p7/w+//avpbYFsDvJ8f3j23Rxe/k2dOrsW30D71+lNL8XvvqX9o+nKIoQK41vWcffXVpfi9/JM3S8vVbWPoWwO8Lv93dPnsO/znf9p0dVvfc+Ah+x26a0cnv3U8jrHNuQUEeA3j89juO8qPvvedhX3Q1WU9tvvOhoqudp3c4jZ9Jd3VOcrGLbT1ZLuedE8mAd5Ewk8CBLoIbPXZ0WUZ5iWwCgLzBngXLlwo874mAd688//WX/7rygd4tz52cm6/egJ77ty5uef/xd1fW/kA7/yzfzK3Rxe/t8+9vRYB3jP/+cxS/D549p/XIsD76M03l+L3b//4X2sR4M37f0GXf7t1mc/s/7OFndcOGcp1Wdffff+2TvveKhyX2IbVEhDgNYzn/d/8+sI/5OoyFzHVq+uud/XdZB1xvslttfFZVpNwL16BF+fbah1xnrq86SCx/r2+6vv171stZ1KnnwQI5BaInyW5K1UdgeUKzBvgdamiBng1gJp3+u2HTqx8gOcZeJ6BN+/z6pYxn2fgeQbedl2t5xZat9DOeyxgvtUUEOA1jGuXtL/LvA0lfdm1hmPz3MpWA7T4jLytrqKZXNEXw7jJM/biema9NwnpYohX66vfanvgwIEva9YgQGCcApPPjRrimQisu4AAzzPwWm9vHaK/Z+B5Bl7rbbAt/T0DzzPwupwbL3JeX2Kx7kdp499+AV7DGC7ywyQuq6GkL7suK8CbPkGfDgCnf58UNF1P/d1VdxMdPwmMW6D+u/fvedxjqPrFCQjwBHhDBHCt6xDgCfBaArjWvgI8AV489x2yLcBb3PGOJW2PgACvwT3zLbTTgdlWmzl94j25kqa+H6d4C+30+/UKvsn8db31NT1N3143b33Ty/E7AQL5BLb6d5+vUhURWL6AAE+A1xquDdFfgCfAaw3hWvoL8AR4Q4Z2cV0CvOUfB1nDcgUEeA2+9QsnFv8lFnc0VHS163RgdvUvm1t1vngLbEuAN7l9Nj5DL7bjFTpO+DePg98IjFVgcnv9JMQf63aom0BmAc/AuzYU9Aw8z8BbxrPt5l2mZ+B5Bp5n4LU9iy6GakO2BXiZj3bUNo+AAG8epS3mOXbwofLjPX+4sC+y+PGeO8uxgw9usbZub89zUj0J3OJtsS0BXq1w3mBu3vm6bbW5CRAYWmD6Kt6h1299BNZBQIAnwFv0VXmuwHMFXssVdK19XYHnCrwhQ7u4LgHeOhw1rfY2CvAaxvet4y+Uf9r3w3L/Lb/bHOLVZdRl1WUuaqohWbzqbXq5s67Saw3wZi1zer31dwHeLBXvERifgH/L4xszFY9PQIAnwBPgtQVu815ZN+98rsBzBZ4r8FyBN76jCRWvgoAAr2EUf/nR/5RXDz9R/ua23y8PfOsbvUO82rcu45XDT5S6zEVNkyvs4rfMTpZdT7pnvT/pU/8ep3mfgTcJAONtuXU59SqdeIudk/6oq01gnAKTf+/xKt5xbomqCeQWEOAJ8AR4ArzWq94y9XcFnivw4lVxQ7ZdgZf7eEd1Xy0gwPtqo+vO8e4b/1H+5cAD5cB3v10euKV7iFf71L51GXVZy5hqcBafRVfb0wFdXG+cfzLfvAFeXc4kBIzrnA70BHhRXJvAOAXmuVV/nFumagK5BAR4AjwBngAvUwDXWosAT4A3ZGgX1yXAy3V8o5ruAgK87mabevzfx+fLmZ+9Vp78we4vQrwOV+LVK+9qeFf71mXUZZkIECBAgAABAlFAgCfAE+AJ8FpDs0z9BXgCvBiqDdkW4MWjC+0xCgjwGkfts8uXyy/Pf1ROn/z3javo6q2w8zwTr85T561X3tW+dRl1WSYCBAgQIECAQBQQ4AnwBHgCvEwBXGstAjwB3pChXVyXAC8eXWiPUUCAt4BRq8FbvXqu3gL76uEnN76Mon6j7I++951rnotX36vfXFu/sKLOW/vUvsK7BQyERRAgQIAAgRUUEOAJ8AR4ArzW0CxTfwGeAC+GakO2BXgreJC0ZpskwFvwgNcvoajfJHvs4IPlsd13lPu/+fUvQ7zafmz3neXYwYc25lnkF1YseDMsjgABAgQIEEgiIMAT4AnwBHiZArjWWgR4ArwhQ7u4LgFekgMbZfQWEOD1ptORAAECBAgQILB8AQGeAE+AJ8BrDc0y9RfgCfBiqDZkW4C3/GMWa1iugABvub6WToAAAQIECBBoEhDgCfAEeAK8TAFcay0CPAHekKFdXJcAr+lwROcEAgK8BIOgBAIECBAgQIDAVgICPAGeAE+A1xqaZeovwBPgxVBtyLYAb6sjDe+PRUCAN5aRUicBAgQIECCwlgICPAGeAE+AlymAa61FgCfAGzK0i+sS4K3lYdRKbbQAb6WG08YQIECAAAECqyYgwBPgCfAEeK2hWab+AjwBXgzVhmwL8FbtCGn9tkeAt35jbosJECBAgACBEQkI8AR4AjwBXqYArrUWAZ4Ab8jQLq5LgDeigx+lzhQQ4M1k8SYBAgQIECBAIIeAAE+AJ8AT4LWGZpn6C/AEeDFUG7ItwMtxXKOK/gICvP52ehIgQIAAAQIEli4gwBPgCfAEeJkCuNZaBHgCvCFDu7guAd7SD1msYMkCArwlA1s8AQIECBAgQKBFQIAnwBPgCfBaQ7NM/QV4ArwYqg3ZFuC1HI3om0FAgJdhFNRAgAABAgQIENhCQIAnwBPgCfAyBXCttQjwBHhDhnZxXQK8LQ40vD0aAQHeaIZKoQQIECBAgMA6CgjwBHgCPAFea2iWqb8AT4AXQ7Uh2wK8dTyKWq1tFuCt1njaGgIECBAgQGDFBAR4AjwBngAvUwDXWosAT4A3ZGgX1yXAW7EDpDXcHAHeGg66TSZAgAABAgTGIyDAE+AJ8AR4raFZpv4CPAFeDNWGbAvwxnPso9LZAgK82S7eJUCAAAECBAikEBDgCfAEeAK8TAFcay0CPAHekKFdXJcAL8VhjSIaBAR4DXi6EiBAgAABAgSWLSDAE+AJ8AR4raFZpv4CPAFeDNWGbAvwln3EYvnLFhDgLVvY8gkQIECAAAECDQICPAGeAE+AlymAa61FgCfAGzK0i+sS4DUcjOiaQkCAl2IYFEGAAAECBAgQmC0gwBPgCfAEeK2hWab+AjwBXgzVhmwL8GYfZ3h3PAICvPGMlUoJECBAgACBhAKXL18uzz//fNm1a1fZsWNH2bNnTzl+/Hj5/PPPF1KtAE+AJ8AT4GUK4FprEeAJ8IYM7eK6BHgLOSyxkG0UEOBtI75VEyBAgAABAuMWuHLlSnn88cfLzp07y8MPP1xOnDhR9u/fv/F7DfUWMQnwBHgCPAFea2iWqb8AT4AXQ7Uh2wK8RRyVWMZ2CgjwtlPfugkQIECAAIFRC5w9e3bjyrsjR46UGubVqV6Rd+jQobJ3797ywQcfNG+fAE+AJ8AT4GUK4FprEeAJ8IYM7eK6BHjNhyQWsM0CArxtHgCrJ0CAAAECBMYr8PLLL5e77rqrvPvuu5s24q233iq33357OXXq1Kb3+/wiwBPgCfAEeK2hWab+AjwBXgzVhmwL8PocheiTSUCAl2k01EKAAAECBAiMSuCRRx4p99xzT/n444831V2vvNu9e3d59tlnN73f5xcBngBPgCfAyxTAtdYiwBPgDRnaxXUJ8PocheiTSUCAl2k01EKAAAECBAiMSqA+9+6+++4rFy9e3FR3DfRqsFf/3joJ8AR4AjwBXmtolqm/AE+AF0O1IdsCvNYjEv23W0CAt90jYP0ECBAgQIDAKAXqt8weOHCg7Nu3r3z66aebtkGAd23o9mt7F/feHU+8vsn7er+8//775cKFC9ebZdPffnH318qiA7Nsy/vk2A83bfP1funi997/vld+49FcYdsy6vnpf//0emSb/tbF78ILL5ZMQduyahHgCfCGDO3iugR4mz6e/DJCAQHeCAdNyQQIECBAgEAOgYxX4P3RT94o6/Cadw/oEqDUZV74+99b+dcyA7y9J/aWVX8tM8B7/+4/Lqv+WmaA98Ijb5dVfy3rs+/oX/1FWYdXF7955zUfgaEEBHhDSVsPAQIECBAgsHICfZ+BV0OleV/1Fto33nij1J9eDOwD9gH7gH3APjDMPrBy4h6VrwAAC/FJREFUBy02aPQCArzRD6ENIECAAAECBLZL4HrfQnvbbbeVkydPziyt3tI57+v06dMbAd65c+eKV3eD6ldf7LrbVTN+/dwm+xs/fpN9Yeif9r32fW/mf+DeJLCNAgK8bcS3agIECBAgQGDcAmfPni27du0qR44cKVeuXNnYmMuXL5dDhw6VvXv3lvpttK1T11tAW9e3av35tY0oP35tAm297X/9/dj1t6s9q5+JQDYBAV62EVEPAQIECBAgMBqBzz77rDz66KPl1ltvLQcPHiwnTpwo+/fvLzt37izPPffcl6FeywY5CWvR++IkrMuXWLStbfV62//axpQfvzaB/r3te/3tas/qZyKQTUCAl21E1EOAAAECBAiMSuDSpUvl6NGjG1fi7dixo+zZs6ccP3681G+pXcTkJKxNkR+/NoG23vY/fm0C/Xvb9/rb1Z7Vz0Qgm4AAL9uIqIcAAQIECBAgEASchAWMHk1+PdBCF34Bo0eTXw+00IVfwOjYZNcRbGr26mcikE1AgJdtRNRDgAABAgQIEAgCTsICRo8mvx5ooQu/gNGjya8HWujCL2B0bLLrCDY1e/UzEcgmIMDLNiLqIUCAAAECBAgEASdhAaNHk18PtNCFX8Do0eTXAy104RcwOjbZdQSbmr36mQhkExDgZRsR9RAgQIAAAQIEgoCTsIDRo8mvB1rowi9g9Gjy64EWuvALGB2b7DqCTc1e/UwEsgkI8LKNiHoIECBAgAABAkHASVjA6NHk1wMtdOEXMHo0+fVAC134BYyOTXYdwaZmr34mAtkEBHjZRkQ9BAgQIECAAIEg4CQsYPRo8uuBFrrwCxg9mvx6oIUu/AJGxya7jmBTs1c/E4FsAgK8bCOiHgIECBAgQIBAEHASFjB6NPn1QAtd+AWMHk1+PdBCF34Bo2OTXUewqdmrn4lANgEBXrYRUQ8BAgQIECBAIAh88sknpb5M/QT49XOb9OI3kej3k18/t0kvfhOJ7j/ZdTeLPaqfiUA2AQFethFRDwECBAgQIECAAAECBAgQIECAAIEgIMALGJoECBAgQIAAAQIECBAgQIAAAQIEsgkI8LKNiHoIECBAgAABAgQIECBAgAABAgQIBAEBXsDQJECAAAECBAgQIECAAAECBAgQIJBNQICXbUTUQ4AAAQIECBAgQIAAAQIECBAgQCAICPAChiYBAgQIECBAgAABAgQIECBAgACBbAICvGwjoh4CBAgQIECAAAECBAgQIECAAAECQUCAFzA0CRAgQIAAAQIECBAgQIAAAQIECGQTEOBlGxH1ECBAgAABAgQIECBAgAABAgQIEAgCAryAoUmAAAECBAgQIECAAAECBAgQIEAgm4AAL9uIqIcAAQIECBAgQIAAAQIECBAgQIBAEBDgBQxNAgQIECBAgAABAgQIECBAgAABAtkEBHjZRkQ9BAgQIECAAAECBAgQIECAAAECBIKAAC9gaBIgQIAAAQIECBAgQIAAAQIECBDIJiDAyzYi6iFAgAABAgQIECBAgAABAgQIECAQBAR4AUOTAAECBAgQIECAAAECBAgQIECAQDYBAV62EVEPAQIECBAgQIAAAQIECBAgQIAAgSAgwAsYmgQIECBAgAABAgQIECBAgAABAgSyCQjwso2IeggQIECAAAECBAgQIECAAAECBAgEAQFewNAkQIAAAQIECBAgQIAAAQIECBAgkE1AgJdtRNRDgAABAgQIECBAgAABAgQIECBAIAgI8AKGJgECBAgQIECAAAECBAgQIECAAIFsAgK8bCOiHgIECBAgQIAAAQIECBAgQIAAAQJBQIAXMDQJECBAgAABAgQIECBAgAABAgQIZBMQ4GUbEfUQIECAAAECBAgQIECAAAECBAgQCAICvIChSYAAAQIECBAgQIAAAQIECBAgQCCbgAAv24iohwABAgQIECBAgAABAgQIECBAgEAQEOAFDE0CBAgQIECAAAECBAgQIECAAAEC2QQEeNlGRD0ECBAgQIAAAQIECBAgQIAAAQIEgoAAL2BoEiBAgAABAgQIECBAgAABAgQIEMgmIMDLNiLqIUCAAAECBAgQIECAAAECBAgQIBAEBHgBQ5MAAQIECBAgQIAAAQIECBAgQIBANgEBXrYRUQ8BAgQIECBAgAABAgQIECBAgACBICDACxiaBAgQIECAAAECBAgQIECAAAECBLIJCPCyjYh6CBAgQIAAAQIECBAgQIAAAQIECAQBAV7A0CRAgAABAgQIECBAgAABAgQIECCQTUCAl21E1EOAAAECBAgQIECAAAECBAgQIEAgCAjwAoYmAQIECBAgQIAAAQIECBAgQIAAgWwCArxsI6IeAgQIECBAgAABAgQIECBAgAABAkFAgBcwNAkQIECAAAECBAgQIECAAAECBAhkExDgZRsR9RAgQIAAAQIECBAgQIAAAQIECBAIAgK8gKFJgAABAgQIECBAgAABAgQIECBAIJuAAC/biKiHAAECBAgQIECAAAECBAgQIECAQBAQ4AUMTQIECBAgQIAAAQIECBAgQIAAAQLZBAR42UZEPQQIECBAgAABAgQIECBAgAABAgSCgAAvYGgSIECAAAECBAgQIECAAAECBAgQyCYgwMs2IuohQIAAAQIECBAgQIAAAQIECBAgEAQEeAFDkwABAgQIECBAgAABAgQIECBAgEA2AQFethFRDwECBAgQIECAAAECBAgQIECAAIEgIMALGJoECBAgQIAAAQIECBAgQIAAAQIEsgkI8LKNiHoIECBAgAABAgQIECBAgAABAgQIBAEBXsDQJECAAAECBAgQIECAAAECBAgQIJBNQICXbUTUQ4AAAQIECBAgQIAAAQIECBAgQCAICPAChiYBAgQIECBAgAABAgQIECBAgACBbAICvGwjoh4CBAgQIECAAAECBAgQIECAAAECQUCAFzA0CRAgQIAAAQIECBAgQIAAAQIECGQTEOBlGxH1ECBAgAABAgQIECBAgAABAgQIEAgCAryAoUmAAAECBAgQIECAAAECBAgQIEAgm4AAL9uIqIcAAQIECBAgQIAAAQIECBAgQIBAEBDgBQxNAgQIECBAgAABAgQIECBAgAABAtkE0gR4H374YfFiYB+wD9gH7AP2AfuAfcA+YB+wD9gH7AP2AfuAfcA+sB37QLbQLtaTJsCLRWkTIECAAAECBAgQIECAAAECBAgQIPCFgADPnkCAAAECBAgQIECAAAECBAgQIEAgsYAAL/HgKI0AAQIECBAgQIAAAQIECBAgQICAAM8+QIAAAQIECBAgQIAAAQIECBAgQCCxgAAv8eAojQABAgQIECBAgAABAgQIECBAgIAAzz5AgAABAgQIECBAgAABAgQIECBAILGAAC/x4CiNAAECBAgQIECAAAECBAgQIECAgADPPkCAAAECBAgQIECAAAECBAgQIEAgsYAAL/HgKI0AAQIECBAgQIAAAQIECBAgQICAAM8+QIAAAQIECBAgQIAAAQIECBAgQCCxgAAv8eAojQABAgQIECBAgAABAgQIECBAgIAAzz5AgAABAgQIECBAgAABAgQIECBAILGAAC/x4CiNAAECBAgQIECAAAECBAgQIECAgADPPkCAAAECBAgQIECAAAECBAgQIEAgsYAAL/HgKI0AAQIECBAgQIAAAQIECBAgQICAAM8+QIAAAQIECBAgQIAAAQIECBAgQCCxgAAv8eAojQABAgQIECBAgAABAgQIECBAgIAAzz5AgAABAgQIECBAgAABAgQIECBAILGAAC/x4CiNAAECBAgQIECAAAECBAgQIECAgADPPkCAAAECBAgQIECAAAECBAgQIEAgsYAAL/HgKI0AAQIECBAgQIAAAQIECBAgQICAAM8+QIAAAQIECBAgQIAAAQIECBAgQCCxgAAv8eAojQABAgQIECBAgAABAgQIECBAgIAAzz5AgAABAgQIECBAgAABAgQIECBAILGAAC/x4CiNAAECBAgQIECAAAECBAgQIECAgADPPkCAAAECBAgQIECAAAECBAgQIEAgsYAAL/HgKI0AAQIECBAgQIAAAQIECBAgQIDA/wOpJlN8yS/KIgAAAABJRU5ErkJggg==" alt="image">

Reply:
@erdembayar I think you have the same syntax twice, did you mean to compare between `dotnet nuget explain package newtonsoft.json`and `dotnet nuget explain newtonsoft.json`?

I think explain is a fine term - I have a personal preference towards `why` based on existing commands like `npm why`, `yarn why`, `pnpm why`, `paket why`, etc, but explain definitely usable. Out of curiosity, what were the `Other` entries in that survey?

One thing I worry about is the similarity between `dotnet nuget explain` and a potential future command I made up just now called `dotnet nuget info <package Id>` that dumps a formatting of the package's metadata in the console. How would you differentiate between these terms if we had such a command? It's a bit of a made-up exercise, because as far as I know no-one has asked for `dotnet nuget info` but 'explain' is a fairly wide design space so I wanted to call that out.

Reply:
@baronfel 
You're right, I corrected it.

Reply:
I like `why` better than `explain` as well, due to what the commands in other ecosystems are named.

Reply:
Here's some context on the `paket why` command, for comparison/inspiration in case folks haven't seen it before :)

Paket manages all dependencies at a solution level in a file that's essentially equivalent to a combination of CMPV _and_ Package Source Mapping. After the resolution step, all resolved versions are written to a lock file, and both of these files are intended to be checked in to source control.  As a result, `paket why` is pretty fast. 

The command takes a single argument, which is a dependency that's either a direct dependency, a transitive dependency, or an unknown dependency. The following output is taken from the fsharp/fsautocomplete repository.

Direct dependency:

```
➜ dotnet paket why Serilog
NuGet Serilog - 2.11 is a direct (paket.dependencies) dependency.
It is part of following dependency chains:

-> Destructurama.FSharp - 1.2
  -> Serilog - 2.11

-> Serilog.Sinks.Async - 1.5
  -> Serilog - 2.11

-> Serilog.Sinks.Console - 4.0.1
  -> Serilog - 2.11

-> Serilog.Sinks.File - 5.0
  -> Serilog - 2.11
```

This calls out the kind of dependency (direct/transitive), the location of the file that introduced the dependency, and then all of the dependencies that influenced the resolution of the requested dependency.

Transitive dependency:

```
➜ dotnet paket why System.Security.Cryptography.Pkcs
NuGet System.Security.Cryptography.Pkcs - 6.0.1 is a transitive dependency.
It is part of following dependency chains:

-> FSharp.Analyzers.SDK - 0.11
  -> FSharp.Compiler.Service - 41.0.5
    -> Microsoft.Build.Tasks.Core - 17.2
      -> System.Security.Cryptography.Pkcs - 6.0.1

-> FSharp.Formatting - 14.0.1
  -> FSharp.Compiler.Service - 41.0.5
    -> Microsoft.Build.Tasks.Core - 17.2
      -> System.Security.Cryptography.Pkcs - 6.0.1

-> FSharpLint.Core - 0.21.2
  -> FSharp.Compiler.Service - 41.0.5
    -> Microsoft.Build.Tasks.Core - 17.2
      -> System.Security.Cryptography.Pkcs - 6.0.1
```

Here you just get the dependency kind and the chains.

Missing/unknown dependency:

```
➜ dotnet paket why MyCoolPackage
NuGet MyCoolPackage was not found in paket.lock
```

Reply:
After @baronfel 's comment specifically, I think it would be best to go back to `dotnet nuget why` because it is very similar to paket why in terms of printing out dependency paths. Also I agree with @nkolev92 that the survey results can't be taken at face value. Therefore my vote is for `dotnet nuget why`. 

I know @JonDouglas was leaning towards `dotnet nuget explain`, do you have any further thoughts?

Reply:
For the sake of time, I think moving forward with the original proposal is ideal. At a later date we can alias to `explain` or another verb. entirely 

While my personal leaning is opposite due to my surprise of people's first impressions (i.e. start with `explain` and alias later to `why`), it really isn't a concern from my end.

I believe the best verbs here to describe the functionality as written in this proposal are `why` and `explain`. I believe that `tree` and `graph` would not be justified. What was clear to me when we first proposed this work is that majority of other package ecosystems have a "why" command with an alias(i.e. `explain`, `depends`, etc). Let's start here.